-- phpMyAdmin SQL Dump
-- version 4.2.12deb2+deb8u2
-- http://www.phpmyadmin.net
--
-- Host: localhost
-- Generation Time: Nov 15, 2016 at 07:01 PM
-- Server version: 5.5.52-0+deb8u1
-- PHP Version: 5.6.27-0+deb8u1

SET SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
SET time_zone = "+00:00";


/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8 */;

--
-- Database: `dbx`
--
CREATE DATABASE IF NOT EXISTS `dbx` DEFAULT CHARACTER SET latin1 COLLATE latin1_swedish_ci;
USE `dbx`;

-- --------------------------------------------------------

--
-- Table structure for table `agendamentos`
--

CREATE TABLE IF NOT EXISTS `agendamentos` (
`id` int(11) unsigned NOT NULL,
  `nome` varchar(64) NOT NULL,
  `comentario` varchar(255) DEFAULT NULL,
  `inicio` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `intervalo` int(11) NOT NULL,
  `ultimaExecucao` timestamp NULL DEFAULT NULL,
  `paralelismo` tinyint(1) NOT NULL DEFAULT '1',
  `ativo` tinyint(1) NOT NULL DEFAULT '1'
) ENGINE=MyISAM AUTO_INCREMENT=22 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `agendamentos`
--

INSERT INTO `agendamentos` (`id`, `nome`, `comentario`, `inicio`, `intervalo`, `ultimaExecucao`, `paralelismo`, `ativo`) VALUES
(21, 'Refinamento2', '2', '2016-11-03 02:00:00', 5, '2016-11-15 06:10:05', 1, 1),
(20, 'Refinamento', '1', '2016-11-02 02:00:00', 7, '2016-11-15 06:06:57', 1, 1),
(19, 'Coleta', '1', '2016-11-01 02:00:00', 7, '2016-11-15 06:06:57', 1, 1);

-- --------------------------------------------------------

--
-- Table structure for table `agendamentos_componentes`
--

CREATE TABLE IF NOT EXISTS `agendamentos_componentes` (
  `idAgendamento` int(10) unsigned NOT NULL,
  `idComponente` varchar(255) NOT NULL,
  `ordem` tinyint(3) unsigned NOT NULL
) ENGINE=MyISAM DEFAULT CHARSET=latin1;

--
-- Dumping data for table `agendamentos_componentes`
--

INSERT INTO `agendamentos_componentes` (`idAgendamento`, `idComponente`, `ordem`) VALUES
(19, 'ComponentePDB', 0),
(20, 'Graficos', 0),
(21, 'Graficos', 0);

-- --------------------------------------------------------

--
-- Table structure for table `artigos`
--

CREATE TABLE IF NOT EXISTS `artigos` (
`id` int(11) unsigned NOT NULL,
  `componente` int(10) unsigned NOT NULL,
  `proteina` int(10) unsigned NOT NULL,
  `idExterno` varchar(64) NOT NULL,
  `titulo` varchar(511) CHARACTER SET utf16 NOT NULL,
  `abstract` text CHARACTER SET utf8 COLLATE utf8_bin,
  `data` date NOT NULL,
  `link` varchar(512) DEFAULT NULL
) ENGINE=MyISAM AUTO_INCREMENT=159 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `artigos`
--

INSERT INTO `artigos` (`id`, `componente`, `proteina`, `idExterno`, `titulo`, `abstract`, `data`, `link`) VALUES
(1, 37, 1, '27663094', 'A new ''golden age'' for the antitubercular target InhA.', 'The increasing prevalence of multidrug-resistant strains of Mycobacterium tuberculosis is the main contributing factor in unfavorable outcomes in the treatment of tuberculosis. Studies suggest that direct inhibitors of InhA, an enoyl-ACP-reductase, might yield promising clinical candidates that can be developed into new antitubercular drugs. In this review, we describe the application of different hit-identification strategies to InhA, which clearly illustrate the druggability of its active site through distinct binding mechanisms. We further characterize four classes of InhA inhibitors that show novel binding modes, and provide evidence of their successful target engagement as well as their in vivo activity.', '2016-09-20', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=27663094'),
(2, 37, 1, '27490025', 'Pyrrolidinone and pyrrolidine derivatives: Evaluation as inhibitors of InhA and Mycobacterium tuberculosis.', 'A series of GEQ analogues bearing pyrrolidinone or pyrrolidine cores were synthesized and evaluated against InhA, essential target for Mycobacterium tuberculosis (M.tb) survival. The compounds were also evaluated against M.tb H37Rv growth. Interestingly, some of the compounds, not efficient as InhA inhibitors, are active against M.tb with MICs up to 1.4 μM. In particular, compound 4b was screened with different M.tb mutated strains in order to identify the cellular target, but without success, suggesting a new possible mode of action.', '2016-07-16', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=27490025'),
(3, 37, 1, '27428438', 'Antitubercular drugs for an old target: GSK693 as a promising InhA direct inhibitor.', 'Despite being one of the first antitubercular agents identified, isoniazid (INH) is still the most prescribed drug for prophylaxis and tuberculosis (TB) treatment and, together with rifampicin, the pillars of current chemotherapy. A high percentage of isoniazid resistance is linked to mutations in the pro-drug activating enzyme KatG, so the discovery of direct inhibitors (DI) of the enoyl-ACP reductase (InhA) has been pursued by many groups leading to the identification of different enzyme inhibitors, active against Mycobacterium tuberculosis (Mtb), but with poor physicochemical properties to be considered as preclinical candidates. Here, we present a series of InhA DI active against multidrug (MDR) and extensively (XDR) drug-resistant clinical isolates as well as in TB murine models when orally dosed that can be a promising foundation for a future treatment.', '2016-05-08', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=27428438'),
(4, 37, 1, '27393546', 'Detection of katG and inhA mutations to guide isoniazid and ethionamide use for drug-resistant tuberculosis.', 'Depending on the presence of mutations that determine isoniazid (INH) susceptibility (katG and inhA), Mycobacterium tuberculosis may be susceptible to high doses of INH or ethionamide (ETH).', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=27393546'),
(5, 37, 1, '27301022', 'Evaluation of the inhibitory activity of (aza)isoindolinone-type compounds: toward in vitro InhA action, Mycobacterium tuberculosis growth and mycolic acid biosynthesis.', 'Inhibitors of the Mycobacterium tuberculosis enoyl-ACP reductase (InhA) are considered as potential promising therapeutics for the treatment of tuberculosis. Previously, we reported that azaisoindolinone-type compounds displayed, in vitro, inhibitory activity toward InhA. Herein, we describe chemical modifications of azaisoindolinone scaffold, the synthesis of 15 new compounds and their evaluations toward the in vitro InhA activity. Based on these results, a structure-InhA inhibitory activity relationship analysis and a molecular docking study, using the conformation of InhA found in the 2H7M crystal structure, were carried out to predict a possible mode of interaction of the best (aza)isoindolinone-type inhibitors with InhA in vitro. Then, the work was extended toward evaluations of these compounds against Mycobacterium tuberculosis (Mtb) growth, and finally, some of them were also investigated in respect of their ability to inhibit mycolic acid biosynthesis inside mycobacteria. Although, some azaisoindolinones were able to inhibit InhA activity and Mtb growth in vitro, they did not inhibit the mycolic acid biosynthesis inside Mtb.', '2016-07-16', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=27301022'),
(6, 37, 1, '27206346', 'Molecular dynamics-based investigation of InhA substrate binding loop for diverse biological activity of direct InhA inhibitors.', 'The closed conformation of substrate binding loop (SBL) is considered significant for biological activity of direct InhA inhibitors (DIIs). However, exact interactions of SBL with inhibitors are not characterized yet to emphasize over SBL conformations. The seven InhA-DII complexes are analyzed using molecular dynamics simulation to deduce the mechanism for closed and open conformation of SBL. MMGBSA binding energy calculations and decompositions help to identify Ala198, Met199, Ile202, Val203, Ile215, and Leu218 in SBL region as the key residues. The interactions of DIIs with SBL residues particularly Ile202, Val203, Ile215, and Leu218 are found considerable for closed SBL conformation. This difference is accounted for closed state of SBL in 2X23, and open/moderately open state in other complexes. This study substantiates the loop ordering property of DIIs as the basis for high-affinity InhA inhibitors under the molecular recognition phenomena. This property can be used as a parameter to identify potential DIIs using virtual screening approaches.', '2016-05-20', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=27206346'),
(7, 37, 1, '26989290', 'Minor Contribution of inhA-15 Mutations to the Rapid Detection of Isoniazid Resistance in Mycobacterium Tuberculosis Isolates.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26989290'),
(8, 37, 1, '26934341', 'N-Benzyl-4-((heteroaryl)methyl)benzamides: A New Class of Direct NADH-Dependent 2-trans Enoyl-Acyl Carrier Protein Reductase (InhA) Inhibitors with Antitubercular Activity.', 'Isoniazid (INH) remains one of the cornerstones of antitubercular chemotherapy for drug-sensitive strains of M. tuberculosis bacteria. However, the increasing prevalence of multidrug-resistant (MDR) and extensively drug-resistant (XDR) strains containing mutations in the KatG enzyme, which is responsible for the activation of INH into its antitubercular form, have rendered this drug of little or no use in many cases of drug-resistant tuberculosis. Presented herein is a novel family of antitubercular direct NADH-dependent 2-trans enoyl-acyl carrier protein reductase (InhA) inhibitors based on an N-benzyl-4-((heteroaryl)methyl)benzamide template; unlike INH, these do not require prior activation by KatG. Given their direct InhA target engagement, these compounds should be able to circumvent KatG-related resistance in the clinic. The lead molecules were shown to be potent inhibitors of InhA and showed activity against M. tuberculosis bacteria. This new family of inhibitors was found to be chemically tractable, as exemplified by the facile synthesis of analogues and the establishment of structure-activity relationships. Furthermore, a co-crystal structure of the initial hit with the enzyme is disclosed, providing valuable information toward the design of new InhA inhibitors for the treatment of MDR/XDR tuberculosis.', '2016-03-02', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26934341'),
(9, 37, 1, '26900657', 'New direct inhibitors of InhA with antimycobacterial activity based on a tetrahydropyran scaffold.', 'Tetrahydropyran derivative 1 was discovered in a high-throughput screening campaign to find new inhibitors of mycobacterial InhA. Following initial in-vitro profiling, a structure-activity relationship study was initiated and a focused library of analogs was synthesized and evaluated. This yielded compound 42 with improved antimycobacterial activity and low cytotoxicity. Additionally, the crystal structure of InhA in complex with inhibitor 1 was resolved, to reveal the binding mode and provide hints for further optimization.', '2016-02-08', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26900657'),
(10, 37, 1, '26820895', 'Molecular docking studies on InhA, MabA and PanK enzymes from Mycobacterium tuberculosis of ellagic acid derivatives from Ludwigia adscendens and Trewia nudiflora.', 'There is an urgent need to discover and develop new drugs to combat Mycobacterium tuberculosis, the causative agent of tuberculosis (TB) in humans. In recent years, there has been a renewed interest in the discovery of new anti-TB agents from natural sources. In the present investigation, molecular docking studies were carried out on two ellagic acid derivatives, namely pteleoellagic acid (1) isolated from Ludwigia adscendens, and 3,3''-di-O-methyl ellagic acid 4-O-α-rhamnopyranoside (2) isolated from Trewia nudiflora, to investigate their binding to two enzymes involved in M. tuberculosis cell wall biogenesis, namely 2-trans-enoyl-ACP reductase (InhA) and β-ketoacyl-ACP reductase (MabA), and to pantothenate kinase (PanK type I) involved in the biosynthesis of coenzyme A, essential for the growth of M. tuberculosis.', '2015-12-08', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26820895'),
(11, 37, 1, '26745525', 'Feeding Anthrax: The Crystal Structure of Bacillus anthracis InhA Protease.', 'Pathogenic bacteria secrete proteases to evade host defense and to acquire nutrients. In this issue of Structure, Arolas et al. (2016) describe the structural basis of activation and latency of InhA, a major secreted protease of Bacillus anthracis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26745525'),
(12, 37, 1, '26730986', 'In silico driven design and synthesis of rhodanine derivatives as novel antibacterials targeting the enoyl reductase InhA.', 'Here, we report on the design, synthesis and biological evaluation of 4-thiazolidinone (rhodanine) de-rivatives targeting Mycobacterial tuberculosis (Mtb) trans-2-enoyl-acyl carrier protein reductase (InhA). Compounds having bulky aromatic substituents at position 5 and a tryptophan residue at position N-3 of the rhodanine ring were the most active against InhA with IC50 values ranging from 2.7 µM to 30 µM. The experimental data showed consistent correlations with computational studies. Their anti-microbial activity was assessed against Mycobacterium marinum (Mm) (a model for Mtb), Pseudomonas aeruginosa (Pa), Legionella pneumophila (Lp) and Enterococcus faecalis (Ef), by using anti-infective, anti-virulence and antibiotic assays. 19 out of 34 compounds reduced Mm virulence at 10 µM. 33 exhibited promising antibiotic activity against Mm with a MIC of 0.21 µM and showed up to 89% reduction of Lp growth in an anti-infective assay at 30 µM. 32 showed high antibiotic activity against Ef with a MIC of 0.57 µM.', '2016-01-05', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26730986'),
(13, 37, 1, '26658674', 'Molecular Dynamics Assisted Mechanistic Study of Isoniazid-Resistance against Mycobacterium tuberculosis InhA.', 'Examination of InhA mutants I16T, I21V, I47T, S94A, and I95P showed that direct and water mediated H-bond interactions between NADH and binding site residues reduced drastically. It allowed conformational flexibility to NADH, particularly at the pyrophosphate region, leading to weakening of its binding at dinucleotide binding site. The highly scattered distribution of pyrophosphate dihedral angles and chi1 side chain dihedral angles of corresponding active site residues therein confirmed weak bonding between InhA and NADH. The average direct and water mediated bridged H-bond interactions between NADH and mutants were observed weaker as compared to wild type. Further, estimated NADH binding free energy in mutants supported the observed weakening of InhA-NADH interactions. Similarly, per residue contribution to NADH binding was also found little less as compared to corresponding residues in wild type. This investigation clearly depicted and supported the effect of mutations on NADH binding and can be accounted for isoniazid resistance as suggested by previous biochemical and mutagenic studies. Further, structural analysis of InhA provided the crucial points to enhance the NADH binding affinity towards InhA mutants in the presence of direct InhA inhibitors to combat isoniazid drug resistance. This combination could be a potential alternative for treatment of drug resistant tuberculosis.', '2015-12-14', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26658674'),
(14, 37, 1, '26654447', 'Placental DNA methylation at term reflects maternal serum levels of INHA and FN1, but not PAPPA, early in pregnancy.', 'Early detection of pregnancies at risk of complications, such as intrauterine growth restriction (IUGR) and preeclampsia (PE), is critical for improved monitoring and preventative treatment to optimize health outcomes. We predict that levels of placental-derived proteins circulating in maternal blood reflect placental gene expression, which is associated with placental DNA methylation (DNAm) profiles. As such, placental DNAm profiling may be useful to distinguish pregnancies at risk of developing complications and correlation between DNAm and protein levels in maternal blood may give further evidence for a protein''s use as a biomarker. However, few studies investigate all clinical parameters that may influence DNAm and/or protein expression, which can significantly affect the relationship between these measures.', '2015-12-11', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26654447'),
(15, 37, 1, '26332235', 'High Prevalence of inhA Promoter Mutations among Patients with Drug-Resistant Tuberculosis in KwaZulu-Natal, South Africa.', 'Drug-resistant tuberculosis (TB) remains extremely difficult to treat because there are often few remaining active medications and limited diagnostic options to detect resistance. Resistance to isoniazid is typically caused by mutations in either katG or the inhA promoter. inhA mutations confer low-level resistance to isoniazid and cross-resistance to ethionamide while katG mutations confer high-level isoniazid resistance and no cross-resistance. Line Probe Assays (LPAs) that detect mutations in katG and inhA are currently performed on all positive TB cultures in KwaZulu-Natal province, South Africa, but the frequency of inhA mutations in drug-resistant TB patients has not been examined.', '2015-09-02', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26332235'),
(16, 37, 1, '26227776', 'Radiolabelling and positron emission tomography of PT70, a time-dependent inhibitor of InhA, the Mycobacterium tuberculosis enoyl-ACP reductase.', 'PT70 is a diaryl ether inhibitor of InhA, the enoyl-ACP reductase in the Mycobacterium tuberculosis fatty acid biosynthesis pathway. It has a residence time of 24 min on the target, and also shows antibacterial activity in a mouse model of tuberculosis infection. Due to the interest in studying target tissue pharmacokinetics of PT70, we developed a method to radiolabel PT70 with carbon-11 and have studied its pharmacokinetics in mice and baboons using positron emission tomography.', '2015-07-14', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26227776'),
(17, 37, 1, '26218832', 'An Effective Approach for Clustering InhA Molecular Dynamics Trajectory Using Substrate-Binding Cavity Features.', 'Protein receptor conformations, obtained from molecular dynamics (MD) simulations, have become a promising treatment of its explicit flexibility in molecular docking experiments applied to drug discovery and development. However, incorporating the entire ensemble of MD conformations in docking experiments to screen large candidate compound libraries is currently an unfeasible task. Clustering algorithms have been widely used as a means to reduce such ensembles to a manageable size. Most studies investigate different algorithms using pairwise Root-Mean Square Deviation (RMSD) values for all, or part of the MD conformations. Nevertheless, the RMSD only may not be the most appropriate gauge to cluster conformations when the target receptor has a plastic active site, since they are influenced by changes that occur on other parts of the structure. Hence, we have applied two partitioning methods (k-means and k-medoids) and four agglomerative hierarchical methods (Complete linkage, Ward''s, Unweighted Pair Group Method and Weighted Pair Group Method) to analyze and compare the quality of partitions between a data set composed of properties from an enzyme receptor substrate-binding cavity and two data sets created using different RMSD approaches. Ensembles of representative MD conformations were generated by selecting a medoid of each group from all partitions analyzed. We investigated the performance of our new method for evaluating binding conformation of drug candidates to the InhA enzyme, which were performed by cross-docking experiments between a 20 ns MD trajectory and 20 different ligands. Statistical analyses showed that the novel ensemble, which is represented by only 0.48% of the MD conformations, was able to reproduce 75% of all dynamic behaviors within the binding cavity for the docking experiments performed. Moreover, this new approach not only outperforms the other two RMSD-clustering solutions, but it also shows to be a promising strategy to distill biologically relevant information from MD trajectories, especially for docking purposes.', '2015-07-28', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26218832'),
(18, 37, 1, '26157154', 'Characterization of a Clone of Mycobacterium tuberculosis Clinical Isolates with Mutations in KatG (A110V), EthA (Q269STOP), and the inhA Promoter (-15C→T).', NULL, '2015-07-08', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26157154'),
(19, 37, 1, '26147157', 'Rational Modulation of the Induced-Fit Conformational Change for Slow-Onset Inhibition in Mycobacterium tuberculosis InhA.', 'Slow-onset enzyme inhibitors are the subject of considerable interest as an approach to increasing the potency of pharmaceutical compounds by extending the residence time of the inhibitor on the target (the lifetime of the drug-receptor complex). However, rational modulation of residence time presents significant challenges because it requires additional mechanistic insight, such as the nature of the transition state for postbinding isomerization. Our previous work, based on X-ray crystallography, enzyme kinetics, and molecular dynamics simulation, suggested that the slow step in inhibition of the Mycobacterium tuberculosis enoyl-ACP reductase InhA involves a change in the conformation of the substrate binding loop from an open state in the initial enzyme-inhibitor complex to a closed state in the final enzyme-inhibitor complex. Here, we use multidimensional free energy landscapes for loop isomerization to obtain a computational model for the transition state. The results suggest that slow-onset inhibitors crowd key side chains on helices that slide past each other during isomerization, resulting in a steric clash. The landscapes become significantly flatter when residues involved in the steric clash are replaced with alanine. Importantly, this lower barrier can be increased by rational inhibitor redesign to restore the steric clash. Crystallographic studies and enzyme kinetics confirm the predicted effects on loop structure and flexibility, as well as inhibitor residence time. These loss and regain of function studies validate our mechanistic hypothesis for interactions controlling substrate binding loop isomerization, providing a platform for the future design of inhibitors with longer residence times and better in vivo potency. Similar opportunities for slow-onset inhibition via the same mechanism are identified in other pathogens.', '2015-07-24', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26147157'),
(20, 37, 1, '26142487', 'Design, synthesis and evaluation of new GEQ derivatives as inhibitors of InhA enzyme and Mycobacterium tuberculosis growth.', 'A series of fluorene-based derivatives was synthesized and evaluated for inhibiting both InhA and Mycobacterium tuberculosis growth. These compounds were inspired by the previously reported Genz-10850 molecule, a good InhA inhibitor, but with a poor activity against M. tuberculosis growth. Structure-activity relationships were performed by introducing the following chemical modifications: 1) the piperazine ring; 2) the amide group; 3) the aryl moiety; and 4) the fluorene moiety. Among these new derivatives, one of them was more effective against both the InhA activity and mycobacterial growth, compared to the hit compound. Docking studies were also performed to rationalize activities of these derivatives. Furthermore, we showed for the first time that efflux pump inhibitors potentiated the efficacy of Genz-10850 (GEQ) derivatives against M. tuberculosis growth, demonstrating that these compounds could be substrates of some efflux pumps.', '2015-06-20', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=26142487'),
(21, 37, 1, '25996598', 'Slow-Onset Inhibition of Mycobacterium tuberculosis InhA: Revealing Molecular Determinants of Residence Time by MD Simulations.', 'An important kinetic parameter for drug efficacy is the residence time of a compound at a drug target, which is related to the dissociation rate constant koff. For the essential antimycobacterial target InhA, this parameter is most likely governed by the ordering of the flexible substrate binding loop (SBL). Whereas the diphenyl ether inhibitors 6PP and triclosan (TCL) do not show loop ordering and thus, no slow-binding inhibition and high koff values, the slightly modified PT70 leads to an ordered loop and a residence time of 24 minutes. To assess the structural differences of the complexes from a dynamic point of view, molecular dynamics (MD) simulations with a total sampling time of 3.0 µs were performed for three ligand-bound and two ligand-free (perturbed) InhA systems. The individual simulations show comparable conformational features with respect to both the binding pocket and the SBL, allowing to define five recurring conformational families. Based on their different occurrence frequencies in the simulated systems, the conformational preferences could be linked to structural differences of the respective ligands to reveal important determinants of residence time. The most abundant conformation besides the stable EI* state is characterized by a shift of Ile202 and Val203 toward the hydrophobic pocket of InhA. The analyses revealed potential directions for avoiding this conformational change and, thus, hindering rapid dissociation: (1) an anchor group in 2''-position of the B-ring for scaffold stabilization, (2) proper occupation of the hydrophobic pocket, and (3) the introduction of a barricade substituent in 5''-position of the diphenyl ether B-ring.', '2015-05-21', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25996598'),
(22, 37, 1, '25891098', 'Crystal structure of the enoyl-ACP reductase of Mycobacterium tuberculosis (InhA) in the apo-form and in complex with the active metabolite of isoniazid pre-formed by a biomimetic approach.', 'InhA is an enoyl-ACP reductase of Mycobacterium tuberculosis implicated in the biosynthesis of mycolic acids, essential constituents of the mycobacterial cell wall. To date, this enzyme is considered as a promising target for the discovery of novel antitubercular drugs. In this work, we describe the first crystal structure of the apo form of the wild-type InhA at 1.80Å resolution as well as the crystal structure of InhA in complex with the synthetic metabolite of the antitubercular drug isoniazid refined to 1.40Å. This metabolite, synthesized in the absence of InhA, is able to displace and replace the cofactor NADH in the enzyme active site. This work provides a unique opportunity to enlighten the structural adaptation of apo-InhA to the binding of the NADH cofactor or of the isoniazid adduct. In addition, a differential scanning fluorimetry study of InhA, in the apo-form as well as in the presence of NAD(+), NADH and INH-NADH was performed showing that binding of the INH-NADH adduct had a strong stabilizing effect.', '2015-04-17', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25891098'),
(23, 37, 1, '25800133', 'Synthesis, evaluation and in silico molecular modeling of pyrroyl-1,3,4-thiadiazole inhibitors of InhA.', 'Enoyl acyl carrier protein reductase (ENR)is an essential type II fatty acid synthase (FAS-II) pathway enzyme that is an attractive target for designing novel antitubercular agents.Herein, we report sixty-eight novel pyrrolyl substituted aryloxy-1,3,4-thiadiazoles synthesized by three-step optimization processes. Three-dimensional quantitative structure-activity relationships (3D-QSAR) were established for pyrrolyl substituted aryloxy-1,3,4-thiadiazole series of InhA inhibitors using the comparative molecular field analysis (CoMFA).Docking analysis of the crystal structure of ENR performed by using Surflex-Dock in Sybyl-X 2.0 software indicates the occupation of pyrrolyl substituted aryloxy 1,3,4-thiadiazole into hydrophobic pocket of InhA enzyme. Based on docking and database alignment rules, two computational models were established to compare their statistical results. The analysis of 3D contour plots allowed us to investigate the effect of different substituent groups at different positions of the common scaffold. In vitro testing of ligands using biological assays substantiated the efficacy of ligands that were screened through in silico methods.', '2015-03-11', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25800133'),
(24, 37, 1, '25778993', 'Discovery of InhA inhibitors with anti-mycobacterial activity through a matched molecular pair approach.', 'The Mycobacterium tuberculosis (M. tuberculosis) enoyl-acyl carrier protein reductase (mtInhA) is an attractive enzyme and a thoroughly studied target for tuberculosis therapy. In this study, to identify novel structure-activity relationships (SARs) of mtInhA inhibitors, a series of diphenyl ether derivatives were designed based on the matched molecular pair (MMP) method, and the binding energies of these compounds were subsequently estimated by in silico structure-based drug screening (SBDS) to provide more useful data. Consequently, the 10 unique candidate compounds (KEM1-KEM10) were identified and assessed for the inhibition of mtInhA enzymatic activity, in vitro antibiotic effects against model mycobacteria and toxicity level on both intestinal bacteria and mammalian cells. Among the compounds tested, phenyl group (KEM4) and 2-fluorobenzyl group (KEM7) substitutions produced preferable inhibitory effects on mtInhA enzymatic activity relative to those provided by a furyl group (KES4: base compound) at the terminal of the compound, and KEM7 inhibited the growth of the mycobacteria strain with a lower IC50 value. Moreover, most of the candidate compounds exhibited neither inhibition of the growth of enterobacteria nor toxic effects on mammalian cells, though KEM10 exhibited toxicity against cultured MDCK cells. The structural and experimental information concerning these mtInhA inhibitors identified through MMP-based in silico screening will likely contribute to the lead optimisation of novel antibiotics for M. tuberculosis.', '2015-03-05', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25778993'),
(25, 37, 1, '25636146', 'A virtual screen discovers novel, fragment-sized inhibitors of Mycobacterium tuberculosis InhA.', 'Isoniazid (INH) is usually administered to treat latent Mycobacterium tuberculosis (Mtb) infections and is used in combination therapy to treat active tuberculosis (TB). Unfortunately, resistance to this drug is hampering its clinical effectiveness. INH is a prodrug that must be activated by Mtb catalase-peroxidase (KatG) before it can inhibit InhA (Mtb enoyl-acyl-carrier-protein reductase). Isoniazid-resistant cases of TB found in clinical settings usually involve mutations in or deletion of katG, which abrogate INH activation. Compounds that inhibit InhA without requiring prior activation by KatG would not be affected by this resistance mechanism and hence would display continued potency against these drug-resistant isolates of Mtb. Virtual screening experiments versus InhA in the GO Fight Against Malaria (GO FAM) project were designed to discover new scaffolds that display base-stacking interactions with the NAD cofactor. GO FAM experiments included targets from other pathogens, including Mtb, when they had structural similarity to a malaria target. Eight of the 16 soluble compounds identified by docking against InhA plus visual inspection were modest inhibitors and did not require prior activation by KatG. The best two inhibitors discovered are both fragment-sized compounds and displayed Ki values of 54 and 59 μM, respectively. Importantly, the novel inhibitors discovered have low structural similarity to known InhA inhibitors and thus help expand the number of chemotypes on which future medicinal chemistry efforts can be focused. These new fragment hits could eventually help advance the fight against INH-resistant Mtb strains, which pose a significant global health threat.', '2015-02-17', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25636146'),
(26, 37, 1, '25568071', 'Direct inhibitors of InhA are active against Mycobacterium tuberculosis.', 'New chemotherapeutic agents are urgently required to combat the global spread of multidrug-resistant tuberculosis (MDR-TB). The mycobacterial enoyl reductase InhA is one of the few clinically validated targets in tuberculosis drug discovery. We report the identification of a new class of direct InhA inhibitors, the 4-hydroxy-2-pyridones, using phenotypic high-throughput whole-cell screening. This class of orally active compounds showed potent bactericidal activity against common isoniazid-resistant TB clinical isolates. Biophysical studies revealed that 4-hydroxy-2-pyridones bound specifically to InhA in an NADH (reduced form of nicotinamide adenine dinucleotide)-dependent manner and blocked the enoyl substrate-binding pocket. The lead compound NITD-916 directly blocked InhA in a dose-dependent manner and showed in vivo efficacy in acute and established mouse models of Mycobacterium tuberculosis infection. Collectively, our structural and biochemical data open up new avenues for rational structure-guided optimization of the 4-hydroxy-2-pyridone class of compounds for the treatment of MDR-TB.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25568071'),
(27, 37, 1, '25517015', 'Design, synthesis, and evaluation of new thiadiazole-based direct inhibitors of enoyl acyl carrier protein reductase (InhA) for the treatment of tuberculosis.', 'Mycobacterial enoyl acyl carrier protein reductase (InhA) is a clinically validated target for the treatment of tuberculosis infections, a disease that still causes the death of at least a million people annually. A known class of potent, direct, and competitive InhA inhibitors based on a tetracyclic thiadiazole structure has been shown to have in vivo activity in murine models of tuberculosis infection. On the basis of this template, we have here explored the medicinal chemistry of truncated analogues that have only three aromatic rings. In particular, compounds 8b, 8d, 8f, 8l, and 8n show interesting features, including low nanomolar InhA IC50, submicromolar antimycobacterial potency, and improved physicochemical profiles in comparison with the tetracyclic analogues. From this series, 8d is identified as having the best balance of potency and properties, whereby the resolved 8d S-enatiomer shows encouraging in vivo efficacy.', '2014-12-29', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25517015'),
(28, 37, 1, '25344830', 'Automated liquid culture system misses isoniazid heteroresistance in Mycobacterium tuberculosis isolates with mutations in the promoter region of the inhA gene.', 'Heteroresistance in Mycobacterium tuberculosis isolates remains the major challenge for phenotypic drug susceptibility testing (DST) methods to detect drug resistance. The aim of this study was to investigate the abilities of phenotypic DST methods to identify the isoniazid (INH) heteroresistance in M. tuberculosis. We found that the broth dilution method was able to detect INH resistance if 0.5 % resistant bacteria with mutations in the katG and oxyR-ahpC regions were present, while the detection limit ranged from 1 to 10 % for the INH-resistant strains harboring inhA mutations, which was associated with the different mutant types. Additionally, MGIT DST was able to find the recommended 1 % INH resistance due to katG mutations. In contrast, MGIT DST detected resistance in suspensions with 20 % resistant bacteria with inhA mutations. Statistical analysis revealed that the ability of the broth dilution method to detect heteroresistance was better than that of the MGIT DST (p = 0.004). When we further pairwise compared the two methods for detecting heteroresistance according to different mutant loci, the broth dilution method found more heteroresistance due to inhA mutations than MGIT DST (p = 0.001), while the differences for katG and oxyR-ahpC mutations were both not statistically significant (p > 0.05). In conclusion, our findings demonstrate that MGIT DST fails to detect INH heteroresistance in M. tuberculosis isolates with mutations in the promoter region of inhA. In addition, the broth dilution method is more sensitive than MGIT DST in finding INH heteroresistance, indicating that this method may serve as an alternative method to detect the heteroresistance of M. tuberculosis.', '2014-10-26', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25344830'),
(29, 37, 1, '25282650', 'Development of benzo[d]oxazol-2(3H)-ones derivatives as novel inhibitors of Mycobacterium tuberculosis InhA.', 'A series of twenty seven substituted 2-(2-oxobenzo[d]oxazol-3(2H)-yl)acetamide derivatives were designed based on our earlier reported Mycobacterium tuberculosis (MTB) enoyl-acyl carrier protein reductase (InhA) lead. Compounds were evaluated for MTB InhA inhibition study, in vitro activity against drug-sensitive and -resistant MTB strains, and cytotoxicity against RAW 264.7 cell line. Among the compounds tested, 2-(6-nitro-2-oxobenzo[d]oxazol-3(2H)-yl)-N-(5-nitrothiazol-2-yl)acetamide (30) was found to be the most promising compound with IC50 of 5.12 ± 0.44 μM against MTB InhA, inhibited drug sensitive MTB with MIC 17.11 μM and was non-cytotoxic at 100 μM. The interaction with protein and enhancement of protein stability in complex with compound 30 was further confirmed biophysically by differential scanning fluorimetry.', '2014-09-04', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25282650'),
(30, 37, 1, '25218910', 'Development of 2-(4-oxoquinazolin-3(4H)-yl)acetamide derivatives as novel enoyl-acyl carrier protein reductase (InhA) inhibitors for the treatment of tuberculosis.', 'InhA, the enoyl acyl carrier protein reductase of Mycobacterium tuberculosis (MTB) is an attractive target for developing novel anti-tubercular agents. Twenty eight 2-(4-oxoquinazolin-3(4H)-yl)acetamide derivatives were synthesized and evaluated for their in vitro MTB InhA inhibition. Compounds were further evaluated for their in vitro activity against drug sensitive and resistant MTB strains and cytotoxicity against RAW 264.7 cell line. Compounds were docked at the active site of InhA to understand their binding mode and differential scanning fluorimetry was performed to ascertain their protein interaction and stability.', '2014-09-09', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25218910'),
(31, 37, 1, '25165007', 'Biological evaluation of potent triclosan-derived inhibitors of the enoyl-acyl carrier protein reductase InhA in drug-sensitive and drug-resistant strains of Mycobacterium tuberculosis.', 'New triclosan (TRC) analogues were evaluated for their activity against the enoyl-acyl carrier protein reductase InhA in Mycobacterium tuberculosis (Mtb). TRC is a well-known inhibitor of InhA, and specific modifications to its positions 5 and 4'' afforded 27 derivatives; of these compounds, seven derivatives showed improved potency over that of TRC. These analogues were active against both drug-susceptible and drug-resistant Mtb strains. The most active compound in this series, 4-(n-butyl)-1,2,3-triazolyl TRC derivative 3, had an MIC value of 0.6 μg mL(-1) (1.5 μM) against wild-type Mtb. At a concentration equal to its MIC, this compound inhibited purified InhA by 98 %, and showed an IC50 value of 90 nM. Compound 3 and the 5-methylisoxazole-modified TRC 14 were able to inhibit the biosynthesis of mycolic acids. Furthermore, mc(2) 4914, an Mtb strain overexpressing inhA, was found to be less susceptible to compounds 3 and 14, supporting the notion that InhA is the likely molecular target of the TRC derivatives presented herein.', '2014-08-27', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25165007'),
(32, 37, 1, '25111790', 'Inhibin alpha-subunit (INHA) expression in adrenocortical cancer is linked to genetic and epigenetic INHA promoter variation.', 'Adrenocortical carcinoma (ACC) is a rare, but highly malignant tumor of unknown origin. Inhibin α-subunit (Inha) knockout mice develop ACCs following gonadectomy. In man, INHA expression varies widely within ACC tissues and its circulating peptide inhibin pro-αC has been described as a novel tumor marker for ACC. We investigated whether genetic and epigenetic changes of the INHA gene in human ACC cause loss or variation of INHA expression. To this end, analyses of INHA sequence, promoter methylation and mRNA expression were performed in human adrenocortical tissues. Serum inhibin pro-αC levels were also measured in ACC patients. INHA genetic analysis in 37 unique ACCs revealed 10 novel, heterozygous rare variants. Of the 3 coding bases affected, one variant was synonymous and two were missense variants: S72F and S184F. The minor allele of rs11893842 at -124 bp was observed at a low frequency (24%) in ACC samples and was associated with decreased INHA mRNA levels: 4.7±1.9 arbitrary units for AA, compared to 26±11 for AG/GG genotypes (P = 0.034). The methylation of four proximal INHA promoter CpGs was aberrantly increased in five ACCs (47.7±3.9%), compared to normal adrenals (18.4±0.6%, P = 0.0052), whereas the other 14 ACCs studied showed diminished promoter methylation (9.8±1.1%, P = 0.020). CpG methylation was inversely correlated to INHA mRNA levels in ACCs (r = -0.701, p = 0.0036), but not associated with serum inhibin pro-αC levels. In conclusion, aberrant methylation and common genetic variation in the INHA promoter occur in human ACCs and are associated with decreased INHA expression.', '2014-08-11', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=25111790'),
(33, 37, 1, '24895891', 'Insights into the bonding pattern for characterizing the open and closed state of the substrate-binding loop in Mycobacterium tuberculosis InhA.', 'Direct InhA inhibitors, which interact with the substrate-binding loop (SBL) and order it into a closed state, are thought to be potential anti-multidrug-resistant tuberculosis molecules. Thus, developing parameters to distinguish between the open and closed state of SBL can help in screening the potent inhibitors with loop ordering properties.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24895891'),
(34, 37, 1, '24821057', 'In house reverse membrane hybridisation assay versus GenoType MTBDRplus and their performance to detect mutations in the genes rpoB, katG and inhA.', 'Drug-resistant tuberculosis (TB) threatens global TB control and is a major public health concern in several countries. We therefore developed a multiplex assay (LINE-TB/MDR) that is able to identify the most frequent mutations related to rifampicin (RMP) and isoniazid (INH) resistance. The assay is based on multiplex polymerase chain reaction, membrane hybridisation and colorimetric detection targeting of rpoB and katG genes, as well as the inhA promoter, which are all known to carry specific mutations associated with multidrug-resistant TB (MDR-TB). The assay was validated on a reference panel of 108 M. tuberculosis isolates that were characterised by the proportion method and by DNA sequencing of the targets. When comparing the performance of LINE-TB/MDR with DNA sequencing, the sensitivity, specificity and agreement were 100%, 100% and 100%, respectively, for RMP and 77.6%, 90.6% and 88.9%, respectively, for INH. Using drug sensibility testing as a reference standard, the performance of LINE-TB/MDR regarding sensitivity, specificity and agreement was 100%, 100% and 100% (95%), respectively, for RMP and 77%, 100% and 88.7% (82.2-95.1), respectively, for INH. LINE-TB/MDR was compared with GenoType MTBDRplus for 65 isolates, resulting in an agreement of 93.6% (86.7-97.5) for RIF and 87.4% (84.3-96.2) for INH. LINE-TB/MDR warrants further clinical validation and may be an affordable alternative for MDR-TB diagnosis.', '2014-05-07', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24821057'),
(35, 37, 1, '24785640', 'Rational design of InhA inhibitors in the class of diphenyl ether derivatives as potential anti-tubercular agents using molecular dynamics simulations.', 'A series of diphenyl ether derivatives were developed and showed promising potency for inhibiting InhA, an essential enoyl acyl carrier protein reductase involved in mycolic acid biosynthesis, leading to the lysis of Mycobacterium tuberculosis. To understand the structural basis of diphenyl ether derivatives for designing more potent inhibitors, molecular dynamics (MD) simulations were performed. Based on the obtained results, the dynamic behaviour in terms of flexibility, binding free energy, binding energy decomposition, conformation, and the inhibitor-enzyme interaction of diphenyl ether inhibitors were elucidated. Phe149, Tyr158, Met161, Met199, Val203 and NAD+ are the key residues for binding of diphenyl ether inhibitors in the InhA binding pocket. Our results could provide the structural concept to design new diphenyl ether inhibitors with better enzyme inhibitory activity against M. tuberculosis InhA. The present work facilitates the design of new and potentially more effective anti-tuberculosis agents.', '2014-04-30', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24785640'),
(36, 37, 1, '24616444', 'Time-dependent diaryl ether inhibitors of InhA: structure-activity relationship studies of enzyme inhibition, antibacterial activity, and in vivo efficacy.', 'The diaryl ethers are a novel class of antituberculosis drug candidates that inhibit InhA, the enoyl-ACP reductase involved in the fatty acid biosynthesis (FASII) pathway, and have antibacterial activity against both drug-sensitive and drug-resistant strains of Mycobacterium tuberculosis. In the present work, we demonstrate that two time-dependent B-ring modified diaryl ether InhA inhibitors have antibacterial activity in a mouse model of TB infection when delivered by intraperitoneal injection. We propose that the efficacy of these compounds is related to their residence time on the enzyme, and to identify structural features that modulate drug-target residence time in this system, we have explored the inhibition of InhA by a series of B-ring modified analogues. Seven ortho-substituted compounds were found to be time-dependent inhibitors of InhA, where the slow step leading to the final enzyme-inhibitor complex (EI*) is thought to correlate with closure and ordering of the InhA substrate binding loop. A detailed mechanistic understanding of the molecular basis for residence time in this system will facilitate the development of InhA inhibitors with improved in vivo activity.', '2014-03-11', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24616444'),
(37, 37, 1, '24527857', 'A structural and energetic model for the slow-onset inhibition of the Mycobacterium tuberculosis enoyl-ACP reductase InhA.', 'Slow-onset enzyme inhibitors are of great interest for drug discovery programs since the slow dissociation of the inhibitor from the drug-target complex results in sustained target occupancy leading to improved pharmacodynamics. However, the structural basis for slow-onset inhibition is often not fully understood, hindering the development of structure-kinetic relationships and the rational optimization of drug-target residence time. Previously we demonstrated that slow-onset inhibition of the Mycobacterium tuberculosis enoyl-ACP reductase InhA correlated with motions of a substrate-binding loop (SBL) near the active site. In the present work, X-ray crystallography and molecular dynamics simulations have been used to map the structural and energetic changes of the SBL that occur upon enzyme inhibition. Helix-6 within the SBL adopts an open conformation when the inhibitor structure or binding kinetics is substrate-like. In contrast, slow-onset inhibition results in large-scale local refolding in which helix-6 adopts a closed conformation not normally populated during substrate turnover. The open and closed conformations of helix-6 are hypothesized to represent the EI and EI* states on the two-step induced-fit reaction coordinate for enzyme inhibition. These two states were used as the end points for nudged elastic band molecular dynamics simulations resulting in two-dimensional potential energy profiles that reveal the barrier between EI and EI*, thus rationalizing the binding kinetics observed with different inhibitors. Our findings indicate that the structural basis for slow-onset kinetics can be understood once the structures of both EI and EI* have been identified, thus providing a starting point for the rational control of enzyme-inhibitor binding kinetics.', '2014-03-10', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24527857'),
(38, 37, 1, '24450589', 'Encoded library technology as a source of hits for the discovery and lead optimization of a potent and selective class of bactericidal direct inhibitors of Mycobacterium tuberculosis InhA.', 'Tuberculosis (TB) is one of the world''s oldest and deadliest diseases, killing a person every 20 s. InhA, the enoyl-ACP reductase from Mycobacterium tuberculosis, is the target of the frontline antitubercular drug isoniazid (INH). Compounds that directly target InhA and do not require activation by mycobacterial catalase peroxidase KatG are promising candidates for treating infections caused by INH resistant strains. The application of the encoded library technology (ELT) to the discovery of direct InhA inhibitors yielded compound 7 endowed with good enzymatic potency but with low antitubercular potency. This work reports the hit identification, the selected strategy for potency optimization, the structure-activity relationships of a hundred analogues synthesized, and the results of the in vivo efficacy studies performed with the lead compound 65.', '2014-02-05', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24450589'),
(39, 37, 1, '24292073', 'Pyridomycin bridges the NADH- and substrate-binding pockets of the enoyl reductase InhA.', 'Pyridomycin, a natural product with potent antituberculosis activity, inhibits a major drug target, the InhA enoyl reductase. Here, we unveil the co-crystal structure and unique ability of pyridomycin to block both the NADH cofactor- and lipid substrate-binding pockets of InhA. This is to our knowledge a first-of-a-kind binding mode that discloses a new means of InhA inhibition. Proof-of-principle studies show how structure-assisted drug design can improve the activity of new pyridomycin derivatives.', '2013-12-01', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24292073'),
(40, 37, 1, '24269516', 'Synthesis of 3-heteryl substituted pyrrolidine-2,5-diones via catalytic Michael reaction and evaluation of their inhibitory activity against InhA and Mycobacterium tuberculosis.', 'In the present paper, we report the synthesis via catalytic Michael reaction and biological results of a series of 3-heteryl substituted pyrrolidine-2,5-dione derivatives as moderate inhibitors against Mycobacterium tuberculosis H37Rv growth. Some of them present also inhibition activities against InhA.', '2013-11-04', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24269516'),
(41, 37, 1, '24184004', 'The mutations of katG and inhA genes of isoniazid-resistant Mycobacterium tuberculosis isolates in Taiwan.', 'The isoniazid (INH) resistance of Mycobacterium tuberculosis is caused by mutations in the katG and inhA genes encoding for catalase-peroxidase and inhA, respectively. Sequences of the katG and inhA gene of 70 isolates were analyzed to identify the mutations and to compare the mutations with their related susceptibilities.', '2013-11-01', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24184004'),
(42, 37, 1, '24140915', 'Design, chemical synthesis of 3-(9H-fluoren-9-yl)pyrrolidine-2,5-dione derivatives and biological activity against enoyl-ACP reductase (InhA) and Mycobacterium tuberculosis.', 'We report here the discovery, synthesis and screening results of a series of 3-(9H-fluoren-9-yl)pyrrolidine-2,5-dione derivatives as a novel class of potent inhibitors of Mycobacterium tuberculosis H37Rv strain as well as the enoyl acyl carrier protein reductase (ENR) InhA. Among them, several compounds displayed good activities against InhA which is one of the key enzymes involved in the type II fatty acid biosynthesis pathway of the mycobacteria cell wall. Furthermore, some exhibited promising activities against M. tuberculosis and multi-drug resistant M. tuberculosis strains.', '2013-10-02', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24140915');
INSERT INTO `artigos` (`id`, `componente`, `proteina`, `idExterno`, `titulo`, `abstract`, `data`, `link`) VALUES
(43, 37, 1, '24107081', 'Methyl-thiazoles: a novel mode of inhibition with the potential to develop novel inhibitors targeting InhA in Mycobacterium tuberculosis.', 'InhA is a well validated Mycobacterium tuberculosis (Mtb) target as evidenced by the clinical success of isoniazid. Translating enzyme inhibition to bacterial cidality by targeting the fatty acid substrate site of InhA remains a daunting challenge. The recent disclosure of a methyl-thiazole series demonstrates that bacterial cidality can be achieved with potent enzyme inhibition and appropriate physicochemical properties. In this study, we report the molecular mode of action of a lead methyl-thiazole, along with analogues with improved CYP inhibition profile. We have identified a novel mechanism of InhA inhibition characterized by a hitherto unreported "Y158-out" inhibitor-bound conformation of the protein that accommodates a neutrally charged "warhead". An additional novel hydrophilic interaction with protein residue M98 allows the incorporation of favorable physicochemical properties for cellular activity. Notably, the methyl-thiazole prefers the NADH-bound form of the enzyme with a Kd of ~13.7 nM, as against the NAD(+)-bound form of the enzyme.', '2013-10-25', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24107081'),
(44, 37, 1, '24106584', 'Survey of Potentially Inappropriate Prescription Using STOPP/START Criteria in Inha University Hospital.', 'Prescribing potentially harmful drugs and omitting essential drugs to older patients is a common problem because they take so many medications. In this study, our goal was to identify potentially inappropriate medications (PIMs) and potential prescribing omissions (PPOs) using Screening Tool of Older Persons'' potentially inappropriate Prescriptions (STOPP) and Screening Tool to Alert doctors to the Right Treatment (START) criteria to improve proper prescription and reduce improper prescription.', '2013-09-26', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24106584'),
(45, 37, 1, '24088266', 'Characterisation of the flexibility of substrate binding loop in the binding of direct InhA inhibitors.', 'The substrate binding loop (SBL) of inhA shows conformational changes on binding of direct inhA inhibitors (DIIs). The knowledge of conformational changes and its importance in binding of DII to inhA has not been explored before. This study initially focused on studying the conformational changes of SBL in selected inhA crystal structures. These conformational changes are measured as angle of rotation for SBL from the static hinge region, Ile194, in the crystal structures. The maximal angle difference of ∼41° was observed between most open and closed conformation of SBL. To gain insights into these conformational changes, comparative molecular dynamics simulations of inhA bound with a direct inhibitor (Genz10850) and apoprotein were performed. A considerable variation in the angle of rotation (∼24° to ∼12°) for the SBL which led to the closed conformation was observed during binding of Genz10850 with a consistent increase in electrostatic energy, whereas no change was observed in apoprotein. Hence, conformational changes in the SBL under the influence of inhibitor can be utilised as a parameter for enhanced binding inhibitor with inhA to screen the potent DIIs.', '2013-09-30', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=24088266'),
(46, 37, 1, '23889525', 'Discovery of new inhibitors of Mycobacterium tuberculosis InhA enzyme using virtual screening and a 3D-pharmacophore-based approach.', 'Mycobacterium tuberculosis InhA (MtInhA) is an attractive enzyme to drug discovery efforts due to its validation as an effective biological target for tuberculosis therapy. In this work, two different virtual-ligand-screening approaches were applied in order to identify new InhA inhibitors'' candidates from a library of ligands selected from the ZINC database. First, a 3-D pharmacophore model was built based on 36 available MtInhA crystal structures. By combining structure-based and ligand-based information, four pharmacophoric points were designed to select molecules able to satisfy the binding features of MtInhA substrate-binding cavity. The second approach consisted of using four well established docking programs, with different search algorithms, to compare the binding mode and score of the selected molecules from the aforementioned library. After detailed analyses of the results, six ligands were selected for in vitro analysis. Three of these molecules presented a satisfactory inhibitory activity with IC50 values ranging from 24 (±2) μM to 83 (±5) μM. The best compound presented an uncompetitive inhibition mode to NADH and 2-trans-dodecenoyl-CoA substrates, with Ki values of 24 (±3) μM and 20 (±2) μM, respectively. These molecules were not yet described as antituberculars or as InhA inhibitors, making its novelty interesting to start efforts on ligand optimization in order to identify new effective drugs against tuberculosis having InhA as a target. More studies are underway to dissect the discovered uncompetitive inhibitor interactions with MtInhA.', '2013-08-20', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=23889525'),
(47, 37, 1, '23539241', 'High-level resistance to isoniazid and ethionamide in multidrug-resistant Mycobacterium tuberculosis of the Lisboa family is associated with inhA double mutations.', 'The purpose of this study was to determine the levels of isoniazid and ethionamide resistance and to identify associated mutations in endemic multidrug-resistant (MDR) strains of Mycobacterium tuberculosis from the Lisbon metropolitan area, Portugal.', '2013-03-28', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=23539241'),
(48, 37, 1, '23464516', 'Discovery of novel InhA reductase inhibitors: application of pharmacophore- and shape-based screening approach.', 'InhA is a promising and attractive target in antimycobacterial drug development. InhA is involved in the reduction of long-chain trans-2-enoyl-ACP in the type II fatty acid biosynthesis pathway of Mycobacterium tuberculosis. Recent studies have demonstrated that InhA is one of the targets for the second line antitubercular drug ethionamide.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=23464516'),
(49, 37, 1, '23446452', 'GDF9 modulates the reproductive and tumor phenotype of female inha-null mice.', 'Intraovarian factors play important roles in coordinating germ cell and somatic cell growth in the ovary. Prior to the onset of gonadotropin stimulation and reproductive cyclicity, follicle development is dependent upon locally produced growth factors, such as the transforming growth factor beta family members inhibin, activin, and GDF9. In the absence of inhibin in prepubertal mice (Inha(-/-)), there are marked alterations in preantral follicle growth, but no evidence of ovarian tumors characteristic of adult Inha-null mice. To ascertain the contribution of GDF9 to the Inha-null phenotype, we analyzed folliculogenesis in postnatal Inha Gdf9 double knockout mice. Deletion of Gdf9 from Inha(-/-) rescues the initial growth defects found at early follicle stages in Inha(-/-) ovaries, but surprisingly enhances the onset of pretumor lesions. The normalization of growth dynamics between granulosa cells and oocytes of Inha Gdf9 double knockout mice is also accompanied by a reduction in levels of the activin/inhibin beta B subunit, Inhbb, which is upregulated in Inha(-/-) ovaries. However, at later ages, Inha Gdf9 double knockout ovaries are similar to Inha(-/-) ovaries, and show upregulation of the activin/inhibin subunits and downregulation of the growth factor, kit ligand, thus resulting in a local environment that is growth-promoting for granulosa cells but growth-inhibitory for oocytes. These data suggest a sequential mechanism of action initiated by GDF9 in the Inha knockout mouse that promotes defective folliculogenesis. These studies thus provide a novel role for GDF9 in causing reproductive defects and suppressing tumor initiation in the Inha(-/-) mouse model.', '2013-04-04', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=23446452'),
(50, 37, 1, '23350278', 'Docking studies on novel alkaloid tryptanthrin and its analogues against enoyl-acyl carrier protein reductase (InhA) of Mycobacterium tuberculosis.', 'Isoniazid resistance is a serious threat in the battle against the treatment of multi-drug resistant tuberculosis (MDR-TB) and extremely drug-resistant tuberculosis (XDR-TB). Isoniazid is an inhibitor of enoyl-acyl carrier protein reductase (InhA) of Mycobacterium tuberculosis, which is an important and functional enzyme of the type II fatty acid synthesis system and important therapeutic target. Natural alkaloid tryptanthrin and its analogues have shown anti-tubercular activity against MDR-TB, but their cellular target is unknown. In this work, in silico molecular docking was performed using docking server in order to see the interaction of tryptanthrin and its 15 analogues with InhA of M. tuberculosis. Results showed that among tryptanthrin and its 15 analogues, tryptanthrin and its two analogues exhibited good affinity to the binding site of InhA with free binding energy of -7.94 kcal/mol and inhibition constant (Ki) of 1.50 microm. Active site residues of InhA interacting with tryptanthrin were Ser13, Thr39, Phe41, Leu63, Asp64, Val65, Ile95, Phe97 and Ile122. In binding mode, polar bond were found between O1 (1) with Asp64 of bond length (3.34 A) and hydrophobic bonds were found between Leu63 with C15 and C12, Val65 with C7, Val65 with C12 and C4, Ile95 with C6 and C7, Ile95 with C10, C12 and C14. Important pi-pi bonds were found between Phe41 with C2, C5, C7, C12, C4, C6, C8, C9, C13 and Phe97 with C9. These interactions indicated stability of tryptanthrin in active residue and suggested it as a potential drug candidate. Thus, good affinity of tryptanthrin to binding site of InhA may lead to synthesis of anti-tubercular drug capable of combating MDR strains of M. tuberculosis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=23350278'),
(51, 37, 1, '23226944', 'Expression and cellular distribution of INHA and INHB before and after in vitro cultivation of porcine oocytes isolated from follicles of different size.', 'Cumulus-oocyte-complexes (COCs) were collected from small (<3 mm), medium (3-5 mm), and large (>5 mm) porcine follicles, and the INHA and INHB expression and cellular localization were studied. Developmentally competent (BCB+) COCs were cultured for 44 h. Samples of mRNA were isolated before and after in vitro maturation (IVM) from oocytes collected from follicles of different size for RQ-PCR assay. The INHA and INHB protein distribution within the oocytes was observed by confocal microscopy. INHA mRNA expression was increased in oocytes from large compared to medium and small follicles before IVM (P < 0.001), and to oocytes of small follicles after IVM (P < 0.001). The INHB expression was not different before IVM, but the IHNB mRNA level was gradually higher in oocytes from large follicles after IVM (P < 0.01). INHA was not differently expressed before IVM; however, in large follicle oocytes the protein was distributed in the peripheral area of the cytoplasm; in oocytes from small follicles it was in the entire cytoplasm. After IVM, INHA was strongly expressed in oocytes from small follicles and distributed particularly in the zona pellucida (ZP). Similarly and both before and after IVM, INHB protein was highly expressed in small follicle oocytes and within the cytoplasm. In summary, INHs can be recognized as a marker of porcine oocyte quality.', '2012-11-20', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=23226944'),
(52, 37, 1, '23146620', 'Can inhA mutation predict ethionamide resistance?', 'A tertiary care centre in Mumbai, India.', '2012-11-08', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=23146620'),
(53, 37, 1, '22751934', 'Cross-docking study on InhA inhibitors: a combination of Autodock Vina and PM6-DH2 simulations to retrieve bio-active conformations.', 'InhA, the NADH-dependent enoyl-acyl carrier protein reductase from Mycobacterium tuberculosis (Mtb) is the proposed main target of the first-line antituberculosis drug isoniazid (INH). INH activity is dependent on activation by the catalase peroxidase KatG, a Mtb enzyme whose mutations are linked to clinical resistance to INH. Other inhibitors of InhA that do not require any preliminary activation are known. The design of such direct potent inhibitors represents a promising approach to circumvent this resistance mechanism. An ensemble-docking process with four known InhA X-ray crystal structures and employing the Autodock Vina software was performed. Five InhA inhibitors whose bioactive conformations are known were sequentially docked in the substrate cavity of each protein. The efficiency of the docking was assessed and validated by comparing the calculated conformations to the crystallographic structures. For a same inhibitor, the docking results differed from one InhA conformation to another; however, docking poses that matched correctly or were very close to the expected bioactive conformations could be identified. The expected conformations were not systematically well ranked by the Autodock Vina scoring function. A post-docking optimization was carried out on all the docked conformations with the AMMP force field implemented on the VEGAZZ software, followed by a single point calculation of the interaction energy, using the MOPAC PM6-DH2 semi-empirical quantum chemistry method. The conformations were subsequently submitted to a PM6-DH2 optimization in partially flexible cavities. The resulting interaction energies combined with the multiple receptor conformations approach allowed us to retrieve the bioactive conformation of each ligand.', '2012-06-29', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=22751934'),
(54, 37, 1, '22727920', 'Rare case of multi-drug resistant endometrial tuberculosis unveiled by DNA signature studies of the rpoB, katG and inhA genes.', NULL, '2012-06-22', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=22727920'),
(55, 37, 1, '22515685', 'Exploring polymorphisms and their effects on reproductive traits of the INHA and INHβA genes in three goat breeds.', 'In this study, we report the analysis of INHA and INHβA gene polymorphisms in 786 goats of three breeds: Xinong Saanen (SN), Guanzhong (GZ) and Boer (BG). We identified three new allelic variants: P1-C80G and/126G (GenBank accession no. HQ202573) in the three goat breeds and P2-C936T (GenBank accession no. HQ202572) in SN and GZ goat breeds. At P1 locus, AA, AB and BB genotypes were found in the three goat breeds. At P2 locus, CC and CT genotypes were found in SN and GZ goat breeds. After comparing genotype distribution within the three goat breeds, BG had conspicuous differences from SN and GZ (P < 0.001) at P2 locus. The SNP locus was in Hardy-Weinberg disequilibrium at P1 locus in the three goat breeds (P < 0.05). At P2 locus, the SNP locus was in Hardy-Weinberg disequilibrium in SN and GZ goat breeds (P < 0.05). Association of polymorphisms with litter size was done at P1 locus in the three goat breeds. The result showed that AA genotype had remarkable litter size at P1 locus in the three goat breeds (P < 0.05). Therefore, these results suggest that INHA gene is a strong candidate gene that affects litter size in goats.', '2011-10-18', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=22515685'),
(56, 37, 1, '22483635', 'Chemical synthesis and biological evaluation of triazole derivatives as inhibitors of InhA and antituberculosis agents.', 'A series of triazoles have been prepared and evaluated as inhibitors of InhA as well as inhibitors of Mycobacterium tuberculosis H(37)R(v). Several of these new compounds possess a good activity against InhA, particularly compounds 17 and 18 for which molecular docking has been performed. Concerning their activities against M. tuberculosis H(37)R(V) strain, two of them, 3 and 12, were found to be good inhibitors with MIC values of 0.50 and 0.25 μg/mL, respectively. Particularly, compound 12 presenting the best MIC value of all compounds tested (0.6 μM) is totally inactive against InhA.', '2012-03-23', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=22483635'),
(57, 37, 1, '22452315', 'Bacillus anthracis protease InhA regulates BslA-mediated adhesion in human endothelial cells.', 'To achieve widespread dissemination in the host, Bacillus anthracis cells regulate their attachment to host endothelium during infection. Previous studies identified BslA (Bacillus anthracis S-layer Protein A), a virulence factor of B. anthracis, as necessary and sufficient for adhesion of vegetative cells to human endothelial cells. While some factors have been identified, bacteria-specific contributions to BslA mediated adhesion remain unclear. Using the attenuated vaccine Sterne 7702 strain of B. anthracis, we tested the hypothesis that InhA (immune inhibitor A), a B. anthracis protease, regulates BslA levels affecting the bacteria''s ability to bind to endothelium. To test this, a combination of inhA mutant and complementation analysis in adhesion and invasion assays, Western blot and InhA inhibitor assays were employed. Results show InhA downregulates BslA activity reducing B. anthracis adhesion and invasion in human brain endothelial cells. BslA protein levels in ΔinhA bacteria were significantly higher than wild-type and complemented strains showing InhA levels and BslA expression are inversely related. BslA was sensitive to purified InhA degradation in a concentration- and time-dependent manner. Taken together these data support the role of InhA regulation of BslA-mediated vegetative cell adhesion and invasion.', '2012-04-17', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=22452315'),
(58, 37, 1, '22369213', 'Effect of the explicit flexibility of the InhA enzyme from Mycobacterium tuberculosis in molecular docking simulations.', 'Protein/receptor explicit flexibility has recently become an important feature of molecular docking simulations. Taking the flexibility into account brings the docking simulation closer to the receptors'' real behaviour in its natural environment. Several approaches have been developed to address this problem. Among them, modelling the full flexibility as an ensemble of snapshots derived from a molecular dynamics simulation (MD) of the receptor has proved very promising. Despite its potential, however, only a few studies have employed this method to probe its effect in molecular docking simulations. We hereby use ensembles of snapshots obtained from three different MD simulations of the InhA enzyme from M. tuberculosis (Mtb), the wild-type (InhA_wt), InhA_I16T, and InhA_I21V mutants to model their explicit flexibility, and to systematically explore their effect in docking simulations with three different InhA inhibitors, namely, ethionamide (ETH), triclosan (TCL), and pentacyano(isoniazid)ferrate(II) (PIF).', '2011-12-22', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=22369213'),
(59, 37, 1, '22358357', 'Detection of genomic mutations in katG, inhA and rpoB genes of Mycobacterium tuberculosis isolates using polymerase chain reaction and multiplex allele-specific polymerase chain reaction.', 'Isoniazid (INH) and rifampin (RIF) are the most effective first line antibiotics against Mycobacterium tuberculosis. Mutations in several genes determine resistance of M. tuberculosis to INH, with the most common gene target of katG, and resistance to RIF is due to mutation in rpoB gene. The aim of present study was to assess the mutations in the regions related to RIF and INH resistance.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=22358357'),
(60, 37, 1, '22283812', 'Targeting InhA, the FASII enoyl-ACP reductase: SAR studies on novel inhibitor scaffolds.', 'The bacterial type II fatty acid biosynthesis (FASII) pathway is an essential but unexploited target for drug discovery. In this review we summarize SAR studies on inhibitors of InhA, the enoyl-ACP reductase from the FASII pathway in M. tuberculosis. Inhibitor scaffolds that are described include the diaryl ethers, pyrrolidine carboxamides, piperazine indoleformamides, pyrazoles, arylamides, fatty acids and imidazopiperidines, all of which form ternary complexes with InhA and the NAD cofactor, as well as isoniazid and the diazaborines which covalently modify the cofactor. Analysis of the structural data has enabled the development of a common binding mode for the ternary complex inhibitors, which includes a hydrogen bond network, a large hydrophobic pocket and a third '' size-limited'' binding area comprised of both polar and non-polar groups. A critical factor in InhA inhibition involves ordering of the substrate binding loop, located close to the active site, and a direct link is proposed between loop ordering and slow onset enzyme inhibition. Slow onset inhibitors have long residence times on the enzyme target, a property that is of critical importance for in vivo activity.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=22283812'),
(61, 37, 1, '22128068', 'Variation in sequences and mRNA expression levels of inhibin subunits α (INHA) and βA (INHBA) genes between prolific and nonprolific goat breeds.', NULL, '2011-11-29', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=22128068'),
(62, 37, 1, '21944473', 'Synthesis and biological activities of triazole derivatives as inhibitors of InhA and antituberculosis agents.', 'InhA, the enoyl reductase from the mycobacterial type II fatty acid biosynthesis pathway, is a target for the development of novel drugs against tuberculosis. We exploited copper-catalyzed [3+2] cycloaddition between alkynes and different azides to afford 1,4-disubstituted triazole or α-ketotriazole derivatives. Several compounds bearing a lipophilic chain mimicking the substrate were able to inhibit InhA. Among them, 1-dodecyl-4-phenethyl-1H-1,2,3-triazole displayed a minimum inhibitory concentration inferior to 2 μg/mL against Mycobacterium tuberculosis H37Rv.', '2011-09-16', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=21944473'),
(63, 37, 1, '21833828', 'Conformational changes in 2-trans-enoyl-ACP (CoA) reductase (InhA) from M. tuberculosis induced by an inorganic complex: a molecular dynamics simulation study.', 'InhA, the NADH-dependent 2-trans-enoyl-ACP reductase enzyme from Mycobacterium tuberculosis (MTB), is involved in the biosynthesis of mycolic acids, the hallmark of mycobacterial cell wall. InhA has been shown to be the primary target of isoniazid (INH), one of the oldest synthetic antitubercular drugs. INH is a prodrug which is biologically activated by the MTB catalase-peroxidase KatG enzyme. The activation reaction promotes the formation of an isonicotinyl-NAD adduct which inhibits the InhA enzyme, resulting in reduction of mycolic acid biosynthesis. As a result of rational drug design efforts to design alternative drugs capable of inhibiting MTB''s InhA, the inorganic complex pentacyano(isoniazid)ferrate(II) (PIF) was developed. PIF inhibited both wild-type and INH-resistant Ile21Val mutants of InhA and this inactivation did not require activation by KatG. Since no three-dimensional structure of the InhA-PIF complex is available to confirm the binding mode and to assess the molecular interactions with the protein active site residues, here we report the results of molecular dynamics simulations of PIF interaction with InhA. We found that PIF strongly interacts with InhA and that these interactions lead to macromolecular instabilities reflected in the long time necessary for simulation convergence. These instabilities were mainly due to perturbation of the substrate binding loop, particularly the partial denaturation of helices α6 and α7. We were also able to correlate the changes in the SASAs of Trp residues with the recent spectrofluorimetric investigation of the InhA-PIF complex and confirm their suggestion that the changes in fluorescence are due to InhA conformational changes upon PIF binding. The InhA-PIF association is very strong in the first 20.0 ns, but becomes very week at the end of the simulation, suggesting that the PIF binding mode we simulated may not reflect that of the actual InhA-PIF complex.', '2011-08-12', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=21833828'),
(64, 37, 1, '21628538', 'Novel inhibitors of InhA efficiently kill Mycobacterium tuberculosis under aerobic and anaerobic conditions.', 'Drug resistance in Mycobacterium tuberculosis has become a serious global health threat, which is now complicated by the emergence of extensively drug-resistant strains. New drugs that are active against drug-resistant tuberculosis (TB) are needed. We chose to search for new inhibitors of the enoyl-acyl carrier protein (ACP) reductase InhA, the target of the first-line TB drug isoniazid (also known as isonicotinoic acid hydrazide [INH]). A subset of a chemical library, composed of 300 compounds inhibiting Plasmodium falciparum enoyl reductase, was tested against M. tuberculosis. Four compounds were found to inhibit M. tuberculosis growth with MICs ranging from 1 μM to 10 μM. Testing of these compounds against M. tuberculosis in vitro revealed that only two compounds (CD39 and CD117) were bactericidal against drug-susceptible and drug-resistant M. tuberculosis. These two compounds were also bactericidal against M. tuberculosis incubated under anaerobic conditions. Furthermore, CD39 and CD117 exhibited increased bactericidal activity when used in combination with INH or rifampin, but CD39 was shown to be toxic to eukaryotic cells. The compounds inhibit InhA as well the fatty acid synthase type I, and CD117 was found to also inhibit tuberculostearic acid synthesis. This study provides the TB drug development community with two chemical scaffolds that are suitable for structure-activity relationship study to improve on their cytotoxicities and bactericidal activities in vitro and in vivo.', '2011-05-31', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=21628538'),
(65, 37, 1, '21437287', 'Bacillus anthracis protease InhA increases blood-brain barrier permeability and contributes to cerebral hemorrhages.', 'Hemorrhagic meningitis is a fatal complication of anthrax, but its pathogenesis remains poorly understood. The present study examined the role of B. anthracis-secreted metalloprotease InhA on monolayer integrity and permeability of human brain microvasculature endothelial cells (HBMECs) which constitute the blood-brain barrier (BBB). Treatment of HBMECs with purified InhA resulted in a time-dependent decrease in trans-endothelial electrical resistance (TEER) accompanied by zonula occluden-1 (ZO-1) degradation. An InhA-expressing B. subtilis exhibited increased permeability of HBMECs, which did not occur with the isogenic inhA deletion mutant (ΔinhA) of B. anthracis, compared with the corresponding wild-type strain. Mice intravenously administered with purified InhA or nanoparticles-conjugated to InhA demonstrated a time-dependent Evans Blue dye extravasation, leptomeningeal thickening, leukocyte infiltration, and brain parenchymal distribution of InhA indicating BBB leakage and cerebral hemorrhage. Mice challenged with vegetative bacteria of the ΔinhA strain of B. anthracis exhibited a significant decrease in leptomeningeal thickening compared to the wildtype strain. Cumulatively, these findings indicate that InhA contributes to BBB disruption associated with anthrax meningitis through proteolytic attack on the endothelial tight junctional protein zonula occluden (ZO)-1.', '2011-03-17', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=21437287'),
(66, 37, 1, '21333101', 'inhA promoter mutations: a gateway to extensively drug-resistant tuberculosis in South Africa?', 'Western Cape and Eastern Cape Provinces, South Africa.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=21333101'),
(67, 37, 1, '21143326', 'Phosphorylation of InhA inhibits mycolic acid biosynthesis and growth of Mycobacterium tuberculosis.', 'The remarkable survival ability of Mycobacterium tuberculosis in infected hosts is related to the presence of cell wall-associated mycolic acids. Despite their importance, the mechanisms that modulate expression of these lipids in response to environmental changes are unknown. Here we demonstrate that the enoyl-ACP reductase activity of InhA, an essential enzyme of the mycolic acid biosynthetic pathway and the primary target of the anti-tubercular drug isoniazid, is controlled via phosphorylation. Thr-266 is the unique kinase phosphoacceptor, both in vitro and in vivo. The physiological relevance of Thr-266 phosphorylation was demonstrated using inhA phosphoablative (T266A) or phosphomimetic (T266D/E) mutants. Enoyl reductase activity was severely impaired in the mimetic mutants in vitro, as a consequence of a reduced binding affinity to NADH. Importantly, introduction of inhA_T266D/E failed to complement growth and mycolic acid defects of an inhA-thermosensitive Mycobacterium smegmatis strain, in a similar manner to what is observed following isoniazid treatment. This study suggests that phosphorylation of InhA may represent an unusual mechanism that allows M. tuberculosis to regulate its mycolic acid content, thus offering a new approach to future anti-tuberculosis drug development.', '2010-11-09', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=21143326'),
(68, 37, 1, '20864541', 'Phosphorylation of enoyl-acyl carrier protein reductase InhA impacts mycobacterial growth and survival.', 'InhA, the primary target for the first line anti-tuberculosis drug isoniazid, is a key enzyme of the fatty-acid synthase II system involved in mycolic acid biosynthesis in Mycobacterium tuberculosis. In this study, we show that InhA is a substrate for mycobacterial serine/threonine protein kinases. Using a novel approach to validate phosphorylation of a substrate by multiple kinases in a surrogate host (Escherichia coli), we have demonstrated efficient phosphorylation of InhA by PknA, PknB, and PknH, and to a lower extent by PknF. Additionally, the sites targeted by PknA/PknB have been identified and shown to be predominantly located at the C terminus of InhA. Results demonstrate in vivo phosphorylation of InhA in mycobacteria and validate Thr-266 as one of the key sites of phosphorylation. Significantly, our studies reveal that the phosphorylation of InhA by kinases modulates its biochemical activity, with phosphorylation resulting in decreased enzymatic activity. Co-expression of kinase and InhA alters the growth dynamics of Mycobacterium smegmatis, suggesting that InhA phosphorylation in vivo is an important event in regulating its activity. An InhA-T266E mutant, which mimics constitutive phosphorylation, is unable to rescue an M. smegmatis conditional inhA gene replacement mutant, emphasizing the critical role of Thr-266 in mediating post-translational regulation of InhA activity. The involvement of various serine/threonine kinases in modulating the activity of a number of enzymes of the mycolic acid synthesis pathway, including InhA, accentuates the intricacies of mycobacterial signaling networks in parallel with the changing environment.', '2010-09-23', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=20864541'),
(69, 37, 1, '20696503', 'Development of isoniazid-NAD truncated adducts embedding a lipophilic fragment as potential bi-substrate InhA inhibitors and antimycobacterial agents.', 'Isoniazid-NAD truncated adducts embedding a lipophilic fragment were designed, synthesized and evaluated as inhibitors of the enoyl-acyl carrier protein (ACP) reductase (InhA) of Mycobacterium tuberculosis and as antimycobacterial agents. These compounds, planned as bi-substrate inhibitors and inspired from the active metabolite of isoniazid, combine both the nicotinamide moiety of the cofactor NAD and a lipophilic hydrocarbon chain mimic of the InhA substrate. The lipophilic fragment was introduced using either Suzuki-Miyaura cross-coupling or a classical nucleophilic substitution reaction. Several compounds developed in this work were indeed able to inhibit the InhA activity and showed promising antimycobacterial activities. However a direct correlation between the expressed activity and the bi-substrate mode of action could not yet be unambiguously demonstrated.', '2010-07-15', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=20696503'),
(70, 37, 1, '20491829', 'Detection of rpoB, katG and inhA gene mutations in Mycobacterium tuberculosis clinical isolates from Chongqing as determined by microarray.', 'The emergence of multidrug-resistance Mycobacterium tuberculosis is an increasing threat to tuberculosis control programmes. Susceptibility testing of Mycobacterium tuberculosis complex isolates by traditional methods requires a minimum of 14 days. This can be reduced significantly if molecular analysis is used. DNA sequencing is a good method for detecting mutation, but cannot be used routinely because of its relatively high cost. A sensitive and specific microarray has been designed to detect mutations in the rifampin resistance determining region of rpoB and loci in katG and inhA associated with isoniazid (INH) resistance. A panel of Mycobacterium tuberculosis isolates containing 13 different rpoB genotypes, two mutation genotypes within codon 315 of katG and one mutation genotypes at inhA was used to validate the microarray. The results obtained indicate that 100% of rifampicin-resistant M. tuberculosis strains isolated in Chongqing had rpoB mutations, with 531-Ser and 526-His being the most common positions substituted. Of the total 50 INH resistant isolates, 82% had a katG315 mutation and 18% had an inhA mutation. All the mutations detected by the microarray method were also confirmed by conventional DNA sequencing. It is demonstrated that the microarray is an efficient, specialized technique and can be used as a rapid method for detecting rifampin and isoniazid resistance.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=20491829'),
(71, 37, 1, '20408801', 'Recent progress in the identification and development of InhA direct inhibitors of Mycobacterium tuberculosis.', 'The InhA-related enoyl-ACP reductase, an enzyme involved in fatty acid synthesis, is one of the best validated targets for the development of anti-tubercular agents. However, the majority of isoniazid (INH)-resistant clinical strains are observed mainly due to the emergence of KatG mutants that do not form an INH-NAD adduct. Thus compounds that directly inhibit InhA avoiding activation by KatG would be promising candidates for combating MDR-TB. Herein, some predominant examples of InhA direct inhibitors recently developed are reviewed and special attention is paid to 3D-structures of InhA in drug design process.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=20408801'),
(72, 37, 1, '20200152', 'A slow, tight binding inhibitor of InhA, the enoyl-acyl carrier protein reductase from Mycobacterium tuberculosis.', 'InhA, the enoyl-ACP reductase in Mycobacterium tuberculosis is an attractive target for the development of novel drugs against tuberculosis, a disease that kills more than two million people each year. InhA is the target of the current first line drug isoniazid for the treatment of tuberculosis infections. Compounds that directly target InhA and do not require activation by the mycobacterial catalase-peroxidase KatG are promising candidates for treating infections caused by isoniazid-resistant strains. Previously we reported the synthesis of several diphenyl ethers with nanomolar affinity for InhA. However, these compounds are rapid reversible inhibitors of the enzyme, and based on the knowledge that long drug target residence times are an important factor for in vivo drug activity, we set out to generate a slow onset inhibitor of InhA using structure-based drug design. 2-(o-Tolyloxy)-5-hexylphenol (PT70) is a slow, tight binding inhibitor of InhA with a K(1) value of 22 pm. PT70 binds preferentially to the InhA x NAD(+) complex and has a residence time of 24 min on the target, which is 14,000 times longer than that of the rapid reversible inhibitor from which it is derived. The 1.8 A crystal structure of the ternary complex between InhA, NAD(+), and PT70 reveals the molecular details of enzyme-inhibitor recognition and supports the hypothesis that slow onset inhibition is coupled to ordering of an active site loop, which leads to the closure of the substrate-binding pocket.', '2010-03-03', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=20200152'),
(73, 37, 1, '20158608', 'Triclosan inhibition of mycobacterial InhA in Saccharomyces cerevisiae: yeast mitochondria as a novel platform for in vivo antimycolate assays.', 'To demonstrate the suitability of yeast to act as a novel biotechnological platform for conducting in vivo inhibition assays using drugs with low efficacies towards their mycobacterial targets, such as occurs in the situation with triclosan and InhA.', '2010-01-27', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=20158608'),
(74, 37, 1, '20028393', '3D-QSAR studies of arylcarboxamides with inhibitory activity on InhA using pharmacophore-based alignment.', 'Enoyl acyl carrier protein reductase (InhA) is a promising target for the development of antituberculosis drugs. The InhA-bound conformation of an indole-5-amide inhibitor (Genz 10850) (PDB code: IP44) was used to build a pharmacophore model by LigandScout. This model was then successfully used to identify the bioactive conformation and align 40 structurally diverse arylcarboxamide derivatives. Comparative molecular field analysis (CoMFA) and comparative molecular similarity indices analysis (CoMSIA) were performed on arylcarboxamides-based InhA inhibitors based on pharmacophore alignment. The best prediction was obtained with CoMSIA model combining steric and electrostatic fields (, r(2) = 0.972). The model was validated by an external test set, which gave a good predictive value (). Graphical interpretation of the results revealed important structural features of the zarylcarboxamides related to the active site of InhA. The results may be exploited for further design and virtual screening for some novel InhA inhibitors.', '2009-12-17', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=20028393'),
(75, 37, 1, '19893355', 'Detection of isoniazid and rifampicin resistance by sequencing of katG, inhA, and rpoB genes in Korea.', 'In Korea, tuberculosis is resistant to isoniazid (INH) and/or rifampicin (RIF) in more than 10% of cases. To prevent the spread of resistant Mycobacterium tuberculosis strains, it is crucial to develop more rapid resistance detection methods.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19893355'),
(76, 37, 1, '19837797', 'The InhA metalloproteases of Bacillus cereus contribute concomitantly to virulence.', 'The virulence of Bacillus cereus requires that bacteria have the capacity to colonize their host, degrade specific tissues, and circumvent the host immune system. To study this aspect of pathogenesis, we focused on three metalloproteases, InhA1, InhA2, and InhA3, which share more than 66% identity. The expression of these metalloprotease genes was assessed by transcriptional fusions with a lacZ reporter gene. The expression profiles suggest a complementary time course of InhA production. Indeed, the genes are simultaneously expressed but are oppositely controlled during stationary phase. We constructed single and multiple inhA mutants and assessed the bacterial locations of the proteins as well as their individual or additive roles in macrophage escape and toxicity, antibacterial-peptide cleavage, and virulence. InhA1, a major component of the spore exosporium, is the only InhA metalloprotease involved in bacterial escape from macrophages. A mutant lacking inhA1, inhA2, and inhA3 shows a strong decrease in the level of virulence for insects. Taken together, these results show that the InhA metalloproteases of B. cereus are important virulence factors that may allow the bacteria to counteract the host immune system.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19837797'),
(77, 37, 1, '19673965', 'A pyrosequencing method for molecular monitoring of regions in the inhA, ahpC and rpoB genes of Mycobacterium tuberculosis.', 'In this study, a pyrosequencing method for monitoring two genes related to isoniazid (INH)-resistance and a region of the rpoB gene linked to rifampin (RMP)-resistance in Mycobacterium tuberculosis was developed and evaluated. Specifically, a 20-base pair (bp) region of inhA (from -24 to -4), a 35-bp region of ahpC (from -39 to -4), and a 57-bp region of rpoB (from codon 515 to 533) were analysed by pyrosequencing. For the development of the method, selected non-consecutive clinical isolates of M. tuberculosis were analysed, including: 25 isolates susceptible to both INH and RMP, 18 RMP-monoresistant isolates, 17 INH-monoresistant isolates, and 15 multidrug-resistant strains. Our pyrosequencing methodology was further evaluated using 96 M. tuberculosis isolates. Mutations in ahpC were found to be associated with INH resistance (p <0.05). By setting any mutation in ahpC as a marker of resistance, the specificity and the positive predictive value (PPV) were 100%. Similarly, any mutation in the rpoB gene was associated with a RMP resistance phenotype (p <0.01). Using any mutation in rpoB as a marker of RMP resistance, the sensitivity of this assay was 73% and the specificity and PPV were 100%. The use of this pyrosequencing method to analyse the ahpC and rpoB genes allowed us to detect INH- and/or RMP-resistant isolates. Furthermore, this method represents an opportunity to expedite the description of novel mutations related to drug resistance.', '2009-08-07', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19673965'),
(78, 37, 1, '19604720', 'Rolling circle amplification and multiplex allele-specific PCR for rapid detection of katG and inhA gene mutations in Mycobacterium tuberculosis.', 'The aim of the study was to compare a novel, rolling circle amplification (RCA) assay for detection of common isoniazid (INH) resistance mutations in Mycobacterium tuberculosis with a multiplex allele-specific PCR (MAS-PCR) and sequencing of katG and the fabG1-inhA promoter region. One or more mutations were identified by RCA, MAS-PCR, and sequencing in 21 (68%), 22 (71%), and 23 (74%), respectively, of 31 epidemiologically unrelated INH-resistant isolates, and in none of 8 INH-susceptible isolates. The RCA assay is a rapid, inexpensive, and practical screening method for INH resistance in M. tuberculosis in countries with high prevalence of INH resistance.', '2009-07-14', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19604720'),
(79, 37, 1, '19602521', 'Characterization of inhibin alpha subunit (inha) in the zebrafish: evidence for a potential feedback loop between the pituitary and ovary.', 'Inhibin and activin are closely related disulphide-linked dimers that belong to the transforming growth factor beta superfamily. Although inhibin has been extensively studied in mammals, the information about its existence and function in lower vertebrates is very scarce. Using zebrafish as a model, the present study demonstrated that the inhibin-specific alpha subunit (inha) was predominantly expressed in the gonads and no transcript could be detected in other tissues including the pituitary and brain. In the ovary, the expression of inha was restricted to the somatic follicle cells surrounding the oocyte, together with the beta subunits (inhbaa and inhbb). This was further supported by the absence of its expression in the ovulated unfertilized eggs. During folliculogenesis, inha expression in the follicles slightly but steadily increased from primary growth to the mid-vitellogenic stage; however, its expression surged dramatically at the full-grown stage. Interestingly, the expression level of inha decreased significantly in the follicles whose oocytes were undergoing spontaneous maturation or germinal vesicle breakdown. When tested on cultured ovarian fragments, both goldfish pituitary extract and forskolin significantly stimulated inha expression. Further experiments showed that recombinant zebrafish FSH but not LH significantly increased inha expression in the same assay system. When tested in vitro, human inhibin A exhibited a slight but significant inhibitory effect on 17alpha, 20beta-dihydroxyprogesterone-induced oocyte maturation after 4 h incubation. The stimulation of inha expression by FSH and the potential inhibition of FSH by inhibin suggest a possible existence of a negative feedback loop between the pituitary and ovary in the zebrafish.', '2009-07-14', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19602521'),
(80, 37, 1, '19581462', 'Contribution of dfrA and inhA mutations to the detection of isoniazid-resistant Mycobacterium tuberculosis isolates.', 'Screening of 127 isoniazid (INH)-resistant Mycobacterium tuberculosis isolates from Singapore for mutations within the dfrA and inhA genes revealed mutations in 0 and 5 (3.9%) isolates respectively, implying that mutations in dfrA do not contribute to the detection of INH-resistant M. tuberculosis and that mutations within inhA are rare. Thirty-seven (29%) of the 127 isolates had no mutations in any of the genes implicated in INH resistance (katG, kasA, and ndh; inhA and ahpC promoters), suggesting that there are new INH targets yet to be discovered.', '2009-07-06', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19581462'),
(81, 37, 1, '19494698', 'Spore display using Bacillus thuringiensis exosporium protein InhA.', 'A new spore display method is presented that enables recombinant proteins to be displayed on the surface of Bacillus spores via fusion with InhA, an exosporium component of Bacillus thuringiensis. The green fluorescent protein and beta-galactosidase as model proteins were fused to the C-terminal region of InhA, respectively. The surface expression of the proteins on the spores was confirmed by flow cytometry, confocal laser scanning microscopy, measurement of the enzyme activity, and an immunogold electron microscopy analysis. InhA-mediated anchoring of foreign proteins in the exosporium of Bacillus spores can provide a new method of microbial display, thereby broadening the potential for novel applications of microbial display.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19494698'),
(82, 37, 1, '19429749', 'Activation of plasminogen activator inhibitor implicates protease InhA in the acute-phase response to Bacillus anthracis infection.', 'Anthrax is a zoonotic disease caused by Bacillus anthracis. The infection is associated with inflammation and sepsis, but little is known about the acute-phase response during disease and the nature of the bacterial factors causing it. In this study, we examined the levels of the acute-phase proteins (APPs) in comparative experiments using mice challenged with spores and a purified B. anthracis protease InhA as a possible factor mediating the response. A strong increase in the plasma levels of APPs such as haptoglobin and serum amyloid A was observed during infection. Protein and mRNA levels of plasminogen activator inhibitor (PAI)-1 in the liver were also increased concurrently with bacterial dissemination at 72 h post-infection. Similar effects were observed at 6 h post injection with InhA. Induction of hepatic transforming growth factor-beta1, a PAI-1 inducer, was also found in the liver of InhA-injected mice. PAI-1 elevation by InhA resulted in an increased level of urokinase-type plasminogen activator complex with PAI-1 and a decreased level of D-dimers indicating inhibition of blood fibrinolysis. These results reveal an acute liver response to anthrax infection and provide a plausible pathophysiological link between the host inflammatory response and the pro-thrombotic haemostatic imbalance in the course of disease through PAI-1 induction in the liver.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19429749'),
(83, 37, 1, '19428156', 'Discovery of potential new InhA direct inhibitors based on pharmacophore and 3D-QSAR analysis followed by in silico screening.', 'This study develops an efficient approach for discovering new InhA direct inhibitors in theory. The InhA-bound conformation of a pyrrolidine carboxamide inhibitor was used to build a pharmacophore model. This model with feature-shape query was successfully used to identify and align the bioactive conformations of pyrrolidine carboxamide analogues and screen SPECS database. A statistically valid 3D-QSAR with good results (r(2)(cv)=0.660 and r(2)=0.962) was obtained. From database screening, 30 hits were selected and identified as potential leads, which exhibit good estimated activities by 3D-QSAR model. Docking studies were carried out on two representative hits to analyze their interactions with InhA. Also, the interactions between existing pyrazole inhibitors and InhA were explored based on the pharmacophore model.', '2009-04-08', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19428156'),
(84, 37, 1, '19363042', 'A large-scale association study to assess the impact of known variants of the human INHA gene on premature ovarian failure.', 'Three variants of the human INHA gene have been reported to be associated with premature ovarian failure (POF) in case-control studies involving a small number of patients and controls. Since inhibin has a fundamental role in the control of ovarian function, it is important to establish the relevance of the reported variants for disease risk.', '2009-04-10', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19363042');
INSERT INTO `artigos` (`id`, `componente`, `proteina`, `idExterno`, `titulo`, `abstract`, `data`, `link`) VALUES
(85, 37, 1, '19228426', 'Correlations of mutations in katG, oxyR-ahpC and inhA genes and in vitro susceptibility in Mycobacterium tuberculosis clinical strains segregated by spoligotype families from tuberculosis prevalent countries in South America.', 'Mutations associated with resistance to rifampin or streptomycin have been reported for W/Beijing and Latin American Mediterranean (LAM) strain families of Mycobacterium tuberculosis. A few studies with limited sample sizes have separately evaluated mutations in katG, ahpC and inhA genes that are associated with isoniazid (INH) resistance. Increasing prevalence of INH resistance, especially in high tuberculosis (TB) prevalent countries is worsening the burden of TB control programs, since similar transmission rates are noted for INH susceptible and resistant M. tuberculosis strains.', '2009-02-19', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19228426'),
(86, 37, 1, '19172878', '[Distribution of mutations in the rpoB, katG, inhA, ahpC gene of rifampicin and isoniazid resistant M. tuberculosis strains isolated in Kyrgyz Republic].', 'The aim of this work was to evaluate the prevalence mutations in the rpoB, katG, inhA, ahpC gene of rifampicin and isoniazid resistant M. tuberculosis (Tb) isolates from Kyrgyz Republic using OA Biochip MDR. In the rifampicin-resistant strains, the mutations were identified in the codons 531, 526, 516, 511, 513, 512, 533, and 522. The most prevalent point mutations were Ser531RLeu at the codon 531 (59.7%). Resistance to INH was associated with mutations found in the katG gene (94.5%), inhA gene (3.5%), and ahpC gene (1.0%). The most prevalent mutations were SerRThr at the codon 315 (93.0%). The rifampicin and isoniasid resistance of the M. Tb strains isolated in Kyrgyzstan is associated mostly with Ser531RLeu mutation of the rpoB gene, Ser315RThr mutation of the katG gene, and InhT15 mutation.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19172878'),
(87, 37, 1, '19012578', 'Structure-based design of a novel class of potent inhibitors of InhA, the enoyl acyl carrier protein reductase from Mycobacterium tuberculosis: a computer modelling approach.', 'The NADH-dependent Enoyl-ACP reductase (InhA) of Mycobacterium tuberculosis has been shown to be the primary target of the frontline drug isoniazid (INH). However, INH must be first activated by katG gene, mutations in which have mediated resistance to INH. Recently, direct inhibitors of InhA have been reported. Using a structure-based approach, we have identified a tripeptide inhibitor with the sequence WYW, which is 100 times more potent than the existing inhibitors. It is therefore, a potential lead compound for the development of new anti-TB drugs.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=19012578'),
(88, 37, 1, '18955002', 'Binding of the tautomeric forms of isoniazid-NAD adducts to the active site of the Mycobacterium tuberculosis enoyl-ACP reductase (InhA): a theoretical approach.', 'The front-line antituberculosis drug isoniazid (INH) inhibits InhA, the NADH-dependent fatty acid biosynthesis enoyl ACP-reductase from Mycobacterium tuberculosis, via formation of covalent adducts with NAD (INH-NAD adducts). While ring tautomers were found the main species formed in solution, only the 4S chain INH-NAD tautomer was evidenced in the crystallized InhA:INH-NAD complex. In this study we attempted to explore the modes of interaction and energy binding of the different isomers placed in the active site of InhA with the help of various molecular modelling techniques. Ligand and enzyme models were generated with the help of the Vega ZZ program package. Resulting ligands were then docked into the InhA active site individually using computational automated docking package AUTODOCK 3.0.5. The more relevant docked conformations were then used to compute the interaction energy between the ligands and the InhA cavity. The AM1 Hamiltonian and the QM/MM ONIOM methodologies were used and the results compared. The various tautomers were found docked in almost the same place where INH-NAD was present as predicted by earlier X-ray crystallographic studies. However, some changes of ligand conformation and of the interactions ligand-protein were evidenced. The lower binding energy was observed for the 4S chain adduct that probably represents the effective active form of the INH-NAD adducts, as compared to the 4R epimer. The two 4S,7R and 4R,7S ring tautomers show intermediate and similar binding energies contrasting with their different experimental inhibitory potency on InhA. As a possible explanation based on calculated conformations, we formulated the hypothesis of an initial binding of the two ring tautomers to InhA followed by opening of only the ring hemiamidal 4S,7R tautomer (possibly catalyzed by Tyr158 phenolate basic group) to give the 4S chain INH-NAD tight-binding inhibitor. The predictions of ligand-protein interactions at the molecular level can be of primary importance in elucidating the mechanisms of action of isoniazid and InhA-related resistances, in identifying the effective mycobactericidal entities and, in further step, in the design of a new generation of antitubercular drugs.', '2008-09-17', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=18955002'),
(89, 37, 1, '18552191', 'Function of heterologous Mycobacterium tuberculosis InhA, a type 2 fatty acid synthase enzyme involved in extending C20 fatty acids to C60-to-C90 mycolic acids, during de novo lipoic acid synthesis in Saccharomyces cerevisiae.', 'We describe the physiological function of heterologously expressed Mycobacterium tuberculosis InhA during de novo lipoic acid synthesis in yeast (Saccharomyces cerevisiae) mitochondria. InhA, representing 2-trans-enoyl-acyl carrier protein reductase and the target for the front-line antituberculous drug isoniazid, is involved in the activity of dissociative type 2 fatty acid synthase (FASII) that extends associative type 1 fatty acid synthase (FASI)-derived C(20) fatty acids to form C(60)-to-C(90) mycolic acids. Mycolic acids are major constituents of the protective layer around the pathogen that contribute to virulence and resistance to certain antimicrobials. Unlike FASI, FASII is thought to be incapable of de novo biosynthesis of fatty acids. Here, the genes for InhA (Rv1484) and four similar proteins (Rv0927c, Rv3485c, Rv3530c, and Rv3559c) were expressed in S. cerevisiae etr1Delta cells lacking mitochondrial 2-trans-enoyl-thioester reductase activity. The phenotype of the yeast mutants includes the inability to produce sufficient levels of lipoic acid, form mitochondrial cytochromes, respire, or grow on nonfermentable carbon sources. Yeast etr1Delta cells expressing mitochondrial InhA were able to respire, grow on glycerol, and produce lipoic acid. Commensurate with a role in mitochondrial de novo fatty acid biosynthesis, InhA could accept in vivo much shorter acyl-thioesters (C(4) to C(8)) than was previously thought (>C(12)). Moreover, InhA functioned in the absence of AcpM or protein-protein interactions with its native FASII partners KasA, KasB, FabD, and FabH. None of the four proteins similar to InhA complemented the yeast mutant phenotype. We discuss the implications of our findings with reference to lipoic acid synthesis in M. tuberculosis and the potential use of yeast FASII mutants for investigating the physiological function of drug-targeted pathogen enzymes involved in fatty acid biosynthesis.', '2008-06-13', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=18552191'),
(90, 37, 1, '18457948', 'Synthesis and in vitro antimycobacterial activity of B-ring modified diaryl ether InhA inhibitors.', 'Previous structure-based design studies resulted in the discovery of alkyl substituted diphenyl ether inhibitors of InhA, the enoyl reductase from Mycobacterium tuberculosis. Compounds such as 5-hexyl-2-phenoxyphenol 19 are nM inhibitors of InhA and inhibit the growth of both sensitive and isoniazid-resistant strains of Mycobacterium tuberculosis with MIC(90) values of 1-2 microg/mL. However, despite their promising in vitro activity, these compounds have ClogP values of over 5. In efforts to reduce the lipophilicity of the compounds, and potentially enhance compound bioavailability, a series of B ring analogues of 19 were synthesized that contained either heterocylic nitrogen rings or phenyl rings having amino, nitro, amide, or piperazine functionalities. Compounds 3c, 3e, and 14a show comparable MIC(90) values to that of 19, but have improved ClogP values.', '2008-04-18', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=18457948'),
(91, 37, 1, '17978799', 'InhA-like protease secreted by Bacillus sp. S17110 inhabited in turban shell.', 'A strain producing a potent protease was isolated from turban shell. The strain was identified as Bacillus sp. S17110 based on phylogenetic analysis. The enzyme was purified from culture supernatant of Bacillus sp. S17110 to homogeneity by ammonium sulfate precipitation, SP-Sepharose, and DEAE-Sepharose anion exchange chromatography. Protease activity of the purified protein against casein was found to be stable at pH 7 to pH 10 and around 50 degrees . Approximately 70% of proteolytic activity of the enzyme was detected either in the presence of 100 mM SDS or Tween 20. The enzyme activity was enhanced in the presence of Ca2+, Zn2+, Mg2+, but was inhibited by EDTA, indicating that it requires metal for its activity. The purified enzyme was found to be a monomeric protein with a molecular mass of 75 kDa, as estimated by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) and gel filtration chromatography. The purified enzyme was analyzed through peptide fingerprint mass spectra generated from matrix-assisted laser desorption ionization-time-of-flight-mass spectrometry (MALDI-TOF-MS) and a BLAST search, and identified as immune inhibitor A (inhA) deduced from nucleotide sequence of B. cereus G9241. Since InhA was identified as protease that cleave antibacterial proteins found in insect, inhA-like protease purified from Bacillus sp. S17110 might be pathogenic to sea invertebrates.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17978799'),
(92, 37, 1, '17723305', 'Inhibition of the Mycobacterium tuberculosis enoyl acyl carrier protein reductase InhA by arylamides.', 'InhA, the enoyl acyl carrier protein reductase (ENR) from Mycobacterium tuberculosis, is one of the key enzymes involved in the type II fatty acid biosynthesis pathway of M. tuberculosis. We report here the discovery, through high-throughput screening, of a series of arylamides as a novel class of potent InhA inhibitors. These direct InhA inhibitors require no mycobacterial enzymatic activation and thus circumvent the resistance mechanism to antitubercular prodrugs such as INH and ETA that is most commonly observed in drug-resistant clinical isolates. The crystal structure of InhA complexed with one representative inhibitor reveals the binding mode of the inhibitor within the InhA active site. Further optimization through a microtiter synthesis strategy followed by in situ activity screening led to the discovery of a potent InhA inhibitor with in vitro IC(50)=90 nM, representing a 34-fold potency improvement over the lead compound.', '2007-08-15', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17723305'),
(93, 37, 1, '17681926', '[Hae RFLP of INHA and its relationship to goat litter size].', 'The polymorphism in a fragment within the coding region of the inhibin a subunit ( INHA ) gene was studied in 323 heads of Matou, Nubi, Boar and Haimen goats by PCR-SSCP, PCR-RFLP and sequencing. A new mutation G284A (Accession number: L28815) was identified, which could be detected by Hae digestion of a PCR product spanning this site. Hae PCR-RFLP analysis indicated that allele G was dominant. Association studies indicated that the effect of INHA genotypes on litter size was greatest for GG, followed by AG and AA genotypes. Thus, INHA may be a major gene controlling the prolificacy of goat, and allele G is positively correlated with litter size.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17681926'),
(94, 37, 1, '17664324', 'Targeting fatty acid biosynthesis for the development of novel chemotherapeutics against Mycobacterium tuberculosis: evaluation of A-ring-modified diphenyl ethers as high-affinity InhA inhibitors.', 'Structure-based design was used to develop a focused library of A-ring-modified diphenyl ether InhA inhibitors. From this library of analogs, two high-affinity alkyl-substituted diphenyl ethers, 6PP and 8PP, were selected for advanced study into their in vitro activity against Mycobacterium tuberculosis clinical isolates, their in vivo properties, and their signature response mode of action. 6PP and 8PP demonstrated enhanced activity against whole bacteria and showed activity in a rapid macrophage model of infection. In addition, transcriptional profiling revealed that the A-ring modifications of 6PP and 8PP increased the specificity of each analog for InhA. Both analogs had substantially longer half-lives in serum than did the parent compound, exhibited a fivefold reduction in cytotoxicity compared to the parent compound, and were well tolerated when administered orally at 300 mg/kg of body weight in animal models. Thus, the A-ring modifications increased the affinity and whole-cell specificity of the compounds for InhA and increased their bioavailability. The next step in optimization of the pharmacophore for preclinical evaluation is modification of the B ring to increase the bioavailability to that required for oral delivery.', '2007-07-30', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17664324'),
(95, 37, 1, '17600151', 'Probing mechanisms of resistance to the tuberculosis drug isoniazid: Conformational changes caused by inhibition of InhA, the enoyl reductase from Mycobacterium tuberculosis.', 'The frontline tuberculosis drug isoniazid (INH) inhibits InhA, the NADH-dependent fatty acid biosynthesis (FAS-II) enoyl reductase from Mycobacterium tuberculosis (MTB), via formation of a covalent adduct with NAD(+) (the INH-NAD adduct). Resistance to INH can be correlated with many mutations in MTB, some of which are localized in the InhA cofactor binding site. While the InhA mutations cause a substantial decrease in the affinity of InhA for NADH, surprisingly the same mutations result in only a small impact on binding of the INH-NAD adduct. Based on the knowledge that InhA interacts in vivo with other components of the FAS-II pathway, we have initiated experiments to determine whether enzyme inhibition results in structural changes that could affect protein-protein interactions involving InhA and how these ligand-induced conformational changes are modulated in the InhA mutants. Significantly, while NADH binding to wild-type InhA is hyperbolic, the InhA mutants bind the cofactor with positive cooperativity, suggesting that the mutations permit access to a second conformational state of the protein. While cross-linking studies indicate that enzyme inhibition causes dissociation of the InhA tetramer into dimers, analytical ultracentrifugation and size exclusion chromatography reveal that ligand binding causes a conformational change in the protein that prevents cross-linking across one of the dimer-dimer interfaces in the InhA tetramer. Interestingly, a similar ligand-induced conformational change is also observed for the InhA mutants, indicating that the mutations modulate communication between the subunits without affecting the two conformational states of the protein that are present.', '2007-06-28', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17600151'),
(96, 37, 1, '17560106', 'Synthesis of 4-phenoxybenzamide adenine dinucleotide as NAD analogue with inhibitory activity against enoyl-ACP reductase (InhA) of Mycobacterium tuberculosis.', 'The chemical synthesis of 4-phenoxybenzamide adenine dinucleotide (3), a NAD analogue which mimics isoniazid-NAD adduct and inhibits Mycobacterium tuberculosis NAD-dependent enoyl-ACP reductase (InhA), is reported. The 4-phenoxy benzamide riboside (1) has been prepared as a key intermediate, converted into its 5''-mononucleotide (2), and coupled with AMP imidazolide to give the desired NAD analogue 3. It inhibits InhA with IC50 = 27 microM.', '2007-05-31', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17560106'),
(97, 37, 1, '17539290', 'Multiple mutations in katG and inhA identified in Thai isoniazid-resistant Mycobacterium tuberculosis isolates.', 'A total of 29 Thai multi-drug-resistant/isoniazid-resistant Mycobacterium tuberculosis isolates were analyzed for mutations in katG from codons 254 to 549, inhA promoter and inhA open reading frame by DNA sequencing and single strand conformation polymorphism. Twenty-five multi-drug resistant isolates exhibited single point mutations (17 isolates at Ser315Thr plus Arg463Leu, 1 at Thr308Pro plus Arg463Leu, 7 at either Ser315Thr or Arg463Leu) while the other 4 isoniazid-resistant isolates had single point mutation only at Arg463Leu. Seven of 25 multi-drug-resistant isolates [4 at C(-15)T, 1 at T(-8)C; 1 at C(-15)T plus Ser94Ala and 1 at Ile21Val] and 2 of 4 isoniazid-resistant isolates [1 at C(-15)T, 1 at C (-15)T plus Ile21Thr] had mutations in inhA promoter and open reading frame, while the other 20 isolates had no mutation at any position. No frame shift mutation was observed in any tested isolates. This is the first report of two mutations, Trp308Pro of katG and T (-8)C of inhA in Mycobacterium tuberculosis isolates.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17539290'),
(98, 37, 1, '17510234', 'Effects of granulosa cell-specific deletion of Rb in Inha-alpha null female mice.', 'Our laboratory is interested in the gonadal growth regulatory properties of inhibins, members of the TGFbeta superfamily. We have previously shown that female mice lacking inhibins (Inha(-/-)) develop granulosa cell tumors and that concurrent loss of p27 accelerates tumor development. It has also been shown that the retinoblastoma protein RB regulates the G(1) to S phase transition of the cell cycle by controlling the activity of transcription factors and stabilizing the levels of the cell cycle inhibitor P27. Based on these data, we hypothesized that concurrent loss of Rb and inhibins in the ovary will exacerbate tumor formation. To test this hypothesis, we generated an ovarian granulosa cell conditional knockout (cKO) of Rb using the Cre/lox recombination system in the background of Inha(-/-) mice. Inha(-/-)/Rb cKO females show a modest increase in mortality rates compared with Inha(-/-) females. Although histologically similar to Inha(-/-) ovarian tumors, tumors from Inha(-/-)/Rb cKO females show increased number of mitotic figures and apoptotic rates. Interestingly, P27 levels are decreased in Inha(-/-)/Rb cKO ovarian tumors, likely due to the combined effect of Rb loss and increased Skp2 expression, which targets P27 to the proteosome. We propose that Rb loss may cause cell cycle delay or arrest, followed by apoptosis and that increases in p107 and p130 levels may compensate for Rb loss. These findings confirm the importance of P27 as a cell cycle regulator in granulosa cells and suggest functional compensation between RB-like proteins in ovarian tumorigenesis.', '2007-05-17', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17510234'),
(99, 37, 1, '17472376', 'Evidence from Raman spectroscopy that InhA, the mycobacterial enoyl reductase, modulates the conformation of the NADH cofactor to promote catalysis.', 'InhA, the enoyl reductase from Mycobacterium tuberculosis, catalyzes the NADH-dependent reduction of trans-2-enoyl-ACPs. In the present work, Raman spectroscopy has been used to identify catalytically relevant changes in the conformation of the nicotinamide ring that occur when NADH binds to InhA. For 4(S)-NADD, there is an 11 cm-1 decrease in the wavenumber of the C4-D stretching band (nuC-D) and a 50% decrease in the width of this band upon binding to InhA. While a similar reduction in line width is observed for the corresponding band arising from 4(R)-NADD, nuC-D for this isomer increases 34 cm-1 upon binding to InhA. These changes in nuC-D indicate that the nicotinamide ring adopts a bound conformation in which the 4(S)C-D bond is in a pseudoaxial orientation. Mutagenesis of F149, a conserved active site residue close to the cofactor, demonstrates that this enzyme-induced modulation in cofactor structure is directly linked to catalysis. In contrast to the wild-type enzyme, Raman spectra of NADD bound to F149A InhA resemble those of NADD in solution. Consequently, F149A is no longer able to optimally position the cofactor for hydride transfer, which correlates with the 30-fold decrease in kcat and 2-fold increase in D(V/KNADH) caused by this mutation. These studies thus substantiate the proposal that hydride transfer is promoted by pseudoaxial positioning of the NADH pro-4S bond, and indicate that catalysis of substrate reduction by InhA results, in part, from correct orientation of the cofactor in the ground state.', '2007-05-02', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17472376'),
(100, 37, 1, '17346194', 'Development of modern InhA inhibitors to combat drug resistant strains of Mycobacterium tuberculosis.', 'Strategies for the development of novel tuberculosis chemotherapeutics against existing drug resistant strains involve the identification and inhibition of novel drug targets as well as the design and synthesis of compounds against historical targets. InhA, the enoyl reductase from the mycobacterial type II fatty acid biosynthesis pathway, is a target of the frontline chemotherapeutic, isoniazid (INH). Importantly, the majority of INH-resistant clinical isolates arise from mutations in KatG, the enzyme responsible for activating isoniazid, into its active form. Thus compounds that inhibit InhA without first requiring KatG activation will be active against the majority of INH resistant strains of Mycobacterium tuberculosis. This review describes the role of InhA in cell wall biosynthesis and recent progress in the development of novel diphenyl ether-based InhA inhibitors that have activity against both sensitive and drug resistant strains of M. tuberculosis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17346194'),
(101, 37, 1, '17163639', 'High affinity InhA inhibitors with activity against drug-resistant strains of Mycobacterium tuberculosis.', 'Novel chemotherapeutics for treating multidrug-resistant (MDR) strains of Mycobacterium tuberculosis (MTB) are required to combat the spread of tuberculosis, a disease that kills more than 2 million people annually. Using structure-based drug design, we have developed a series of alkyl diphenyl ethers that are uncompetitive inhibitors of InhA, the enoyl reductase enzyme in the MTB fatty acid biosynthesis pathway. The most potent compound has a Ki'' value of 1 nM for InhA and MIC99 values of 2-3 microg mL(-1) (6-10 microM) for both drug-sensitive and drug-resistant strains of MTB. Overexpression of InhA in MTB results in a 9-12-fold increase in MIC99, consistent with the belief that these compounds target InhA within the cell. In addition, transcriptional response studies reveal that the alkyl diphenyl ethers fail to upregulate a putative efflux pump and aromatic dioxygenase, detoxification mechanisms that are triggered by the lead compound triclosan. These diphenyl ether-based InhA inhibitors do not require activation by the mycobacterial KatG enzyme, thereby circumventing the normal mechanism of resistance to the front line drug isoniazid (INH) and thus accounting for their activity against INH-resistant strains of MTB.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=17163639'),
(102, 37, 1, '16906155', 'Transfer of a point mutation in Mycobacterium tuberculosis inhA resolves the target of isoniazid.', 'Isoniazid is one of the most effective antituberculosis drugs, yet its precise mechanism of action is still controversial. Using specialized linkage transduction, a single point mutation allele (S94A) within the putative target gene inhA was transferred in Mycobacterium tuberculosis. The inhA(S94A) allele was sufficient to confer clinically relevant levels of resistance to isoniazid killing and inhibition of mycolic acid biosynthesis. This resistance correlated with the decreased binding of the INH-NAD inhibitor to InhA, as shown by enzymatic and X-ray crystallographic analyses, and establishes InhA as the primary target of isoniazid action in M. tuberculosis.', '2006-08-13', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=16906155'),
(103, 37, 1, '16723387', 'Expansion of the germline analysis for the INHA gene in Indian women with ovarian failure.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=16723387'),
(104, 37, 1, '16537011', 'Detection of the mutations in katG 315 and inhA -15 of Mycobacterium tuberculosis strains isolated from Chinese patients.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=16537011'),
(105, 37, 1, '16495272', 'Molecular characterization of isoniazid resistance in Mycobacterium tuberculosis: identification of a novel mutation in inhA.', 'Multiplex allele-specific PCRs detecting katG codon 315 and mabA (bp -15) mutations could specifically identify 77.5% of isoniazid-resistant Mycobacterium tuberculosis strains in the South China region. One clinical isolate harboring InhA Ile194Thr was characterized to show strong association with isoniazid resistance in Mycobacterium tuberculosis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=16495272'),
(106, 37, 1, '16390856', 'INHA promoter polymorphisms are associated with premature ovarian failure.', 'Inhibin is an important glycoprotein that is involved in folliculogenesis. INHA, the gene encoding the inhibin alpha subunit, was recently proposed as a candidate for premature ovarian failure (POF), a syndrome that leads to the cessation of ovarian function under the age of 40 years. 70 POF patients and 70 controls were screened for the previously identified INHA -16C>T transition mutation. The T allele was found in 31/70 (44.3%) of controls, but only 18/70 (25.7%) of POF patients. This result indicates that the T allele is significantly underrepresented in the POF patient population (Fisher''s exact test, two-tail: P = 0.033). Sequence analysis of the INHA promoter in 50 POF patients and 50 controls identified a highly polymorphic imperfect TG repeat at approximately -300 bp, that consisted of four common haplotypes (A, B, C and D). The -16T allele is linked to the shortest repeat haplotype (haplotype C). Despite the association between haplotype C and POF, no significant difference was found between the promoter activity of a luciferase reporter construct containing haplotype C, and most of the other haplotypes tested. Interestingly, haplotype B failed to show any promoter activity. We conclude that the inheritance of specific INHA promoter haplotypes predispose to the development of premature ovarian failure.', '2006-01-03', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=16390856'),
(107, 37, 1, '16209020', '[The characteristics of the sensitivity of Mycobacterium tuberculosis to rifampicin and isoniazid through determination of mutations in the genes rpoB, katG, inhA, oxyR, and kasA by different molecular biological assays].', 'Two hundred and two patients with different forms of pulmonary tuberculosis were examined to study the characteristics of sensitivity with the signs of multidrug resistance to rifampicin and isoniazid, by using a microbiological assay of the absolute concentrations and determining mutations in the genes rpoB, katG, inhA, oxyR, and kasA, by employing different molecular biological assays. Mycobacterium tuberculosis (MBT) DNA was isolated from both a diagnostic material (such as sputum, bronchial secretion), and clinical MBT isolates. By showing a higher sensitivity and a higher specificity, as cultural techniques, molecular biological assays of MBT drug sensitivity in patients with tuberculosis were ascertained to accelerate its diagnosis until the patient was admitted to a clinic.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=16209020'),
(108, 37, 1, '15908576', 'Molecular dynamics simulation studies of the wild-type, I21V, and I16T mutants of isoniazid-resistant Mycobacterium tuberculosis enoyl reductase (InhA) in complex with NADH: toward the understanding of NADH-InhA different affinities.', 'The increasing prevalence of tuberculosis in many areas of the world, associated with the rise in drug-resistant Mycobacterium tuberculosis (MTB) strains, presents a major threat to global health. InhA, the enoyl-ACP reductase from MTB, catalyzes the nicotinamide adenine dinucleotide (NADH)-dependent reduction of long-chain trans-2-enoyl-ACP fatty acids, an intermediate in mycolic acid biosynthesis. Mutations in the structural gene for InhA are associated with isoniazid resistance in vivo due to a reduced affinity for NADH, suggesting that the mechanism of drug resistance may be related to specific interactions between enzyme and cofactor within the NADH binding site. To compare the molecular events underlying ligand affinity in the wild-type, I21V, and I16T mutant enzymes and to identify the molecular aspects related to resistance, molecular dynamics simulations of fully solvated NADH-InhA (wild-type and mutants) were performed. Although very flexible, in the wild-type InhA-NADH complex, the NADH molecule keeps its extended conformation firmly bound to the enzyme''s binding site. In the mutant complexes, the NADH pyrophosphate moiety undergoes considerable conformational changes, reducing its interactions with its binding site and probably indicating the initial phase of ligand expulsion from the cavity. This study should contribute to our understanding of specific molecular mechanisms of drug resistance, which is central to the design of more potent antimycobacterial agents for controlling tuberculosis.', '2005-05-20', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=15908576'),
(109, 37, 1, '15854436', '[Characterization of the katG, inhA, ahpC, kasA, and oxyR gene mutations in isoniazid-resistant and susceptible strain of Mycobacterium tuberculosis by automated DNA sequencing].', 'To study the characteristics of katG, inhA, ahpC, kasA, and oxyR gene mutations in isoniazid-resistant clinical isolates of Mycobacterium tuberculosis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=15854436'),
(110, 37, 1, '15703806', '1H and 13C NMR characterization of pyridinium-type isoniazid-NAD adducts as possible inhibitors of InhA reductase of Mycobacterium tuberculosis.', 'Oxidative activation of the antituberculous drug isoniazid (INH) in the presence of the NADH cofactor gives a pool of INH-NAD adducts proposed to be involved in the mechanism of action of this drug through inhibition of the reductase InhA. Among these adducts and besides dihydropyridine derivatives, two pyridinium-type isoniazid-NAD adducts were shown to be formed in solution and have been fully characterized by 1H/13C NMR and MS. One of them results from the oxidation of dihydropyridine-type INH-NAD adducts. The spectral data strongly support its existence under two epimeric structures. These epimers arise from a cyclization process between the carboxamide group and the ketone function with the creation of a new chiral center at C-7. The second pyridinium-type adduct was formed in acidic solution by dehydration of the cyclized dihydropyridine-type INH-NAD adducts and also exists as a cyclized structure. Both of these pyridinium-type compounds were inactive as inhibitors of InhA activity and can be considered as deactivated species.', '2005-01-21', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=15703806'),
(111, 37, 1, '15121773', 'Inhibin alpha-subunit (INHA) gene and locus changes in paediatric adrenocortical tumours from TP53 R337H mutation heterozygote carriers.', 'The R337H TP53 mutation is a low-penetrance molecular defect that predisposes to adrenocortical tumour (ACT) formation in Brazilian and possibly other populations. Additional genetic defects may be responsible for the variable expression of ACTs in these cases. The inhibin alpha-subunit gene (INHA) on 2q33-qter has been implicated in mouse adrenocortical tumourigenesis. We studied 46 pediatric patients with ACTs from Brazil for INHA genetic alterations; 39 of these patients were heterozygous carriers of the R337H TP53 mutation. We first mapped the INHA gene by radiation hybrid analysis and determined 10 linked microsatellite markers in an area flanked by D2S1371 and D2S206 on 2q33-qter. These markers were then used for loss of heterozygozity (LOH) studies in nine paired germline and tumour DNA samples. Mapping placed the INHA gene in close proximity to D2S2848 (SHGC11864) with a log of odds (LOD) score of 5.84. LOH for at least one marker in the region was identified in 8/9 tumours (89%). Six patients were heterozygous for three INHA mutations: one in exon 1, 127C>G, and two in exon 2, 3998G>A and 4088G>A, all leading to amino acid substitutions (P43A, G227R, and A257T, respectively). A257T is located in a conserved INHA region, highly homologous to transforming growth factor-beta; both G227R and A257T change polarity, and, in addition, G227R changes the pH. We conclude that these sequence alterations and the detected 2q allelic changes suggest that INHA may be one of the contributing factors needed for ACT formation in pediatric patient carriers of the R337H TP53 mutation.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=15121773'),
(112, 37, 1, '14638486', 'ethA, inhA, and katG loci of ethionamide-resistant clinical Mycobacterium tuberculosis isolates.', 'Ethionamide (ETH) is a structural analog of the antituberculosis drug isoniazid (INH). Both of these drugs target InhA, an enzyme involved in mycolic acid biosynthesis. INH requires catalase-peroxidase (KatG) activation, and mutations in katG are a major INH resistance mechanism. Recently an enzyme (EthA) capable of activating ETH has been identified. We sequenced the entire ethA structural gene of 41 ETH-resistant Mycobacterium tuberculosis isolates. We also sequenced two regions of inhA and all or part of katG. The MICs of ETH and INH were determined in order to associate the mutations identified with a resistance phenotype. Fifteen isolates were found to possess ethA mutations, for all of which the ETH MICs were > or =50 microg/ml. The ethA mutations were all different, previously unreported, and distributed throughout the gene. In eight of the isolates, a missense mutation in the inhA structural gene occurred. The ETH MICs for seven of the InhA mutants were > or =100 microg/ml, and these isolates were also resistant to > or =8 microg of INH per ml. Only a single point mutation in the inhA promoter was identified in 14 isolates. A katG mutation occurred in 15 isolates, for which the INH MICs for all but 1 were > or =32 microg/ml. As expected, we found no association between katG mutation and the level of ETH resistance. Mutations within the ethA and inhA structural genes were associated with relatively high levels of ETH resistance. Approximately 76% of isolates resistant to > or =50 microg of ETH per ml had such mutations.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=14638486'),
(113, 37, 1, '14623976', 'The isoniazid-NAD adduct is a slow, tight-binding inhibitor of InhA, the Mycobacterium tuberculosis enoyl reductase: adduct affinity and drug resistance.', 'Isoniazid (INH), a frontline antitubercular drug, inhibits InhA, the enoyl reductase from Mycobacterium tuberculosis, by forming a covalent adduct with the NAD cofactor. Here, we report that the INH-NAD adduct is a slow, tight-binding competitive inhibitor of InhA. Demonstration that the adduct binds to WT InhA by a two-step enzyme inhibition mechanism, with initial, weak binding (K(-1) = 16 +/- 11 nM) followed by slow conversion to a final inhibited complex (EI*) with overall Ki = 0.75 +/- 0.08 nM, reconciles existing contradictory values for the inhibitory potency of INH-NAD for InhA. The first order rate constant for conversion of the initial EI complex to EI* (k2 = 0.13 +/- 0.01 min(-1)) is similar to the maximum rate constant observed for InhA inhibition in reaction mixtures containing InhA, INH, NADH, and the INH-activating enzyme KatG (catalase/peroxidase from M. tuberculosis), consistent with an inhibition mechanism in which the adduct forms in solution rather than on the enzyme. Importantly, three mutations that correlate with INH resistance, I21V, I47T, and S94A, have little impact on the inhibition constants. Thus, drug resistance does not result simply from a reduction in affinity of INH-NAD for pure InhA. Instead, we hypothesize that protein-protein interactions within the FASII complex are critical to the mechanism of INH action. Finally, for M161V, an InhA mutation that correlates with resistance to the common biocide triclosan in Mycobacterium smegmatis, binding to form the initial EI complex is significantly weakened, explaining why this mutant inactivates more slowly than WT InhA when incubated with INH, NADH, and KatG.', '2003-11-17', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=14623976'),
(114, 37, 1, '12958298', 'Mutations in katG, inhA, and ahpC genes of Brazilian isoniazid-resistant isolates of Mycobacterium tuberculosis.', 'The presence of mutations in specific regions of the katG, inhA, and ahpC genes was analyzed with 69 Mycobacterium tuberculosis isoniazid-resistant isolates from three Brazilian states. Point mutations in codon 315 of the katG gene were observed in 87.1, 60.9, and 60% of the isolates from Rio Grande do Sul, Rio de Janeiro, and São Paulo, respectively. Mutations in the inhA gene were identified only in one isolate from RJ State, and the ahpC promoter region revealed mutations in distinct positions in 12.9, 21.7, and 6.7% of the isolates from RS, RJ and SP, respectively.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=12958298'),
(115, 37, 1, '12740851', '1H and 13C NMR characterization of hemiamidal isoniazid-NAD(H) adducts as possible inhibitors of InhA reductase of Mycobacterium tuberculosis.', 'Isoniazid (INH) is easily oxidized with manganese(III) pyrophosphate, a chemical model of the KatG protein involved in activation of INH inside the bacteria Mycobacterium tuberculosis. Performed in the presence of NAD(+), this oxidation generates a family of isomeric INH-NAD(H) adducts, which have been shown to be effective inhibitors of InhA, an enzyme essential in mycolic acid biosynthesis. In this work, we fully characterized by (1)H and (13)C NMR spectroscopy four main species of INH-NAD(H) adducts that coexist in solution. Two of them are open diastereoisomers consisting of the covalent attachment of the isonicotinoyl radical at position four of the nicotinamide coenzyme. The other two result from a cyclization involving the amide group from the nicotinamide and the carbonyl group from the isonicotinoyl radical to give diastereoisomeric hemiamidals. Although an INH-NAD(H) adduct with a 4S configuration has been characterized within the active site of InhA from Xray crystallography and this bound adduct interpreted as an open form (Rozwarski et al., Science 1998, 279, 98-102), it is legitimate to raise the question about the effective active form(s), open or cyclic, of INH-NAD(H) adduct(s). Is there a single active form or are several forms able to inhibit the InhA activity with different levels of inhibitory potency?', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=12740851'),
(116, 37, 1, '12654922', 'Inhibition of InhA activity, but not KasA activity, induces formation of a KasA-containing complex in mycobacteria.', 'Isoniazid (INH) remains one of the key drugs used to control tuberculosis, with the enoyl-AcpM reductase InhA being the primary target. However, based on the observation that INH-treated Mycobacterium tuberculosis overproduces KasA, an enzyme involved in the biosynthesis of mycolic acids, and induces the formation of a covalent complex consisting of AcpM, KasA, and INH, it has been proposed that KasA represents the primary target of INH. However, the relevance of this complex to INH action remains obscure. This study was aimed at clarifying the role of InhA and KasA in relation to INH activity. By using anti-KasA antibodies we detected the KasA-containing complex in INH-treated Mycobacterium smegmatis. In addition, INH-treated cells also produced constant levels of KasA that were not sequestered in the complex and presumably were sufficient to ensure mycolic acid biosynthesis. Interestingly, a furA-lacking strain induced the complex at lower concentrations of INH compared with the control strain, whereas higher INH concentrations were necessary to induce the complex in a strain that lacks katG, suggesting that INH needs to be activated by KatG to induce the KasA-containing complex. The InhA inhibitors ethionamide and diazaborine also induced the complex; thus, its formation was not specifically relevant to INH action but was because of InhA inhibition. In addition, in vitro assays using purified InhA and KasA demonstrated that KatG-activated INH, triclosan, and diazaborine inhibited InhA but not KasA activity. Moreover, several thermosensitive InhA mutant strains of M. smegmatis constitutively expressed the KasA-containing complex. This study provides the biochemical and genetic evidence. 1) Only inhibition of InhA, but not KasA, induces the KasA-containing complex. 2) INH is not part of the complex. 3) INH does not target KasA, consistent with InhA being the primary target of INH.', '2003-03-24', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=12654922'),
(117, 37, 1, '12543676', 'Isoniazid activation defects in recombinant Mycobacterium tuberculosis catalase-peroxidase (KatG) mutants evident in InhA inhibitor production.', 'Mycobacterium tuberculosis KatG catalyzes the activation of the antitubercular agent isoniazid to yield an inhibitor targeting enoyl reductase (InhA). However, no firm biochemical link between many KatG variants and isoniazid resistance has been established. In the present study, six distinct KatG variants identified in clinical Mycobacterium tuberculosis isolates resistant to isoniazid were generated by site-directed mutagenesis, and the recombinant mutant proteins (KatG(A110V), KatG(A139P), KatG(S315N), KatG(L619P), KatG(L634F), and KatG(D735A)) were purified and characterized with respect to their catalase-peroxidase activities (in terms of k(cat)/K(m)), rates of free-radical formation from isoniazid oxidation, and, moreover, abilities to activate isoniazid. The A110V amino acid replacement did not result in significant alteration of KatG activities except that the peroxidase activity was enhanced. The other mutations, however, resulted in modestly reduced catalase and peroxidase catalytic efficiencies and, for the four mutants tested, significantly lower activities to oxidize isoniazid. Compared to the wild-type enzyme, the ability of the KatG(L634F), KatG(A139P), and KatG(D735A) variants to activate isoniazid decreased by 36%, 76%, and 73%, respectively, whereas the KatG(S315N) and KatG(L619P) variants completely lost their abilities to convert isoniazid into the InhA inhibitor. In addition, the inclusion of exogenous Mn(2+) to the isoniazid activation reaction mix significantly improved the ability of wild-type and KatG mutants to produce the InhA inhibitor.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=12543676'),
(118, 37, 1, '12406221', 'Overexpression of inhA, but not kasA, confers resistance to isoniazid and ethionamide in Mycobacterium smegmatis, M. bovis BCG and M. tuberculosis.', 'The inhA and kasA genes of Mycobacterium tuberculosis have each been proposed to encode the primary target of the antibiotic isoniazid (INH). Previous studies investigating whether overexpressed inhA or kasA could confer resistance to INH yielded disparate results. In this work, multicopy plasmids expressing either inhA or kasA genes were transformed into M. smegmatis, M. bovis BCG and three different M. tuberculosis strains. The resulting transformants, as well as previously published M. tuberculosis strains with multicopy inhA or kasAB plasmids, were tested for their resistance to INH, ethionamide (ETH) or thiolactomycin (TLM). Mycobacteria containing inhA plasmids uniformly exhibited 20-fold or greater increased resistance to INH and 10-fold or greater increased resistance to ETH. In contrast, the kasA plasmid conferred no increased resistance to INH or ETH in any of the five strains, but it did confer resistance to thiolactomycin, a known KasA inhibitor. INH is known to increase the expression of kasA in INH-susceptible M. tuberculosis strains. Using molecular beacons, quantified inhA and kasA mRNA levels showed that increased inhA mRNA levels corre--lated with INH resistance, whereas kasA mRNA levels did not. In summary, analysis of strains harbouring inhA or kasA plasmids yielded the same conclusion: overexpressed inhA, but not kasA, confers INH and ETH resistance to M. smegmatis, M. bovis BCG and M. tuberculosis. Therefore, InhA is the primary target of action of INH and ETH in all three species.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=12406221'),
(119, 37, 1, '12069966', 'Mn(III) pyrophosphate as an efficient tool for studying the mode of action of isoniazid on the InhA protein of Mycobacterium tuberculosis.', 'The antituberculosis drug isoniazid (INH) is quickly oxidized by stoichiometric amounts of manganese(III) pyrophosphate. In the presence of nicotinamide coenzymes (NAD+, NADH, nicotinamide mononucleotide [NMN+]) and nicotinic acid adenine dinucleotide (DNAD+), INH oxidation produced the formation of INH-coenzyme adducts in addition to known biologically inactive products (isonicotinic acid, isonicotinamide, and isonicotinaldehyde). A pool of INH-NAD(H) adducts preformed in solution allowed the rapid and strong inhibition of in vitro activity of the enoyl-acyl carrier protein reductase InhA, an INH target in the biosynthetic pathway of mycolic acids: the inhibition was 90 or 60% when the adducts were formed in the presence of NAD+ or NADH, respectively. Under similar conditions, no inhibitory activity of INH-NMN(H) and INH-DNAD(H) adducts was detected. When an isolated pool of 100 nM INH-NAD(H) adducts was first incubated with InhA, the enzyme activity was inhibited by 80%; when present in excess, both NADH and decenoyl-coenzyme A are able to prevent this phenomenon. InhA inhibition by several types of INH-coenzyme adducts coexisting in solution is discussed in relation with the structure of the coenzyme, the stereochemistry of the adducts, and their existence as both open and cyclic forms. Thus, manganese(III) pyrophosphate appears to be an efficient and convenient alternative oxidant to mimic the activity of the Mycobacterium tuberculosis KatG catalase-peroxidase and will be useful for further mechanistic studies of INH activation and for structural investigations of reactive INH species in order to promote the design of new inhibitors of InhA as potential antituberculous drugs.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=12069966'),
(120, 37, 1, '12030943', 'SNPs in the porcine INHA gene and linkage mapping to SSC15.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=12030943'),
(121, 37, 1, '11802940', '[Sequencing of inhA in drug-resistant Mycobacterium tuberculosis isolates].', 'To determine and characterize inhA mutations of drug-resistant isolates of Mycobacterium tuberculosis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=11802940'),
(122, 37, 1, '11429458', 'Identification of genes involved in the activation of the Bacillus thuringiensis inhA metalloprotease gene at the onset of sporulation.', 'The immune inhibitor A (InhA) metalloprotease from Bacillus thuringiensis specifically cleaves antibacterial proteins produced by the insect host, suggesting that it may contribute to the overall virulence of B. thuringiensis. The transcriptional regulation of the inhA gene in both B. thuringiensis and Bacillus subtilis was investigated. Using a transcriptional inhA''-lacZ fusion, it was shown that inhA expression is activated at the onset of sporulation. However, the transcriptional start site of inhA is similar to sigma(A)-dependent promoters, and deletion of the sporulation-specific sigma factors sigma(F) or sigma(E) had no effect on inhA expression in B. subtilis. The DNA region upstream from inhA contains two genes encoding polypeptides similar to the SinI and SinR regulators of B. subtilis. SinR is a DNA-binding protein regulating gene expression and SinI inhibits SinR activity. Overexpression of the sin genes affects the expression of the inhA''-lacZ transcriptional fusion in B. thuringiensis: early induction of inhA expression was observed when sinI was overexpressed, whereas inhA expression was reduced in a strain overexpressing sinR, suggesting that inhA transcription is repressed, directly or indirectly, by SinR. inhA transcription was greatly reduced in B. thuringiensis and B. subtilis spo0A mutants. Analysis of the inhA''-lacZ expression in abrB and abrB-spo0A mutants of B. subtilis indicates that the Spo0A-dependent regulation of inhA expression depends on AbrB, which is known to regulate expression of transition state and sporulation genes in B. subtilis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=11429458');
INSERT INTO `artigos` (`id`, `componente`, `proteina`, `idExterno`, `titulo`, `abstract`, `data`, `link`) VALUES
(123, 37, 1, '10985426', 'Rapid communication: genetic linkage mapping of a caprine microsatellite derived from a genomic clone containing the INHA gene.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10985426'),
(124, 37, 1, '10912684', 'Linkage mapping of the ovine alpha-inhibin (INHA) beta(A)-inhibin/activin (INHBA) and beta(B)-inhibin/activin (INHBB) genes.', 'A full-sib mapping population was typed for Taql Southern blot RFLPs at the ovine alpha-inhibin (INHA), beta(A)-inhibin/activin (INHBA), and beta(B)-inhibin/activin (INHBB) loci. The most likely positions were found by genetic linkage analysis. INHA was placed on ovine chromosome 2q, between microsatellite markers BM6444 and INRA135, while INHBB was located 39 cM proximal to INHA between markers BMS1126 and BMS2626. The INHBA gene, a comparative mapping anchor locus, mapped to chromosome 4, between microsatellites OARCP26 and BMS719. These data confirm and refine previous chromosomal assignments by in situ hybridization, provide comparative mapping information, and identify microsatellite markers suitable for QTL association studies with inhibin/activin genes in ruminants.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10912684'),
(125, 37, 1, '10897383', 'Genomic mutations in the katG, inhA and aphC genes are useful for the prediction of isoniazid resistance in Mycobacterium tuberculosis isolates from Kwazulu Natal, South Africa.', 'Genotypic analysis of isoniazid (INH) resistance in 79 isolates of M. tuberculosis (MTB) was undertaken by PCR-single strand conformation polymorphism (SSCP), Msp1 restriction enzyme analysis and sequence analysis of specific regions of three genes (part of the coding sequence of katG, and promoter regions of the inhA operon and ahpC) in order to determine the particular allelic variants within these genes. The epidemiologic relatedness was determined using IS6110 and polymorphic G-C region (PGRS (MTB484(1)) based restriction fragment length polymorphism (RFLP). Mutations in katG, inhA locus and ahpC were identified in 77/79, 19/79 and 10/79 isolates respectively. The ability of PCR-SSCP to detect mutations associated with INH resistance in katG, inhA and ahpC genes was 100% (CI 91.2-99.7%), 98.7% (CI 74.0-99.9%), and 100% (CI 69.2-100%) respectively. Specificity was 100%. All isolates with mutations in the 209 bp fragment of the MTB katG gene containing the Ser315Thr codon were positive by PCR-RFLP using Msp1 enzyme restriction analysis. Sixteen of 19 isolates with alterations on the 3'' end of the ribosome binding site upstream of mabA in inhA locus simultaneously harbored Ser315Thr mutations in KatG. In 9/10 isolates, mutations in the ahpC promoter region were located in the 105 bp oxyR-ahpC intergenic region. None of 17 INH drug susceptible isolates harbored mutations in any of the three genetic regions, although the katG1 allele (Arg 463 Leu) was present in one isolate. Characterization by IS6110/PGRS(MTB484(1))RFLP analysis revealed that a number of drug resistant clones are widespread in the community. We conclude that the frequency of the Ser315Thr katG mutation in the local strain population makes the PCR-RFLP MTB katG assay a reliable, rapid and useful method for detecting INH resistance.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10897383'),
(126, 37, 1, '10869170', 'Inhibition of InhA, the enoyl reductase from Mycobacterium tuberculosis, by triclosan and isoniazid.', 'Structural and genetic studies indicate that the antibacterial compound triclosan, an additive in many personal care products, is an inhibitor of EnvM, the enoyl reductase from Escherichia coli. Here we show that triclosan specifically inhibits InhA, the enoyl reductase from Mycobacterium tuberculosis and a target for the antitubercular drug isoniazid. Binding of triclosan to wild-type InhA is uncompetitive with respect to both NADH and trans-2-dodecenoyl-CoA, with K(i)'' values of 0.22+/-0.02 and 0.21+/-0.01 microM, respectively. Replacement of Y158, the catalytic tyrosine residue, with Phe, reduces the affinity of triclosan for the enzyme and results in noncompetitive inhibition, with K(i) and K(i)'' values of 36+/-5 and 47+/-5 microM, respectively. Consequently, the Y158 hydroxyl group is important for triclosan binding, suggesting that triclosan binds in similar ways to both InhA and EnvM. In addition, the M161V and A124V InhA mutants, which result in resistance of Mycobacterium smegmatis to triclosan, show significantly reduced affinity for triclosan. Inhibition of M161V is noncompetitive with K(i)'' = 4.3+/-0.5 microM and K(i) = 4.4+/-0.9 microM, while inhibition of A124V is uncompetitive with K(i)'' = 0. 81 +/- 0.11 microM. These data support the hypothesis that the mycobacterial enoyl reductases are targets for triclosan. The M161V and A124V enzymes are also much less sensitive to isoniazid compared to the wild-type enzyme, indicating that triclosan can stimulate the emergence of isoniazid-resistant enoyl reductases. In contrast, I47T and I21V, two InhA mutations that occur in isoniazid-resistant clinical isolates of M. tuberculosis, show unimpaired inhibition by triclosan, with uncompetitive inhibition constants (K(i)'') of 0.18+/-0.01 and 0.12+/- 0.01 microM, respectively. The latter result indicates that InhA inhibitors targeted at the enoyl substrate binding site may be effective against existing isoniazid-resistant strains of M. tuberculosis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10869170'),
(127, 37, 1, '10869086', 'Inactivation of the inhA-encoded fatty acid synthase II (FASII) enoyl-acyl carrier protein reductase induces accumulation of the FASI end products and cell lysis of Mycobacterium smegmatis.', 'The mechanism of action of isoniazid (INH), a first-line antituberculosis drug, is complex, as mutations in at least five different genes (katG, inhA, ahpC, kasA, and ndh) have been found to correlate with isoniazid resistance. Despite this complexity, a preponderance of evidence implicates inhA, which codes for an enoyl-acyl carrier protein reductase of the fatty acid synthase II (FASII), as the primary target of INH. However, INH treatment of Mycobacterium tuberculosis causes the accumulation of hexacosanoic acid (C(26:0)), a result unexpected for the blocking of an enoyl-reductase. To test whether inactivation of InhA is identical to INH treatment of mycobacteria, we isolated a temperature-sensitive mutation in the inhA gene of Mycobacterium smegmatis that rendered InhA inactive at 42 degrees C. Thermal inactivation of InhA in M. smegmatis resulted in the inhibition of mycolic acid biosynthesis, a decrease in hexadecanoic acid (C(16:0)) and a concomitant increase of tetracosanoic acid (C(24:0)) in a manner equivalent to that seen in INH-treated cells. Similarly, INH treatment of Mycobacterium bovis BCG caused an inhibition of mycolic acid biosynthesis, a decrease in C(16:0), and a concomitant accumulation of C(26:0). Moreover, the InhA-inactivated cells, like INH-treated cells, underwent a drastic morphological change, leading to cell lysis. These data show that InhA inactivation, alone, is sufficient to induce the accumulation of saturated fatty acids, cell wall alterations, and cell lysis and are consistent with InhA being a primary target of INH.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10869086'),
(128, 37, 1, '10708367', 'InhA, a target of the antituberculous drug isoniazid, is involved in a mycobacterial fatty acid elongation system, FAS-II.', 'Most drug-resistant clinical isolates of the tubercle bacillus are resistant to isoniazid, a first-line antituberculous drug. This antibiotic was shown to act on Mycobacterium tuberculosis by inhibiting a 2-trans-enoyl-acyl carrier protein reductase, called InhA. However, the exact role played by InhA in mycobacteria remained unclear. A mycobacterial enzyme fraction containing InhA was isolated. It displays a long-chain fatty acid elongation activity with the characteristic properties described for the FAS-II (fatty acid synthetase II) system. Inhibition of this activity by InhA inhibitors, namely isoniazid, hexadecynoyl-CoA or octadecynoyl-CoA, showed that InhA belongs to the FAS-II system. Moreover, the InhA inhibitors also blocked the biosynthesis of mycolic acids, which are major lipids of the mycobacterial envelope. The data strongly suggest that isoniazid acts on the mycobacterial cell wall by preventing the FAS-II system from producing long-chain fatty acid precursors for mycolic acid biosynthesis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10708367'),
(129, 37, 1, '10644708', 'Action mechanism of antitubercular isoniazid. Activation by Mycobacterium tuberculosis KatG, isolation, and characterization of inha inhibitor.', 'Activation of the antitubercular isoniazid (INH) by the Mycobacterium tuberculosis KatG produces an inhibitor for enoyl reductase (InhA). The mechanism for INH activation remains poorly understood, and the inhibitor has never been isolated. We have purified the InhA-inhibitor complex generated in the M. tuberculosis KatG-catalyzed INH activation. The complex exhibited a 278-nm absorption peak and a shoulder around 326 nm with a characteristic A(326)/A(278) ratio of 0.16. The complex was devoid of enoyl reductase activity. The inhibitor noncovalently binds to InhA with a K(d) < 0.4 nM and can be dissociated from denatured InhA for chromatographic isolation. The free inhibitor showed absorption peaks at 326 (epsilon(326) 6900 M(-1) cm(-1)) and 260 nm (epsilon(260) 27,000 M(-1) cm(-1)). The inactive complex can be reconstituted from InhA and the isolated inhibitor. The InhA inhibitor from the KatG-catalyzed INH activation was identical to that from a slow, KatG-independent, Mn(2+)-mediated reaction based on high pressure liquid chromatography analysis and absorption and mass spectral characteristics. By monitoring the formation of the InhA-inhibitor complex, we have found that manganese is not essential to the INH activation by M. tuberculosis KatG. Furthermore, the formation of the InhA inhibitor in the KatG reaction was independent of InhA.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10644708'),
(130, 37, 1, '10521269', 'Roles of tyrosine 158 and lysine 165 in the catalytic mechanism of InhA, the enoyl-ACP reductase from Mycobacterium tuberculosis.', 'The role of tyrosine 158 (Y158) and lysine 165 (K165) in the catalytic mechanism of InhA, the enoyl-ACP reductase from Mycobacterium tuberculosis, has been investigated. These residues have been identified as putative catalytic residues on the basis of structural and sequence homology with the short chain alcohol dehydrogenase family of enzymes. Replacement of Y158 with phenylalanine (Y158F) and with alanine (Y158A) results in 24- and 1500-fold decreases in k(cat), respectively, while leaving K(m) for the substrate, trans-2-dodecenoyl-CoA, unaffected. Remarkably, however, replacement of Y158 with serine (Y158S) results in an enzyme with wild-type activity. Kinetic isotope effect studies indicate that the transfer of a solvent-exchangeable proton is partially rate-limiting for the wild-type and Y158S enzymes, but not for the Y158A enzyme. These data indicate that Y158 does not function formally as a proton donor in the reaction but likely functions as an electrophilic catalyst, stabilizing the transition state for hydride transfer by hydrogen bonding to the substrate carbonyl. A conformational change involving rotation of the Y158 side chain upon binding of the enoyl substrate to the enzyme is proposed as an explanation for the inverse solvent isotope effect observed on V/K(DD-CoA) when either NADH or NADD is used as the reductant. These data are consistent with the recently published structure of a C16 fatty acid substrate bound to InhA that shows Y158 hydrogen bonded to the substrate carbonyl group and rotated from the position it occupies in the InhA-NADH binary complex [Rozwarski, D. A., Vilcheze, C., Sugantino, M., Bittman, R., and Sacchettini, J. C. (1999) J. Biol. Chem. 274, 15582-15589]. Finally, the role of K165 has been analyzed using site-directed mutagenesis. Replacement of K165 with glutamine (K165Q) and arginine (K165R) has no effect on the enzyme''s catalytic ability or on its ability to bind NADH. However, the K165A and K165M enzymes are unable to bind NADH, indicating that K165 has a primary role in cofactor binding.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10521269'),
(131, 37, 1, '10336454', 'Crystal structure of the Mycobacterium tuberculosis enoyl-ACP reductase, InhA, in complex with NAD+ and a C16 fatty acyl substrate.', 'Enoyl-ACP reductases participate in fatty acid biosynthesis by utilizing NADH to reduce the trans double bond between positions C2 and C3 of a fatty acyl chain linked to the acyl carrier protein. The enoyl-ACP reductase from Mycobacterium tuberculosis, known as InhA, is a member of an unusual FAS-II system that prefers longer chain fatty acyl substrates for the purpose of synthesizing mycolic acids, a major component of mycobacterial cell walls. The crystal structure of InhA in complex with NAD+ and a C16 fatty acyl substrate, trans-2-hexadecenoyl-(N-acetylcysteamine)-thioester, reveals that the substrate binds in a general "U-shaped" conformation, with the trans double bond positioned directly adjacent to the nicotinamide ring of NAD+. The side chain of Tyr158 directly interacts with the thioester carbonyl oxygen of the C16 fatty acyl substrate and therefore could help stabilize the enolate intermediate, proposed to form during substrate catalysis. Hydrophobic residues, primarily from the substrate binding loop (residues 196-219), engulf the fatty acyl chain portion of the substrate. The substrate binding loop of InhA is longer than that of other enoyl-ACP reductases and creates a deeper substrate binding crevice, consistent with the ability of InhA to recognize longer chain fatty acyl substrates.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10336454'),
(132, 37, 1, '10049298', 'Genetic evidence that InhA of Mycobacterium smegmatis is a target for triclosan.', 'Three Mycobacterium smegmatis mutants selected for resistance to triclosan each had a different mutation in InhA, an enoyl reductase involved in fatty acid synthesis. Two expressed some isoniazid resistance. A mutation originally selected on isoniazid also mediated triclosan resistance, as did the wild-type inhA gene on a multicopy plasmid. Replacement of the mutant chromosomal inhA genes with wild-type inhA eliminated resistance. These results suggest that M. smegmatis InhA, like its Escherichia coli homolog FabI, is a target for triclosan.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=10049298'),
(133, 37, 1, '9802011', 'The mabA gene from the inhA operon of Mycobacterium tuberculosis encodes a 3-ketoacyl reductase that fails to confer isoniazid resistance.', 'A target of the anti-tuberculosis drugs isoniazid (INH) and ethionamide (ETH) has been shown to be an enoyl reductase, encoded by the inhA gene. The mabA (mycolic acid biosynthesis A) gene is located immediately upstream of inhA in Mycobacterium tuberculosis, Mycobacterium bovis and Mycobacterium smegmatis. The MabA protein from M. tuberculosis was expressed in Escherichia coli and shown to have 3-ketoacyl reductase activity, consistent with a role in mycolic acid biosynthesis. In M. smegmatis, inhA and mabA are independently transcribed, but in M. tuberculosis and M. bovis BCG, mabA and inhA constitute a single operon. Several INH-ETH-resistant M. tuberculosis clinical isolates contain point mutations in the ribosome-binding site of mabA in the mabA-inhA operon. However, genetic dissection of this operon reveals that the INH-ETH-resistance phenotype is encoded only by inhA, and not by mabA.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=9802011'),
(134, 37, 1, '9534249', 'Determination of a 15437 bp nucleotide sequence around the inhA gene of Mycobacterium avium and similarity analysis of the products of putative ORFs.', 'A 15437 bp region encompassing the inhA locus from the Mycobacterium avium chromosome was cloned and sequenced. From the sequencing data generated and the results of homology searches, the primary structure of this region was determined. This region contains four known genes (acnA, fabG, inhA and hemH) and two genes, invA and invB, whose products display homology with p60 invasion protein of Listeria monocytogenes. Six proteins encoded by putative ORFs contained an RGD motif (often involved in binding to macrophage integrins), while ORF1 and MoxR are probably transcriptional regulators. The rest of the putative products encoded by ORFs in the sequenced region showed little homology with the proteins contained in the databases and were considered to be unknown proteins.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=9534249'),
(135, 37, 1, '9417034', 'Modification of the NADH of the isoniazid target (InhA) from Mycobacterium tuberculosis.', 'The preferred antitubercular drug isoniazid specifically targets a long-chain enoyl-acyl carrier protein reductase (InhA), an enzyme essential for mycolic acid biosynthesis in Mycobacterium tuberculosis. Despite the widespread use of this drug for more than 40 years, its precise mode of action has remained obscure. Data from x-ray crystallography and mass spectrometry reveal that the mechanism of isoniazid action against InhA is covalent attachment of the activated form of the drug to the nicotinamide ring of nicotinamide adenine dinucleotide bound within the active site of InhA.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=9417034'),
(136, 37, 1, '9432394', 'Usefulness of Mycobacterium tuberculosis genomic mutations in the genes katG and inhA for the prediction of isoniazid resistance.', 'Mutations in two genes of Mycobacterium tuberculosis, inhA and katG, are known to correlate with resistance to isoniazid (INH).', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=9432394'),
(137, 37, 1, '8896513', 'Biochemical and genetic data suggest that InhA is not the primary target for activated isoniazid in Mycobacterium tuberculosis.', 'An examination of the pattern of lipid biosynthetic responses to isoniazid (INH) treatment of Mycobacterium tuberculosis and Mycobacterium smegmatis suggests that the mode of action of activated INH differs between these 2 organisms. Transformation of M. smegmatis with inhA on a plasmid construct conferred high-level resistance to INH, while the same construct failed to confer resistance upon M. tuberculosis. The inhA region from 2 clinical isolates whose resistance has been attributed to changes in the upstream promoter region has been cloned and was not sufficient to impart INH resistance to the level of the parent strain on sensitive M. tuberculosis. These putative mutant promoter elements appear to elevate expression levels of gene fusion reporter constructs, suggesting some noncausal connection between the observed mutations and the lipid metabolism of drug-resistant organisms. These results suggest that InhA is not the major target for activated INH in M. tuberculosis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=8896513'),
(138, 37, 1, '8684371', 'Evaluation of inhA gene and catalase-peroxidase gene among isoniazid-sensitive and resistant Mycobacterium tuberculosis isolates.', 'The katG gene and the inhA gene of 30 INH-resistant (INH-R) and 28 INH-sensitive (INH-S) isolates of M. tuberculosis from Haiti and Maryland were analysed by PCR to establish the presence and frequency of two postulated mechanisms of INH-resistance, total katG gene deletion and inhA Ser94 to Ala94 amino acid substitution. Only two of 30 INH-R isolates (3%) appear to have total katG gene deletions. All 28 INH-S isolates (100%) produced a PCR product at both the 5'' and the 3'' ends of the katG gene. Gene deletion of katG is a rare mechanism of INH resistance. Allele specific oligonucleotide hybridisation analysis of the inhA PCR products from the same 58 isolates revealed no mutation at amino acid 94 or directly surrounding it. Other inhA gene mutations may be responsible for INH resistance in M. tuberculosis. Diagnostic strategies using katG gene deletion or inhA Ser94 mutations would fail to detect almost all INH-R isolates.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=8684371'),
(139, 37, 1, '8537659', 'Characterization of the catalase-peroxidase gene (katG) and inhA locus in isoniazid-resistant and -susceptible strains of Mycobacterium tuberculosis by automated DNA sequencing: restricted array of mutations associated with drug resistance.', 'The catalase-peroxidase gene (katG) and a two-gene locus (inhA) containing mutations associated with resistance to isoniazid in Mycobacterium tuberculosis were sequenced in 34 resistant and 12 susceptible strains. Virtually all resistant organisms had amino acid changes in KatG or nucleotide substitutions upstream of inhA. A region of katG encoding two amino acids frequently altered in resistant strains (residues Ser315 and Arg463) and the inhA locus were sequenced in 10 susceptible and 51 isoniazid-resistant isolates from the Netherlands. Most (84%) of the resistant isolates had mutations in katG or the inhA locus or lacked katG. Together, approximately 75% of isoniazid-resistant isolates had replacements at amino acids 315 or 463 in KatG or nucleotide substitutions upstream of inhA. All 16 strains of Mycobacterium bovis and Mycobacterium microti studied had Leu463 rather than Arg463 in KatG, an observation consistent with the hypothesis that Leu463 is the ancestral condition in M. tuberculosis.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=8537659'),
(140, 37, 1, '8585728', 'Characterization of the katG and inhA genes of isoniazid-resistant clinical isolates of Mycobacterium tuberculosis.', 'Resistance to isoniazid in Mycobacterium tuberculosis has been associated with mutations in genes encoding the mycobacterial catalase-peroxidase (katG) and the InhA protein (inhA). Among the 26 isoniazid-resistant clinical isolates evaluated in this study, mutations in putative inhA regulatory sequences were identified in 2 catalase-positive isolates, katG gene alterations were detected in 20 strains, and 4 isolates had wild-type katG and inhA genes. Mutations in the katG gene were detected in all 11 catalase-negative isolates: one frameshift insertion, two partial gene deletions, and nine different missense mutations were identified. An arginine-to-leucine substitution at position 463 was detected in nine catalase-positive isolates. However, site-directed mutagenesis experiments demonstrated that the presence of a leucine at codon 463 did not alter the activity of the M. tuberculosis catalase-peroxidase and did not affect the capacity of this enzyme to restore isoniazid susceptibility to isoniazid-resistant, KatG-defective Mycobacterium smegmatis BH1 cells. These studies further support the association between katG and inhA gene mutations and isoniazid resistance in M. tuberculosis, while also suggesting that other undefined mechanisms of isoniazid resistance exist.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=8585728'),
(141, 37, 1, '8535088', 'Comparative mapping of sheep inhibin subunits alpha (INHA) and beta B(INHBB) to chromosome 2 in goat by FISH.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=8535088'),
(142, 37, 1, '7639680', 'Enoyl-acyl-carrier-protein reductase and Mycobacterium tuberculosis InhA do not conserve the Tyr-Xaa-Xaa-Xaa-Lys motif in mammalian 11 beta- and 17 beta-hydroxysteroid dehydrogenases and Drosophila alcohol dehydrogenase.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=7639680'),
(143, 37, 1, '7733505', 'RFLP markers for the porcine INHA locus.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=7733505'),
(144, 37, 1, '7733498', 'A TaqI RFLP at the bovine INHA locus.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=7733498'),
(145, 37, 1, '7613047', 'Comparative mapping of the gene coding for inhibin-alpha (INHA) to chromosome 2 in sheep and cattle.', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=7613047'),
(146, 37, 1, '7623658', 'Effect of inhA and katG on isoniazid resistance and virulence of Mycobacterium bovis.', 'Isoniazid (INH) resistance of the Mycobacterium tuberculosis Complex (MtbC) is associated with both loss of catalase activity and mutation of the inhA gene. However, the relative contributions of these changes to resistance and to the loss of virulence for guinea-pigs is unknown. In this study, a virulent strain of Mycobacterium bovis, a member of the MtbC, was exposed to increasing concentrations of INH. Two INH-resistant strains were produced which had lost catalase activity. Strain WAg405, which had a higher resistance to INH, also had a mutation in the inhA gene. This demonstrated that loss of catalase activity and mutation of inhA had a cumulative effect on INH resistance. When a functional katG gene was integrated into the genome of WAg405 the INH resistance was greatly reduced. This indicated that most of the resistance had been caused by loss of catalase activity. While the parent INH-sensitive strain was virulent for guinea-pigs, the INH-resistant strains were significantly less virulent. Integration of a functional katG gene into the most resistant strain restored full virulence. This clearly established that katG is a virulence factor for M. bovis and that mutation of the inhA gene has no effect on virulence.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=7623658'),
(147, 37, 1, '8284673', 'inhA, a gene encoding a target for isoniazid and ethionamide in Mycobacterium tuberculosis.', 'Isoniazid (isonicotinic acid hydrazide, INH) is one of the most widely used antituberculosis drugs, yet its precise target of action on Mycobacterium tuberculosis is unknown. A missense mutation within the mycobacterial inhA gene was shown to confer resistance to both INH and ethionamide (ETH) in M. smegmatis and in M. bovis. The wild-type inhA gene also conferred INH and ETH resistance when transferred on a multicopy plasmid vector to M. smegmatis and M. bovis BCG. The InhA protein shows significant sequence conservation with the Escherichia coli enzyme EnvM, and cell-free assays indicate that it may be involved in mycolic acid biosynthesis. These results suggest that InhA is likely a primary target of action for INH and ETH.', '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=8284673'),
(148, 37, 1, '4873628', '[Treatment of parkinsonism with the Bulgarian preparations INHA-17 and bellazon].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=4873628'),
(149, 37, 1, '5897866', '[Pharmacodynamic and therapeutic effect of the Bulgarian antitubercular preparation INHA-17].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=5897866'),
(150, 37, 1, '14148363', '[STUDY OF THE ABSORPTION, BLOOD CONCENTRATION AND EXCRETION IN THE URINE OF THE BULGARIAN ANTITUBERCULAR PREPARATION INHA-17].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=14148363'),
(151, 37, 1, '13896632', '[Treatment of parkinsonism with a new preparation INHA-17].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=13896632'),
(152, 37, 1, '13896631', '[Treatment of parkinsonism with a new combined preparation INHA-17 with Bellapan (Bellazon)].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=13896631'),
(153, 37, 1, '13884647', '[Is resistance break-through possible after INH and PAS long-term therapy in pulmonary tuberculosis by the use of INHA-PAS?].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=13884647'),
(154, 37, 1, '13373698', '[Clinical experience with INHA-PAS].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=13373698'),
(155, 37, 1, '14389156', '[Difference in the results of bacteriological studies on animals and in vitro, as demonstrated by a new tuberculostatic (INHA-PAS)].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=14389156'),
(156, 37, 1, '13236889', '[Tuberculostatic INHA-PAS in clinical examination].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=13236889'),
(157, 37, 1, '13235249', '[The compound drug INHA-PAS in the treatment of pulmonary tuberculosis].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=13235249'),
(158, 37, 1, '13205960', '[Preliminary observations of pulmonary tuberculosis therapy with the combination drug INHA-PAS].', NULL, '0000-00-00', 'https://www.ncbi.nlm.nih.gov/pubmed/?term=13205960');

-- --------------------------------------------------------

--
-- Table structure for table `autores`
--

CREATE TABLE IF NOT EXISTS `autores` (
`id` int(11) NOT NULL,
  `nome` varchar(255) CHARACTER SET utf16 NOT NULL
) ENGINE=MyISAM AUTO_INCREMENT=858 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `autores`
--

INSERT INTO `autores` (`id`, `nome`) VALUES
(849, 'Stratton, Christopher F'),
(848, 'García-Pérez, Adolfo'),
(847, 'Merget, Benjamin'),
(846, 'Saxena, Shalini'),
(845, 'Weimann, C'),
(844, 'Nikam, C'),
(843, 'Gray, Alexander I'),
(842, 'Young, Robert J'),
(841, 'Singh, Albel'),
(840, 'Hiendleder, S'),
(839, 'Rawat, Richa'),
(838, 'Tardy, Sébastien'),
(837, 'Lemos-Marini, S H V'),
(836, 'Dobner, P'),
(835, 'Whitty, Adrian'),
(834, 'Kuo, M'),
(833, 'Ding, Bei-chuan'),
(26, 'Campelo, Creuza L'),
(27, 'Peng, Jia'),
(28, 'Marozzi, Anna'),
(29, 'Padmalatha, V'),
(30, 'Barry, C E'),
(31, 'Tripathi, Ankit'),
(32, 'Pajk, Stane'),
(33, 'GANEV, G'),
(34, 'O''Donovan, Gary K'),
(35, 'de Souza, Osmar Norberto'),
(36, 'Smith, Paul W'),
(37, 'Romagnoli, M'),
(38, 'Hickey, M J'),
(39, 'Schneewind, Olaf'),
(40, 'Brüssow, Klaus P'),
(41, 'Pang, Y'),
(42, 'Khanna, Shalini C'),
(43, 'Tait, M'),
(44, 'Kulkarni, Manoj S'),
(45, 'Kastaniotis, Alexander J'),
(46, 'Rinder, H'),
(47, 'Huss, Sophie'),
(48, 'Ferreira Junior, Sergio Luiz Montego'),
(49, 'Naik, Maruti'),
(50, 'Gorichko, Marian'),
(51, 'Seidel, Véronique'),
(52, 'Ballell Pages, Lluís'),
(53, 'Chauhan, L S'),
(54, 'Rullás, Joaquín'),
(55, 'Fernández Menéndez, Raquel'),
(56, 'Locht, Camille'),
(57, 'Patel, Amish M'),
(58, 'Camacho, Luis R'),
(59, 'Jung, Sang Chul'),
(60, 'Santos, Diógenes Santiago'),
(61, 'Wang, Qing'),
(62, 'Dorn, Márcio'),
(63, 'Streicher, E M'),
(64, 'So, Wai-Kin'),
(65, 'Sullivan, Todd J'),
(66, 'Perryman, Alexander L'),
(67, 'Hegde, Naina'),
(68, 'Alian, Akram'),
(69, 'Guardia, Ana'),
(70, 'Vijayakrishnan, Rajakrishnan'),
(71, 'Ruiz, Duncan D A'),
(72, 'Harrison, Christopher F'),
(73, 'Alonso, Marta'),
(74, 'Jeankumar, Variam Ullas'),
(75, 'Woźna, Magdalena'),
(76, 'Dhar, Neeraj'),
(77, 'Herve, Maxime'),
(78, 'Meunier, Bernard'),
(79, 'Zhu, Z'),
(80, 'Parikh, S L'),
(81, 'Chen, Liqiang'),
(82, 'Rozwarski, D A'),
(83, 'Ballell, Lluis'),
(84, 'Heaslip, Darragh'),
(85, 'Pojer, Florence'),
(86, 'Figueiredo, B'),
(87, 'Schroeder, Evelyn K'),
(88, 'Parsons, Linda'),
(89, 'Saffon, Nathalie'),
(90, 'Remuiñán, Modesto J'),
(91, 'Bittman, R'),
(92, 'Lee, Sang Yup'),
(93, 'Neres, João'),
(94, 'Miranda, Silvana S'),
(95, 'Caminero, J A'),
(96, 'Bae, Hye Kyung'),
(97, 'Bollela, V R'),
(98, 'Chatterjee, Delphi'),
(99, 'Parikh, Sapan'),
(100, 'Jarp, Sarma'),
(101, 'Jelfs, Peter'),
(102, 'Lü, Cui-huan'),
(103, 'Feldmann, K'),
(104, 'Rodrigues, C'),
(105, 'KARAMALKOV, L'),
(106, 'Couto, Isabel'),
(107, 'Lee, Ann S G'),
(108, 'Boyne, Melissa'),
(109, 'Park, Tae Jung'),
(110, 'James, Rebecca'),
(111, 'Tseng, Shu-Ting'),
(112, 'Ali, Mohammad Tuhin'),
(113, 'Li, Y'),
(114, 'Cafrune, Patricia I'),
(115, 'Reiner, G'),
(116, 'Martínez-Hoyos, María'),
(117, 'Tonry, Jessica H'),
(118, 'Matviiuk, Tetiana'),
(119, 'Jäger, C'),
(120, 'Viveiros, Miguel'),
(121, 'Lu, Hao'),
(122, 'Nosova, E Iu'),
(123, 'Andreu-Vieyra, Claudia'),
(124, 'Hogg, Kirsten'),
(125, 'Chakravarty, B N'),
(126, 'Isakova, Zh T'),
(127, 'NANKOV, I'),
(128, 'Verbiest, Michael M P J'),
(129, 'Guido, Rafael V C'),
(130, 'Silva, Márcia S N'),
(131, 'Morlock, Glenn P'),
(132, 'Mata, Lydia'),
(133, 'Arai, Masayoshi'),
(134, 'Venkatraman, Janani'),
(135, 'Mansouri-Attia, Nadera'),
(136, 'Matzuk, Martin M'),
(137, 'Patthey-Vuadens, Ophélie'),
(138, 'Kysil, Andrii'),
(139, 'Gonzales-Cortez, Carolina'),
(140, 'Parikh, S'),
(141, 'Oh, HyeWon'),
(142, 'Monte, O'),
(143, 'Dover, Lynn G'),
(144, 'Deretic, Vojo'),
(145, 'Sintchenko, Vitali'),
(146, 'Zolfaghari, Mohammad Reza'),
(147, 'Tai, Chun-Hsi'),
(148, 'Espinoza, Roger C'),
(149, 'ATANASOV, K'),
(150, 'PIETSCH, W'),
(151, 'Grandvalet, C'),
(152, 'De Paris, Renata'),
(153, 'Maveyraud, Laurent'),
(154, 'Suffys, Philip Noel'),
(155, 'Feelders, Richard A'),
(156, 'Xu, Hua-Wei'),
(157, 'von Dadelszen, Peter'),
(158, 'Basso, Luiz A'),
(159, 'González, G M'),
(160, 'Ekins, Sean'),
(161, 'Liu, Lin'),
(162, 'Hua, Guo-Hua'),
(163, 'Rifai, M'),
(164, 'Kim, Mi Sun'),
(165, 'Inard, Cyril'),
(166, 'Roy, Rajendra P'),
(167, 'Uitterlinden, Andre G'),
(168, 'Yanase, Toshihiko'),
(169, 'KRISCH, J'),
(170, 'Gurvitz, Aner'),
(171, 'Coetzee, G J'),
(172, 'Boettger, Erik C'),
(173, 'Hou, Jinxing'),
(174, 'Dalla Costa, Elis Regina'),
(175, 'Jannaber, U'),
(176, 'SIMEONOV, M N'),
(177, 'Joshi, Shrinivas D'),
(178, 'Li, Huei-Jiun'),
(179, 'Cao, Binyun'),
(180, 'González Del Valle, Silvia'),
(181, 'Hofland, Leo J'),
(182, 'Sacchettini, James C'),
(183, 'Macheque, D'),
(184, 'Krasnova, M A'),
(185, 'Kumar, Devanand'),
(186, 'Ajbani, K'),
(187, 'Lenaerts, Anne'),
(188, 'Chen, Y D'),
(189, 'Jaśkowski, Jędrzej M'),
(190, 'Patil, Sachindra S'),
(191, 'Morris, S'),
(192, 'Basso, Luiz Augusto'),
(193, 'Martinelli, Leonardo K'),
(194, 'Pedgaonkar, Ganesh S'),
(195, 'Santos, Paula Gonçalves dos'),
(196, 'Leartsakulpanich, U'),
(197, 'Collins, D M'),
(198, 'Kapur, V'),
(199, 'da Costa, André L P'),
(200, 'Hazbon, Manzour H'),
(201, 'Norberto de Souza, Osmar'),
(202, 'Gominet, M'),
(203, 'Walker, John R'),
(204, 'Chen, Xi'),
(205, 'Kicka, Sébastien'),
(206, 'Slayden, Richard A'),
(207, 'Alvarez-Ruíz, Emilio'),
(208, 'Knudson, Dennis L'),
(209, 'Chung, Myung-Chul'),
(210, 'McKinney, John D'),
(211, 'Yam, W C'),
(212, 'Lei, B'),
(213, 'Melo, M B'),
(214, 'Lanéelle, G'),
(215, 'Perron, Karl'),
(216, 'Lin, Chin-Fu'),
(217, 'Vineeth, V P'),
(218, 'TSOLOV, N'),
(219, 'Poon, Shui-Kei'),
(220, 'Tran, Seav-Ly'),
(221, 'Ramos, Jorge'),
(222, 'Broussy, Sylvain'),
(223, 'McMiken, Helen'),
(224, 'Weisbrod, Torin R'),
(225, 'Singh, L'),
(226, 'Dubnau, E'),
(227, 'Knudson, Susan E'),
(228, 'Hazbón, Manzour Hernando'),
(229, 'Young, Robert'),
(230, 'Bishop, K S'),
(231, 'Bvs, Suneel Kumar'),
(232, 'Ng, Pearly Shuyi'),
(233, 'Ballell, Lluís'),
(234, 'Mourey, Lionel'),
(235, 'Zhan, Chang-Guo'),
(236, 'Salfinger, Max'),
(237, 'Rouse, D A'),
(238, 'Castro, M'),
(239, 'Cho, Se-Wook'),
(240, 'TSEKOVA, M'),
(241, 'Korduláková, Jana'),
(242, 'Kreiswirth, B N'),
(243, 'Nagarajan, Sathya Narayanan'),
(244, 'York, D F'),
(245, 'Glynne, Richard'),
(246, 'Lakshminarayana, Suresh B'),
(247, 'Tu, Shiao-Chun'),
(248, 'Truglio, James J'),
(249, 'Abkar, Roya'),
(250, 'García-Bustos, Jose Francisco'),
(251, 'Shi, Zhi-Yuan'),
(252, 'Hilbi, Hubert'),
(253, 'Bernardes-Génisson, Vania'),
(254, 'Turk, Samo'),
(255, 'Sosič, Izidor'),
(256, 'Cole, Stewart T'),
(257, 'Musser, James M'),
(258, 'Quémard, Annaïk'),
(259, 'Barton, D H'),
(260, 'Hartkoorn, Ruben C'),
(261, 'Orena, Beatrice Silvia'),
(262, 'Diagana, Thierry T'),
(263, 'Baumann, Ulrich'),
(264, 'Guerra-Junior, G'),
(265, 'Machado, Karina S'),
(266, 'Rossetti, Maria Lucia Rosa'),
(267, 'Yuen, K Y'),
(268, 'Lam, T H'),
(269, 'Aminabhavi, Tejraj M'),
(270, 'Musser, J M'),
(271, 'Cosson, Pierre'),
(272, 'Chen, Y'),
(273, 'VASILEVA, I'),
(274, 'Blair, John D'),
(275, 'An, Xiaopeng'),
(276, 'Kong, Fanrong'),
(277, 'Mori, Giorgia'),
(278, 'André-Barrès, Christiane'),
(279, 'Koohatammakun, N'),
(280, 'Alavi, Seyed Mohammad'),
(281, 'Hiltunen, J Kalervo'),
(282, 'Jaye, Andrew A'),
(283, 'Bretzel, G'),
(284, 'Lopes, Maria L'),
(285, 'van Helden, P D'),
(286, 'Hooker, Jacob M'),
(287, 'Rivera, G'),
(288, 'Löscher, T'),
(289, 'Koseki, Yuji'),
(290, 'Wassmuth, R'),
(291, 'O''Keefe, Heather'),
(292, 'Trollip, A'),
(293, 'Jarlier, Vincent'),
(294, 'Lereclus, D'),
(295, 'Kritski, Afrânio Lineu'),
(296, 'Garcia-Diaz, Miguel'),
(297, 'Persani, Luca'),
(298, 'Perozzo, Remo'),
(299, 'Chen, Ruihong'),
(300, 'Seto, W H'),
(301, 'Wu, Wei-Sheng'),
(302, 'Shelling, Andrew N'),
(303, 'Ho, John L'),
(304, 'McDermott, P F'),
(305, 'Arico-Muendel, Christopher C'),
(306, 'Choi, Soo-Keun'),
(307, 'Andricopulo, Adriano D'),
(308, 'Labò, M'),
(309, 'Fowler, Joanna S'),
(310, 'Barnes, Susan W'),
(311, 'Li, Guang'),
(312, 'Álvarez-Gómez, Daniel'),
(313, 'Song, Yuxuan'),
(314, 'Nair, Pravin A'),
(315, 'Devi, Parthiban Brindha'),
(316, 'Coppel, Yannick'),
(317, 'Zhang, B'),
(318, 'Wang, Feng'),
(319, 'Read, Jon A'),
(320, 'Sugantino, M'),
(321, 'JAHN, O'),
(322, 'Maschmann, Raquel de Abreu'),
(323, 'van Embden, J D'),
(324, 'Moroz, A M'),
(325, 'Baughn, Anthony D'),
(326, 'Giotti, Irene'),
(327, 'Rao, K L'),
(328, 'Silva, Marcia Susana Nunes'),
(329, 'Wu, Xue-qiong'),
(330, 'Lee, Eun Hee'),
(331, 'Latronico, A C'),
(332, 'Honoré, Nadine'),
(333, 'Sriram, Dharmarajan'),
(334, 'Suffys, Philip N'),
(335, 'O''Brien, K L'),
(336, 'Sikes, David'),
(337, 'Gilbert, Gwendolyn L'),
(338, 'Vilchèze, Catherine'),
(339, 'Bernard, Christine'),
(340, 'Fonseca, Leila de Souza'),
(341, 'He, Xin'),
(342, 'Banerjee, A'),
(343, 'Bennett, Eric M'),
(344, 'Holdgate, Geoffrey'),
(345, 'Sherman, D R'),
(346, 'Kahbazi, Manijeh'),
(347, 'Bonnac, Laurent'),
(348, 'Fernandez-Menendez, Raquel'),
(349, 'Morris, S L'),
(350, 'Mdluli, K'),
(351, 'Shetty, A'),
(352, 'Saha, Sanjib'),
(353, 'Ma, Yu'),
(354, 'Goldammer, T'),
(355, 'Aoki, Shunsuke'),
(356, 'Sadrnia, Maryam'),
(357, 'Wei, C J'),
(358, 'Choi, Sang Ki'),
(359, 'Eampokalap, Boonchuay'),
(360, 'Kremer, Laurent'),
(361, 'Kumar, Manoj'),
(362, 'Zawierucha, Piotr'),
(363, 'Álvarez-Ruíz, Emilio'),
(364, 'Basler, Christopher F'),
(365, 'Colangeli, Roberto'),
(366, 'Fu, W'),
(367, 'Mallet-Ladeira, Sonia'),
(368, 'Li, Z'),
(369, 'Pulakanam, Sridevi'),
(370, 'Noble, Christian G'),
(371, 'Riccardi, G'),
(372, 'Ortega, Fátima'),
(373, 'Chung, Chun-wa'),
(374, 'Guidetti, Brigitte'),
(375, 'Angulo-Barturen, Inigo'),
(376, 'Torres, Pedro'),
(377, 'Ortega-Muro, Fátima'),
(378, 'Jana, Tarakanta'),
(379, 'Ho, P L'),
(380, 'Robinson, Wendy P'),
(381, 'Bailey, Charles'),
(382, 'Toniolo, Daniela'),
(383, 'Ga, Hyuk'),
(384, 'Gulten, Gulcin'),
(385, 'Gingell, Helen'),
(386, 'McDowell, William'),
(387, 'Warren, R M'),
(388, 'Yu, Weixuan'),
(389, 'Paik, Hyoung-Rok'),
(390, 'Jung, Heung-Chae'),
(391, 'Wang, Y'),
(392, 'Scrignari, Tiziana'),
(393, 'You, Qi-Dong'),
(394, 'Rüsch-Gerdes, S'),
(395, 'Bommineni, Gopal R'),
(396, 'Forli, Stefano'),
(397, 'Zhao, Y'),
(398, 'Gersak, Ksenija'),
(399, 'Piotrowska, Hanna'),
(400, 'Smith, A N'),
(401, 'Boyne, Melissa E'),
(402, 'Samantaray, Sharmishtha'),
(403, 'Dixit, H'),
(404, 'Choi, Ji-Ho'),
(405, 'Mlisana, Koleka'),
(406, 'Sobhia, Masilamani E'),
(407, 'ROBEL, G'),
(408, 'Cooksey, Robert C'),
(409, 'Saparpakorn, P'),
(410, 'Guillemet, Elisabeth'),
(411, 'Baltas, Michel'),
(412, 'Ribeiro, Marta O'),
(413, 'Balasubramanian, V'),
(414, 'Kondreddi, Ravinder Reddy'),
(415, 'Rožman, Kaja'),
(416, 'Schwerin, M'),
(417, 'Pérez-Macías, Natalia'),
(418, 'Li, Shao-Gang'),
(419, 'Ambady, Anisha'),
(420, 'Harris, Sarah E'),
(421, 'Brunner, R M'),
(422, 'Lee, Sang-Jin'),
(423, 'Madacki, Jan'),
(424, 'Palaci, Moises'),
(425, 'Prueksaaroon, S'),
(426, 'Han, Dong-Yop'),
(427, 'Singhal, Megha'),
(428, 'Sun, Yong-Jiang'),
(429, 'Olson, Arthur J'),
(430, 'Tonge, Peter J'),
(431, 'Chertow, Daniel S'),
(432, 'Gupta, Sunil'),
(433, 'Rossetti, Maria L R'),
(434, 'Amin, Amol'),
(435, 'More, Uttam A'),
(436, 'Zhang, Guang-yu'),
(437, 'Tonge, P J'),
(438, 'Kritski, Afrânio L'),
(439, 'DEBERTSHAUSER, E'),
(440, 'Shao, H'),
(441, 'Gau, Sylvain'),
(442, 'Stratakis, C A'),
(443, 'Huang, J'),
(444, 'Bandodkar, Balachandra'),
(445, 'Yao, Hong-Wei'),
(446, 'Bocanegra-García, V'),
(447, 'Gruppo, Veronica'),
(448, 'Jimenez, M Soledad'),
(449, 'Rostirolla, Diana C'),
(450, 'Bosman, M E'),
(451, 'Sambandamurthy, Vasan K'),
(452, 'Gegina, Yelizaveta'),
(453, 'Cohen, Marcelo'),
(454, 'Madhavapeddi, Prashanti'),
(455, 'Kuo, Mack'),
(456, 'Malik, Siddharth'),
(457, 'Jung, Woo-Suc'),
(458, 'Wang, Yong'),
(459, 'Delbot, Alexandra'),
(460, 'Sbragia-Neto, L'),
(461, 'Erhardt, G'),
(462, 'Manjunatha, Ujjini H'),
(463, 'Kashanchi, Fatah'),
(464, 'Yang, Li-Guo'),
(465, 'Kritski, Afrânio'),
(466, 'Felczak, Krzysztof'),
(467, 'Levy, S B'),
(468, 'Parikh, Amit'),
(469, 'Steenbergen, Jacobie'),
(470, 'Nandicoori, Vinay Kumar'),
(471, 'Jorgensen, Shelley C'),
(472, 'Mendoza, Alfonso'),
(473, 'Sougakoff, Wladimir'),
(474, 'Bell, Alasdair F'),
(475, 'Hameed, Shahul'),
(476, 'Harrison, L H'),
(477, 'Neu, Margarete'),
(478, 'Müller, B'),
(479, 'Rullas-Triconado, Joaquin'),
(480, 'Umei, Tomohiro'),
(481, 'Ahmadi, Azam'),
(482, 'Remuiñán, Modesto'),
(483, 'Hoek, K G P'),
(484, 'Li, Chia-Ru'),
(485, 'Grant, G A'),
(486, 'Punkvang, A'),
(487, 'Dalla Costa, Elis R'),
(488, 'Ortiz de Montellano, Paul R'),
(489, 'S Rao, Srinivasa P'),
(490, 'Woo, W L'),
(491, 'Victor, T C'),
(492, 'McMurry, L M'),
(493, 'Heifets, Leonid'),
(494, 'Stibitz, Scott'),
(495, 'Alland, David'),
(496, 'Arnold, Liane S'),
(497, 'Lee, Woo-Yiel'),
(498, 'Lu, J'),
(499, 'Kuhen, Kelli'),
(500, 'Molle, Virginie'),
(501, 'Veyron-Churlet, Romain'),
(502, 'Zhang, Z'),
(503, 'Ferrer-Bazaga, Santiago'),
(504, 'Castañeda, Pablo'),
(505, 'Amin, Anita G'),
(506, 'Chand, Ashwini L'),
(507, 'Evindar, Ghotas'),
(508, 'Fabing, Isabelle'),
(509, 'Clark, Matthew A'),
(510, 'Niehaus, Abraham J'),
(511, 'Schuettler, Julia'),
(512, 'Yang, Hongfang'),
(513, 'Deenadayal, M'),
(514, 'Pauli, Ivani'),
(515, 'Panitchakorn, Jarinee'),
(516, 'Ouertatani-Sakouhi, Hajer'),
(517, 'Sola, Christophe'),
(518, 'Xu, Hua'),
(519, 'Wen, Qun-Ying'),
(520, 'Ma, Ng L'),
(521, 'Sakamoto, Hiroshi'),
(522, 'Ho, Yu Min'),
(523, 'Um, K S'),
(524, 'Liang, Jian-qin'),
(525, 'Convery, Maire'),
(526, 'Ballell-Pages, Lluís'),
(527, 'Prasad, Sudha'),
(528, 'Lopez-Roman, Eva Maria'),
(529, 'Ritter, Claudia'),
(530, 'Davie, Christopher P'),
(531, 'Chen, Ya-Dong'),
(532, 'Yao, C'),
(533, 'dos Santos, Ricardo N'),
(534, 'Johnson, Francis'),
(535, 'Grandke, R'),
(536, 'Chiriano, Gianpaolo'),
(537, 'Liu, Nina'),
(538, 'Moynihan, D P'),
(539, 'Beltschewa-Petrowa, M'),
(540, 'Sridevi, Jonnalagadda Padma'),
(541, 'Calvo, David'),
(542, 'de Jong, Frank H'),
(543, 'Blanco, Delia'),
(544, 'Gao, Guang-Yao'),
(545, 'Kim, You-Hoi'),
(546, 'FRANK, F'),
(547, 'Jose, M'),
(548, 'Bailey, Charles L'),
(549, 'Dzapo, V'),
(550, 'Leung, Lawrence W'),
(551, 'Larsen, Michelle H'),
(552, 'Pungpo, P'),
(553, 'Myers, Michelle'),
(554, 'Stigliani, Jean-Luc'),
(555, 'Kao, R Y'),
(556, 'Wei, Chih-Jen'),
(557, 'Gomes, Harrison Magdinier'),
(558, 'Zanella-Cléon, Isabelle'),
(559, 'Gey van Pittius, N C'),
(560, 'Stoliarova, L G'),
(561, 'Hoosain, E'),
(562, 'Schacherl, Magdalena'),
(563, 'Vilchèze, C'),
(564, 'Pratviel, Geneviève'),
(565, 'Luckner, Sylvia R'),
(566, 'Mendoza-Losana, Alfonso'),
(567, 'amEnde, Christopher W'),
(568, 'Bai, G H'),
(569, 'Centrella, Paolo A'),
(570, 'Moreira, A'),
(571, 'Chen, Lin'),
(572, 'Modha, Sundip'),
(573, 'Collins, D'),
(574, 'Li, Chuan-you'),
(575, 'Vadwai, V'),
(576, 'Lei, Benfang'),
(577, 'Dodds, K G'),
(578, 'Kanakavalli, M'),
(579, 'Alvarez-Gomez, Daniel'),
(580, 'de Krijger, Ronald R'),
(581, 'KHADZHIEV, D'),
(582, 'Shinde, Vikas'),
(583, 'Wong, Sin-Yew'),
(584, 'Seong, Chi Nam'),
(585, 'Calliari, L E P'),
(586, 'Wilson, Samantha L'),
(587, 'Besra, Gurdyal S'),
(588, 'Woo, So-Yun'),
(589, 'Kim, TaeSoo'),
(590, 'Portugal, Isabel'),
(591, 'Jin, Qi'),
(592, 'Bishai, William'),
(593, 'Liberatore, R'),
(594, 'Baulard, Alain'),
(595, 'Lu, X Y'),
(596, 'Kitamura, Mitsuru'),
(597, 'Castro-Pichel, Julia'),
(598, 'Soldati, Thierry'),
(599, 'Langlois, Sylvie'),
(600, 'Sotriffer, Christoph A'),
(601, 'Morbidoni, Hector R'),
(602, 'Gobec, Stanislav'),
(603, 'Gurvitz, A'),
(604, 'Chen, Shi-Lin'),
(605, 'Delaine, Tamara'),
(606, 'D, Sriram'),
(607, 'Shilpi, Jamil A'),
(608, 'Encinas, Lourdes'),
(609, 'Ritacco, Viviana'),
(610, 'Chabula-Nxiweni, E M'),
(611, 'Chen, Qi-Kang'),
(612, 'Živec, Matej'),
(613, 'Geldermann, H'),
(614, 'de Lisle, G'),
(615, 'Martín-Casabona, Nuria'),
(616, 'Jacobs, William R'),
(617, 'Kisker, Caroline'),
(618, 'Pan, Jae-Gu'),
(619, 'Telles, Maria A'),
(620, 'Quémard, A'),
(621, 'Namburete, E I'),
(622, 'Torricelli, Francesca'),
(623, 'Martínez, María'),
(624, 'You, Q D'),
(625, 'Cruz-Pulido, W'),
(626, 'Lu, Yang'),
(627, 'Voitenko, Zoia'),
(628, 'Kumar, Vivek'),
(629, 'Machado, Diana'),
(630, 'van Soolingen, D'),
(631, 'Ge, Wei'),
(632, 'Pérez-Herrán, Esther'),
(633, 'Gandhi, Neel R'),
(634, 'Brust, James C M'),
(635, 'Childs, James'),
(636, 'Kiepiela, P'),
(637, 'Feliciano, C S'),
(638, 'Popova, Taissia G'),
(639, 'Baik, Keun Sik'),
(640, 'Altmann, Karl-Heinz'),
(641, 'Khan, Shazia'),
(642, 'Kamsri, P'),
(643, 'Lee, Yeon Ji'),
(644, 'Koli, Deepshikha'),
(645, 'Chollet, Aurélien'),
(646, 'Mahmood, Shaik'),
(647, 'Srisupan, A'),
(648, 'Kang, Seong Ki'),
(649, 'Wadia, Nupur'),
(650, 'CHAIKOV, I'),
(651, 'WILDE, W'),
(652, 'Jackowska, Marta'),
(653, 'Šink, Roman'),
(654, 'Bernadou, Jean'),
(655, 'Arjomandzadegan, Mohammad'),
(656, 'Jiang, Yong-Jun'),
(657, 'de Jesus Lopes Ribeiro, Ana Luisa'),
(658, 'Dietz, H C'),
(659, 'Cai, Lin'),
(660, 'KARAMALAKOV, L'),
(661, 'Tan, Bee H'),
(662, 'Valim, Andréia R M'),
(663, 'Nishi, Yoshihiro'),
(664, 'Popov, Serguei'),
(665, 'Garza-González, E'),
(666, 'Roux, L'),
(667, 'Rossetti, Raffaella'),
(668, 'Pankiewicz, Krzysztof W'),
(669, 'Soman, R'),
(670, 'Vogt, Peter'),
(671, 'Quemard, A'),
(672, 'Esteves, Leonardo Souza'),
(673, 'Nguyen, Michel'),
(674, 'Sobhia, M Elizabeth'),
(675, 'Wang, Hui'),
(676, 'Kumar, Uday Chandra'),
(677, 'Bindal, Deepak'),
(678, 'Blasco, Francesca'),
(679, 'Deshmukh, M'),
(680, 'Ramarao, Nalini'),
(681, 'am Ende, Christopher W'),
(682, 'Tu, S C'),
(683, 'McNichol, Beth A'),
(684, 'Peloquin, Charles A'),
(685, 'Bhatt, Apoorva'),
(686, 'Bukowska, Dorota'),
(687, 'Lun, Shichun'),
(688, 'Gusberti, L'),
(689, 'Wang, Xin'),
(690, 'Iwamoto, H'),
(691, 'Alvarez-Gómez, Daniel'),
(692, 'Hasan, Shihab'),
(693, 'Nandishaiah, Radha'),
(694, 'Cadot, Céline'),
(695, 'Antosik, Paweł'),
(696, 'Schroeder, Evelyn Koeche'),
(697, 'Panda, Manoranjan'),
(698, 'Khusmith, Srisin'),
(699, 'Kremer, Kristin'),
(700, 'Rodriguez, Frédéric'),
(701, 'Williams, D L'),
(702, 'Zaha, Arnaldo'),
(703, 'Perdigão, João'),
(704, 'Scapozza, Leonardo'),
(705, 'Freundlich, Joel S'),
(706, 'Arnaud, Philippe'),
(707, 'Speziale, P'),
(708, 'Kruh, Nicole A'),
(709, 'Baptista, M T M'),
(710, 'Taira, Junichi'),
(711, 'Senna, Simone G'),
(712, 'Messer, Jeff A'),
(713, 'Leung, E T Y'),
(714, 'Horlacher, Oliver P'),
(715, 'Rocha, M N'),
(716, 'Vaiman, D'),
(717, 'Menendez, Christophe'),
(718, 'Wolschann, P'),
(719, 'Novichenok, Polina'),
(720, 'Kaur, Tejinder'),
(721, 'Xiao, G'),
(722, 'Kranjc-Pietrucci, Agata'),
(723, 'Marrakchi, H'),
(724, 'Zhang, Jun-xian'),
(725, 'Nowicki, Michał'),
(726, 'Maughan, William N'),
(727, 'Zhang, Xujie'),
(728, 'Zhu, T'),
(729, 'Morbidoni, H R'),
(730, 'Constant, Patricia'),
(731, 'Slepikas, Liudas'),
(732, 'Bione, Silvia'),
(733, 'Signorino-Gelo, François'),
(734, 'Stover, C K'),
(735, 'Yogeeswari, Perumal'),
(736, 'Blanco Ruano, Delia'),
(737, 'Voorsluijs, Jacoba M'),
(738, 'Kumar, Anupriya'),
(739, 'Crawford, Jack T'),
(740, 'Jiang, Guang-lu'),
(741, 'Ng, Seow H'),
(742, 'Perrone, Victoria'),
(743, 'Silva, Márcia Susana N'),
(744, 'Tufariello, JoAnn'),
(745, 'Brossier, Florence'),
(746, 'Zi, Xiang-Dong'),
(747, 'Corre, Tanguy'),
(748, 'Rullas-Trincado, Joaquin'),
(749, 'Tardelli, A M D'),
(750, 'de Lisle, G W'),
(751, 'Wilson, T M'),
(752, 'Weisbrod, T R'),
(753, 'Khadka, Dhruba Kumar'),
(754, 'Perez-Herran, Esther'),
(755, 'Wilson, T'),
(756, 'Santos, Diógenes S'),
(757, 'Komatsu, Hideyuki'),
(758, 'Meewong, P'),
(759, 'Kanetaka, Hironori'),
(760, 'Guinebretière, Marie-Hélène'),
(761, 'Metchock, Beverly'),
(762, 'Yu, Xiaobin'),
(763, 'Nadagouda, Mallikarjuna N'),
(764, 'Nigri, A'),
(765, 'Alvarez, Emilio'),
(766, 'Shen, Zhong'),
(767, 'Pasca, Maria Rosalia'),
(768, 'Moaddab, Reza'),
(769, 'HELL, M'),
(770, 'Sarmadian, Hossein'),
(771, 'Galkina, K Iu'),
(772, 'Julien, Sylviane'),
(773, 'Pangas, Stephanie A'),
(774, 'Shah, Sonam'),
(775, 'Kumar-Sahu, Prashanta'),
(776, 'Mukherjee, Dhritiman V'),
(777, 'Simmerling, Carlos'),
(778, 'Gómez, Jesus'),
(779, 'Taymans, S E'),
(780, 'Liu, Li'),
(781, 'Hofland, Johannes'),
(782, 'Timmers, Luís F S M'),
(783, 'Veziris, Nicolas'),
(784, 'Goodarzi, Hamed'),
(785, 'Tarasevicius, Eduardas'),
(786, 'Jacobs, W R'),
(787, 'Baker, M E'),
(788, 'Popov, Serguei G'),
(789, 'Pan, Pan'),
(790, 'Longui, C A'),
(791, 'Telles, Maria Alice'),
(792, 'Rossetti, Maria Lucia R'),
(793, 'Ramasoota, Pongrama'),
(794, 'Winship, Ingrid M'),
(795, 'Li, Hong-min'),
(796, 'Murugan, Kannan'),
(797, 'Subba Rao, Gita'),
(798, 'Mendonca, B B'),
(799, 'Convery, Maire A'),
(800, 'Hannongbua, S'),
(801, 'Davies, Gareth'),
(802, 'Cohen, Elisangela M L'),
(803, 'Gupta, N'),
(804, 'Eiden, J'),
(805, 'Quevedo, Christian V'),
(806, 'Skotnikova, O I'),
(807, 'Lu, Xiao-Yun'),
(808, 'Ducati, Rodrigo G'),
(809, 'Barros, David'),
(810, 'Oliveira, Martha Maria'),
(811, 'Kempisty, Bartosz'),
(812, 'Nawata, Hajime'),
(813, 'Cho, Eun Hae'),
(814, 'Rai, Arvind'),
(815, 'Rentería, A'),
(816, 'Lancellotti, C L P'),
(817, 'Angulo-Barturen, Iñigo'),
(818, 'Mathema, Barun'),
(819, 'Zhang, L'),
(820, 'Sacchettini, J C'),
(821, 'Renuka, Janupally'),
(822, 'Lherbet, Christian'),
(823, 'Stec, Jozef'),
(824, 'Beer, David'),
(825, 'da Silva, Pedro E Almeida'),
(826, 'Fernandez, Raquel'),
(827, 'Kozikowski, Alan P'),
(828, 'Ruzsicska, Béla P'),
(829, 'Kim, Kwang Sik'),
(830, 'Watanabe, C'),
(831, 'Shirude, Pravin S'),
(832, 'Bhat, Jyothi'),
(850, 'Lereclus, Didier'),
(851, 'Khosravi, Azar Dokht'),
(852, 'Lai, Cheng-Tsung'),
(853, 'de Herder, Wouter W'),
(854, 'Wang, Yana'),
(855, 'SIRAKOV, A'),
(856, 'Lavandera, Jose Luis'),
(857, 'Rossi, E D');

-- --------------------------------------------------------

--
-- Table structure for table `autores_artigos`
--

CREATE TABLE IF NOT EXISTS `autores_artigos` (
  `idArtigo` int(11) NOT NULL,
  `idAutor` int(11) NOT NULL
) ENGINE=MyISAM DEFAULT CHARSET=latin1;

--
-- Dumping data for table `autores_artigos`
--

INSERT INTO `autores_artigos` (`idArtigo`, `idAutor`) VALUES
(1, 255),
(1, 415),
(1, 472),
(1, 602),
(1, 608),
(1, 826),
(1, 842),
(2, 118),
(2, 138),
(2, 241),
(2, 261),
(2, 277),
(2, 278),
(2, 367),
(2, 411),
(2, 423),
(2, 627),
(2, 700),
(2, 717),
(2, 767),
(2, 822),
(3, 47),
(3, 55),
(3, 76),
(3, 83),
(3, 116),
(3, 180),
(3, 182),
(3, 210),
(3, 233),
(3, 250),
(3, 312),
(3, 372),
(3, 375),
(3, 376),
(3, 384),
(3, 448),
(3, 503),
(3, 504),
(3, 566),
(3, 572),
(3, 579),
(3, 597),
(3, 608),
(3, 615),
(3, 632),
(3, 691),
(3, 733),
(3, 736),
(3, 748),
(3, 754),
(3, 765),
(3, 809),
(3, 817),
(3, 848),
(3, 856),
(4, 95),
(4, 97),
(4, 183),
(4, 476),
(4, 621),
(4, 637),
(5, 253),
(5, 258),
(5, 277),
(5, 554),
(5, 564),
(5, 645),
(5, 654),
(5, 730),
(5, 767),
(5, 822),
(6, 628),
(6, 674),
(7, 146),
(7, 249),
(7, 346),
(7, 356),
(7, 481),
(7, 655),
(7, 768),
(7, 770),
(8, 52),
(8, 54),
(8, 69),
(8, 73),
(8, 132),
(8, 182),
(8, 229),
(8, 318),
(8, 372),
(8, 384),
(8, 482),
(8, 525),
(8, 541),
(8, 543),
(8, 566),
(8, 597),
(8, 623),
(8, 632),
(8, 754),
(8, 778),
(8, 826),
(9, 32),
(9, 116),
(9, 207),
(9, 255),
(9, 312),
(9, 363),
(9, 373),
(9, 477),
(9, 526),
(9, 566),
(9, 579),
(9, 597),
(9, 602),
(9, 608),
(9, 612),
(9, 632),
(9, 653),
(9, 691),
(9, 754),
(9, 799),
(9, 809),
(9, 842),
(10, 51),
(10, 112),
(10, 352),
(10, 607),
(10, 692),
(10, 843),
(11, 263),
(11, 562),
(12, 72),
(12, 137),
(12, 205),
(12, 215),
(12, 252),
(12, 271),
(12, 298),
(12, 392),
(12, 516),
(12, 536),
(12, 598),
(12, 704),
(12, 722),
(12, 731),
(12, 785),
(12, 838),
(13, 628),
(13, 674),
(14, 124),
(14, 157),
(14, 274),
(14, 380),
(14, 586),
(14, 599),
(15, 405),
(15, 510),
(15, 633),
(15, 634),
(15, 818),
(16, 286),
(16, 309),
(16, 430),
(16, 626),
(16, 675),
(16, 780),
(16, 789),
(17, 71),
(17, 152),
(17, 201),
(17, 805),
(18, 293),
(18, 339),
(18, 473),
(18, 517),
(18, 745),
(18, 783),
(19, 178),
(19, 296),
(19, 388),
(19, 395),
(19, 430),
(19, 742),
(19, 774),
(19, 777),
(19, 852),
(20, 241),
(20, 253),
(20, 258),
(20, 277),
(20, 411),
(20, 423),
(20, 508),
(20, 645),
(20, 700),
(20, 717),
(20, 730),
(20, 767),
(20, 822),
(21, 600),
(21, 847),
(22, 153),
(22, 234),
(22, 253),
(22, 411),
(22, 459),
(22, 564),
(22, 645),
(22, 654),
(22, 772),
(22, 822),
(23, 44),
(23, 177),
(23, 269),
(23, 435),
(23, 644),
(23, 763),
(24, 182),
(24, 289),
(24, 355),
(24, 384),
(24, 480),
(24, 521),
(24, 596),
(24, 710),
(24, 757),
(24, 759),
(25, 66),
(25, 160),
(25, 388),
(25, 396),
(25, 418),
(25, 429),
(25, 430),
(25, 689),
(25, 705),
(26, 36),
(26, 58),
(26, 77),
(26, 203),
(26, 232),
(26, 245),
(26, 246),
(26, 262),
(26, 310),
(26, 370),
(26, 388),
(26, 414),
(26, 430),
(26, 462),
(26, 489),
(26, 499),
(26, 520),
(26, 661),
(26, 678),
(26, 741),
(26, 824),
(27, 32),
(27, 139),
(27, 254),
(27, 255),
(27, 312),
(27, 348),
(27, 375),
(27, 479),
(27, 526),
(27, 528),
(27, 579),
(27, 602),
(27, 608),
(27, 612),
(27, 653),
(27, 691),
(27, 809),
(27, 817),
(27, 842),
(28, 41),
(28, 391),
(28, 397),
(28, 498),
(28, 502),
(29, 74),
(29, 194),
(29, 315),
(29, 333),
(29, 540),
(29, 735),
(29, 821),
(29, 846),
(30, 74),
(30, 194),
(30, 315),
(30, 333),
(30, 540),
(30, 735),
(30, 821),
(30, 846),
(31, 66),
(31, 338),
(31, 592),
(31, 616),
(31, 687),
(31, 689),
(31, 705),
(31, 823),
(31, 827),
(32, 128),
(32, 155),
(32, 167),
(32, 181),
(32, 469),
(32, 542),
(32, 580),
(32, 737),
(32, 781),
(32, 853),
(33, 406),
(33, 628),
(34, 48),
(34, 154),
(34, 174),
(34, 195),
(34, 266),
(34, 295),
(34, 322),
(34, 328),
(34, 557),
(34, 672),
(34, 810),
(35, 196),
(35, 279),
(35, 409),
(35, 425),
(35, 486),
(35, 552),
(35, 642),
(35, 647),
(35, 718),
(35, 758),
(35, 800),
(36, 178),
(36, 190),
(36, 206),
(36, 227),
(36, 296),
(36, 395),
(36, 430),
(36, 537),
(36, 777),
(36, 789),
(36, 852),
(37, 178),
(37, 296),
(37, 388),
(37, 395),
(37, 430),
(37, 537),
(37, 777),
(37, 789),
(37, 852),
(38, 32),
(38, 34),
(38, 47),
(38, 52),
(38, 54),
(38, 57),
(38, 69),
(38, 90),
(38, 207),
(38, 291),
(38, 305),
(38, 312),
(38, 363),
(38, 375),
(38, 377),
(38, 386),
(38, 417),
(38, 477),
(38, 504),
(38, 507),
(38, 509),
(38, 512),
(38, 530),
(38, 566),
(38, 569),
(38, 579),
(38, 597),
(38, 608),
(38, 632),
(38, 691),
(38, 712),
(38, 754),
(38, 799),
(38, 817),
(39, 85),
(39, 93),
(39, 256),
(39, 260),
(39, 319),
(39, 385),
(39, 640),
(39, 714),
(40, 50),
(40, 118),
(40, 277),
(40, 374),
(40, 411),
(40, 627),
(40, 700),
(40, 767),
(40, 822),
(41, 111),
(41, 147),
(41, 216),
(41, 251),
(41, 484),
(42, 50),
(42, 89),
(42, 118),
(42, 367),
(42, 411),
(42, 627),
(42, 657),
(42, 700),
(42, 767),
(42, 822),
(43, 49),
(43, 67),
(43, 134),
(43, 223),
(43, 319),
(43, 344),
(43, 419),
(43, 444),
(43, 451),
(43, 454),
(43, 475),
(43, 582),
(43, 693),
(43, 697),
(43, 738),
(43, 796),
(43, 801),
(43, 831),
(43, 832),
(44, 239),
(44, 383),
(44, 404),
(44, 422),
(44, 426),
(44, 457),
(44, 545),
(44, 588),
(44, 643),
(45, 628),
(45, 674),
(46, 129),
(46, 158),
(46, 193),
(46, 201),
(46, 307),
(46, 449),
(46, 514),
(46, 533),
(46, 756),
(46, 782),
(46, 808),
(47, 106),
(47, 120),
(47, 172),
(47, 221),
(47, 529),
(47, 590),
(47, 629),
(47, 703),
(48, 83),
(48, 100),
(48, 231),
(48, 233),
(48, 312),
(48, 369),
(48, 456),
(48, 579),
(48, 606),
(48, 646),
(48, 676),
(48, 691),
(48, 775),
(49, 27),
(49, 110),
(49, 135),
(49, 553),
(49, 773),
(50, 31),
(50, 378),
(50, 649),
(50, 677),
(51, 40),
(51, 75),
(51, 189),
(51, 362),
(51, 399),
(51, 652),
(51, 686),
(51, 695),
(51, 725),
(51, 811),
(52, 104),
(52, 186),
(52, 217),
(52, 351),
(52, 547),
(52, 575),
(52, 669),
(52, 679),
(52, 844),
(53, 253),
(53, 554),
(53, 564),
(53, 654),
(54, 42),
(54, 53),
(54, 427),
(54, 432),
(54, 527),
(54, 814),
(55, 173),
(55, 179),
(55, 275),
(55, 311),
(55, 313),
(55, 854),
(56, 165),
(56, 411),
(56, 645),
(56, 700),
(56, 717),
(56, 767),
(56, 822),
(57, 39),
(57, 117),
(57, 209),
(57, 431),
(57, 463),
(57, 494),
(57, 548),
(57, 664),
(57, 680),
(57, 683),
(57, 829),
(58, 35),
(58, 265),
(58, 453),
(58, 802),
(59, 280),
(59, 784),
(59, 851),
(60, 430),
(60, 789),
(61, 156),
(61, 458),
(61, 746),
(62, 165),
(62, 411),
(62, 441),
(62, 700),
(62, 717),
(62, 767),
(62, 822),
(63, 35),
(63, 62),
(63, 87),
(63, 199),
(63, 235),
(63, 514),
(64, 182),
(64, 325),
(64, 338),
(64, 364),
(64, 455),
(64, 495),
(64, 550),
(64, 616),
(64, 705),
(64, 744),
(65, 117),
(65, 209),
(65, 381),
(65, 638),
(65, 664),
(65, 680),
(65, 776),
(65, 829),
(66, 43),
(66, 63),
(66, 171),
(66, 285),
(66, 292),
(66, 387),
(66, 450),
(66, 478),
(66, 483),
(66, 491),
(66, 559),
(66, 561),
(66, 610),
(67, 182),
(67, 338),
(67, 360),
(67, 384),
(67, 500),
(67, 501),
(67, 558),
(67, 616),
(68, 166),
(68, 185),
(68, 243),
(68, 402),
(68, 468),
(68, 470),
(68, 641),
(68, 685),
(68, 841),
(69, 78),
(69, 253),
(69, 258),
(69, 605),
(69, 654),
(69, 730),
(70, 113),
(70, 317),
(70, 366),
(70, 443),
(70, 532),
(70, 728),
(70, 819),
(71, 188),
(71, 595),
(71, 624),
(72, 430),
(72, 537),
(72, 565),
(72, 617),
(72, 681),
(73, 603),
(74, 393),
(74, 531),
(74, 807),
(75, 96),
(75, 330),
(75, 648),
(75, 813),
(76, 220),
(76, 410),
(76, 680),
(76, 694),
(76, 760),
(76, 850),
(77, 159),
(77, 287),
(77, 446),
(77, 625),
(77, 665),
(77, 815),
(78, 101),
(78, 145),
(78, 276),
(78, 337),
(78, 659),
(79, 64),
(79, 161),
(79, 219),
(79, 631),
(79, 762),
(80, 107),
(80, 428),
(80, 522),
(80, 583),
(81, 92),
(81, 109),
(81, 306),
(81, 390),
(81, 618),
(82, 117),
(82, 209),
(82, 471),
(82, 548),
(82, 638),
(82, 788),
(83, 393),
(83, 531),
(83, 656),
(83, 807),
(84, 28),
(84, 297),
(84, 326),
(84, 382),
(84, 511),
(84, 622),
(84, 667),
(84, 670),
(84, 732),
(84, 747),
(85, 26),
(85, 94),
(85, 114),
(85, 130),
(85, 148),
(85, 284),
(85, 303),
(85, 334),
(85, 340),
(85, 412),
(85, 424),
(85, 433),
(85, 438),
(85, 449),
(85, 487),
(85, 496),
(85, 609),
(85, 619),
(85, 699),
(85, 825),
(86, 126),
(87, 70),
(87, 361),
(87, 797),
(88, 78),
(88, 253),
(88, 554),
(88, 605),
(88, 654),
(88, 706),
(89, 45),
(89, 170),
(89, 281),
(90, 65),
(90, 108),
(90, 206),
(90, 208),
(90, 227),
(90, 430),
(90, 452),
(90, 518),
(90, 534),
(90, 537),
(90, 635),
(90, 681),
(90, 684),
(91, 59),
(91, 164),
(91, 358),
(91, 389),
(91, 497),
(91, 584),
(91, 639),
(92, 68),
(92, 341),
(92, 488),
(93, 162),
(93, 301),
(93, 445),
(93, 464),
(93, 519),
(93, 571),
(93, 604),
(93, 611),
(93, 766),
(94, 65),
(94, 84),
(94, 98),
(94, 121),
(94, 187),
(94, 206),
(94, 401),
(94, 430),
(94, 447),
(94, 505),
(94, 567),
(95, 430),
(95, 708),
(95, 828),
(95, 839),
(96, 81),
(96, 141),
(96, 343),
(96, 347),
(96, 430),
(96, 466),
(96, 518),
(96, 537),
(96, 544),
(96, 589),
(96, 668),
(97, 359),
(97, 515),
(97, 698),
(97, 753),
(97, 793),
(98, 123),
(98, 136),
(98, 299),
(99, 99),
(99, 282),
(99, 314),
(99, 430),
(99, 474),
(99, 719),
(99, 727),
(99, 839),
(99, 849),
(100, 206),
(100, 430),
(100, 617),
(101, 65),
(101, 178),
(101, 206),
(101, 248),
(101, 401),
(101, 430),
(101, 434),
(101, 534),
(101, 617),
(101, 719),
(101, 720),
(101, 727),
(101, 849),
(102, 133),
(102, 182),
(102, 224),
(102, 228),
(102, 318),
(102, 338),
(102, 360),
(102, 365),
(102, 495),
(102, 616),
(103, 29),
(103, 125),
(103, 225),
(103, 327),
(103, 403),
(103, 513),
(103, 578),
(103, 803),
(104, 102),
(104, 329),
(104, 436),
(104, 524),
(104, 626),
(104, 724),
(104, 795),
(104, 833),
(105, 211),
(105, 267),
(105, 268),
(105, 300),
(105, 379),
(105, 490),
(105, 555),
(105, 713),
(106, 168),
(106, 302),
(106, 398),
(106, 420),
(106, 506),
(106, 663),
(106, 794),
(106, 812),
(107, 122),
(107, 184),
(107, 324),
(107, 771),
(107, 806),
(108, 35),
(108, 60),
(108, 192),
(108, 696),
(109, 61),
(109, 204),
(109, 353),
(109, 574),
(109, 591),
(109, 740),
(110, 78),
(110, 222),
(110, 253),
(110, 258),
(110, 316),
(110, 654),
(111, 86),
(111, 142),
(111, 213),
(111, 238),
(111, 264),
(111, 331),
(111, 442),
(111, 460),
(111, 570),
(111, 585),
(111, 593),
(111, 709),
(111, 715),
(111, 749),
(111, 764),
(111, 779),
(111, 790),
(111, 798),
(111, 816),
(111, 830),
(111, 837),
(112, 131),
(112, 336),
(112, 408),
(112, 739),
(112, 761),
(113, 430),
(113, 835),
(113, 839),
(114, 131),
(114, 408),
(114, 412),
(114, 465),
(114, 662),
(114, 702),
(114, 711),
(114, 743),
(114, 791),
(114, 792),
(115, 78),
(115, 222),
(115, 316),
(115, 654),
(115, 673),
(116, 56),
(116, 143),
(116, 144),
(116, 182),
(116, 247),
(116, 332),
(116, 338),
(116, 360),
(116, 587),
(116, 594),
(116, 601),
(116, 616),
(116, 726),
(117, 247),
(117, 257),
(117, 556),
(117, 576),
(118, 88),
(118, 182),
(118, 200),
(118, 236),
(118, 338),
(118, 360),
(118, 493),
(118, 495),
(118, 551),
(118, 587),
(118, 616),
(119, 78),
(119, 222),
(119, 258),
(119, 654),
(119, 673),
(120, 115),
(120, 549),
(120, 613),
(120, 840),
(121, 79),
(121, 272),
(121, 440),
(122, 151),
(122, 202),
(122, 294),
(123, 461),
(123, 716),
(123, 840),
(123, 845),
(124, 290),
(124, 577),
(124, 840),
(125, 230),
(125, 244),
(125, 400),
(125, 636),
(125, 666),
(126, 80),
(126, 437),
(126, 721),
(127, 563),
(127, 690),
(127, 729),
(127, 752),
(127, 786),
(127, 820),
(127, 834),
(128, 214),
(128, 620),
(128, 671),
(128, 723),
(129, 212),
(129, 357),
(129, 682),
(130, 140),
(130, 437),
(130, 538),
(130, 721),
(131, 82),
(131, 91),
(131, 320),
(131, 563),
(131, 820),
(132, 304),
(132, 467),
(132, 492),
(133, 320),
(133, 342),
(133, 786),
(133, 820),
(134, 308),
(134, 371),
(134, 688),
(134, 707),
(134, 857),
(135, 82),
(135, 259),
(135, 485),
(135, 786),
(135, 820),
(136, 46),
(136, 103),
(136, 163),
(136, 283),
(136, 288),
(136, 394),
(136, 836),
(137, 30),
(137, 38),
(137, 191),
(137, 242),
(137, 345),
(137, 350),
(137, 734),
(138, 37),
(138, 335),
(138, 658),
(138, 804),
(139, 198),
(139, 242),
(139, 270),
(139, 323),
(139, 630),
(139, 701),
(140, 237),
(140, 349),
(140, 368),
(140, 568),
(141, 354),
(141, 416),
(141, 421),
(141, 840),
(142, 787),
(143, 175),
(143, 535),
(143, 840),
(143, 845),
(144, 535),
(144, 840),
(144, 845),
(145, 119),
(145, 354),
(145, 416),
(145, 421),
(145, 840),
(146, 197),
(146, 750),
(146, 751),
(147, 226),
(147, 342),
(147, 413),
(147, 523),
(147, 573),
(147, 614),
(147, 620),
(147, 671),
(147, 755),
(147, 786),
(148, 560),
(149, 539),
(150, 176),
(151, 33),
(151, 149),
(151, 218),
(151, 240),
(151, 273),
(151, 581),
(151, 650),
(151, 660),
(151, 855),
(152, 33),
(152, 105),
(152, 127),
(152, 149),
(152, 218),
(152, 240),
(152, 273),
(152, 581),
(152, 855),
(153, 439),
(154, 169),
(155, 651),
(156, 407),
(156, 546),
(157, 769),
(158, 150),
(158, 321);

-- --------------------------------------------------------

--
-- Table structure for table `componentescoletarefinamento`
--

CREATE TABLE IF NOT EXISTS `componentescoletarefinamento` (
`ID` int(10) unsigned NOT NULL,
  `Nome` varchar(255) NOT NULL,
  `Tipo` char(3) NOT NULL,
  `proteina` int(11) DEFAULT NULL,
  `NomeTabela` varchar(255) DEFAULT NULL,
  `componenteVisual` int(11) DEFAULT NULL,
  `Configuracao` text,
  `Criacao` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `Alteracao` timestamp NULL DEFAULT NULL,
  `Ativo` tinyint(1) NOT NULL DEFAULT '1'
) ENGINE=MyISAM AUTO_INCREMENT=47 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `componentescoletarefinamento`
--

INSERT INTO `componentescoletarefinamento` (`ID`, `Nome`, `Tipo`, `proteina`, `NomeTabela`, `componenteVisual`, `Configuracao`, `Criacao`, `Alteracao`, `Ativo`) VALUES
(43, 'ComponentePDB', 'col', 1, 'pdb_InhA', 1, 'InhA', '2016-11-15 06:04:59', '2016-11-15 06:04:59', 1),
(44, 'ComponentePDB', 'col', 151, 'pdb_Arginase', 1, 'Arginase', '2016-11-15 06:04:59', '2016-11-15 06:04:59', 1),
(45, 'ComponentePDB', 'col', NULL, 'pdb_Hemoglobin', 1, 'Hemoglobin', '2016-11-15 06:04:59', '2016-11-15 06:04:59', 1),
(46, 'Graficos', 'ref', NULL, NULL, NULL, NULL, '2016-11-15 06:06:03', '2016-11-15 06:06:03', 1);

-- --------------------------------------------------------

--
-- Table structure for table `componentesvisuais`
--

CREATE TABLE IF NOT EXISTS `componentesvisuais` (
`ID` int(11) NOT NULL,
  `nome` varchar(255) NOT NULL
) ENGINE=MyISAM AUTO_INCREMENT=2 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `componentesvisuais`
--

INSERT INTO `componentesvisuais` (`ID`, `nome`) VALUES
(1, 'pdb');

-- --------------------------------------------------------

--
-- Table structure for table `log`
--

CREATE TABLE IF NOT EXISTS `log` (
`id` int(10) unsigned NOT NULL,
  `dataHora` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
  `mensagem` text NOT NULL,
  `origem` varchar(128) DEFAULT NULL
) ENGINE=InnoDB AUTO_INCREMENT=436 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `log`
--

INSERT INTO `log` (`id`, `dataHora`, `mensagem`, `origem`) VALUES
(3, '2016-11-05 22:13:23', '2', 'BS'),
(4, '2016-11-05 22:27:04', 'logando', 'BS'),
(5, '2016-11-05 22:27:20', 'logando', 'BS'),
(6, '2016-11-05 22:27:41', 'logando', 'BS'),
(7, '2016-11-05 22:29:34', 'logando', 'BS'),
(8, '2016-11-05 22:30:05', 'logando', 'BS'),
(9, '2016-11-05 22:30:29', 'logando', 'BS'),
(10, '2016-11-05 22:31:43', 'logando', 'BS'),
(11, '2016-11-05 22:32:13', 'logando', 'BS'),
(12, '2016-11-06 00:17:47', 'Erro ao tentar executar o componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(13, '2016-11-06 00:17:47', 'Erro ao tentar executar o componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 10', 'BS'),
(14, '2016-11-06 03:04:27', 'Erro ao tentar executar o componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(15, '2016-11-06 03:34:25', 'Erro ao tentar executar o componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(16, '2016-11-06 03:38:17', 'Erro ao tentar executar o componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(17, '2016-11-06 03:38:17', 'Erro ao tentar executar o componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 10', 'BS'),
(18, '2016-11-06 03:38:19', 'Erro ao tentar executar o componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(19, '2016-11-06 03:38:19', 'Erro ao tentar executar o componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 10', 'BS'),
(20, '2016-11-06 03:38:21', 'Erro ao tentar executar o componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(21, '2016-11-06 03:38:21', 'Erro ao tentar executar o componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 10', 'BS'),
(22, '2016-11-06 03:42:06', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(23, '2016-11-06 03:42:06', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 10', 'BS'),
(24, '2016-11-06 03:43:59', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(25, '2016-11-06 04:00:46', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(26, '2016-11-06 04:00:46', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 10', 'BS'),
(27, '2016-11-06 04:00:48', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 10', 'BS'),
(28, '2016-11-06 04:00:48', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 10', 'BS'),
(29, '2016-11-07 03:34:31', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(30, '2016-11-07 03:34:31', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(31, '2016-11-07 03:36:00', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(32, '2016-11-07 03:36:01', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(33, '2016-11-07 03:37:13', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(34, '2016-11-07 03:37:13', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(35, '2016-11-07 03:37:37', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(36, '2016-11-07 03:37:37', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(37, '2016-11-07 03:38:07', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(38, '2016-11-07 03:38:07', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(39, '2016-11-07 03:38:58', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(40, '2016-11-07 03:38:58', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(41, '2016-11-07 03:40:05', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(42, '2016-11-07 03:40:05', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(43, '2016-11-07 03:40:59', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(44, '2016-11-07 03:40:59', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(45, '2016-11-07 03:41:15', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(46, '2016-11-07 03:41:15', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(47, '2016-11-07 03:42:06', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(48, '2016-11-07 03:42:06', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(49, '2016-11-07 03:42:19', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(50, '2016-11-07 03:42:19', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(51, '2016-11-07 03:43:27', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(52, '2016-11-07 03:43:27', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(53, '2016-11-07 03:44:06', 'Erro ao tentar importar componente. ComponenteID:1 ComponenteNome:PubMed AgendamentoID: 9', 'BS'),
(54, '2016-11-07 03:44:06', 'Erro ao tentar importar componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(55, '2016-11-08 00:56:01', 'Erro ao tentar executar o componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(56, '2016-11-08 00:59:41', 'Erro ao tentar executar o componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(57, '2016-11-08 01:16:36', 'Erro ao tentar executar o componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(58, '2016-11-08 01:54:54', 'Erro ao tentar executar o componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(59, '2016-11-08 01:58:28', 'Erro ao tentar executar o componente. ComponenteID:17 ComponenteNome:PDB AgendamentoID: 9', 'BS'),
(60, '2016-11-10 23:50:51', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(61, '2016-11-10 23:52:01', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(62, '2016-11-13 02:48:26', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(63, '2016-11-13 02:50:27', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(64, '2016-11-13 02:50:30', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(65, '2016-11-13 02:50:33', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(66, '2016-11-13 02:50:36', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(67, '2016-11-13 02:50:39', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(68, '2016-11-13 02:50:41', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(69, '2016-11-13 02:50:47', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(70, '2016-11-13 02:50:50', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(71, '2016-11-13 02:50:52', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(72, '2016-11-13 02:50:55', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(73, '2016-11-13 02:50:58', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(74, '2016-11-13 02:51:01', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(75, '2016-11-13 02:51:04', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(76, '2016-11-13 02:51:07', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(77, '2016-11-13 02:51:45', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(78, '2016-11-13 02:53:42', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(79, '2016-11-13 02:53:44', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(80, '2016-11-13 02:53:47', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(81, '2016-11-13 02:53:50', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(82, '2016-11-13 02:53:52', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(83, '2016-11-13 02:54:31', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(84, '2016-11-13 02:54:32', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(85, '2016-11-13 03:42:54', 'Erro ao gerenciar threads, AgendamentoID: 6', 'BS'),
(86, '2016-11-13 03:42:56', 'Erro ao gerenciar threads, AgendamentoID: 6', 'BS'),
(87, '2016-11-13 03:44:04', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(88, '2016-11-13 03:44:04', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(89, '2016-11-13 03:44:04', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(90, '2016-11-13 03:44:04', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(91, '2016-11-13 03:44:42', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(92, '2016-11-13 03:44:42', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(93, '2016-11-13 03:44:42', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(94, '2016-11-13 03:44:42', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(95, '2016-11-13 03:46:35', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(96, '2016-11-13 03:46:35', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(97, '2016-11-13 03:46:35', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(98, '2016-11-13 03:46:35', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(99, '2016-11-13 03:46:37', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(100, '2016-11-13 03:46:37', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(101, '2016-11-13 03:46:37', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(102, '2016-11-13 03:46:37', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(103, '2016-11-13 03:48:32', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(104, '2016-11-13 03:48:32', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(105, '2016-11-13 03:50:00', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(106, '2016-11-13 03:50:00', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(107, '2016-11-13 03:50:00', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(108, '2016-11-13 03:50:00', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(109, '2016-11-13 03:52:41', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(110, '2016-11-13 03:52:41', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(111, '2016-11-13 03:52:41', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(112, '2016-11-13 03:52:41', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(113, '2016-11-13 03:53:17', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(114, '2016-11-13 03:53:17', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(115, '2016-11-13 03:53:17', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(116, '2016-11-13 03:53:17', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(117, '2016-11-13 03:53:32', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(118, '2016-11-13 03:53:32', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(119, '2016-11-13 03:53:32', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(120, '2016-11-13 03:53:32', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(121, '2016-11-13 03:54:05', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(122, '2016-11-13 03:54:05', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(123, '2016-11-13 03:54:05', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(124, '2016-11-13 03:54:05', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(125, '2016-11-13 03:54:42', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(126, '2016-11-13 03:54:42', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(127, '2016-11-13 03:54:42', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(128, '2016-11-13 03:54:42', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(129, '2016-11-13 03:55:07', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(130, '2016-11-13 03:55:07', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(131, '2016-11-13 03:55:38', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(132, '2016-11-13 03:55:38', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(133, '2016-11-13 03:56:10', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(134, '2016-11-13 04:05:59', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(135, '2016-11-13 04:06:01', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(136, '2016-11-13 04:06:03', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(137, '2016-11-13 04:06:15', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(138, '2016-11-13 04:06:15', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(139, '2016-11-13 04:06:17', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(140, '2016-11-13 04:06:17', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(141, '2016-11-13 04:06:19', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(142, '2016-11-13 04:06:19', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(143, '2016-11-13 04:07:36', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(144, '2016-11-13 04:07:36', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(145, '2016-11-13 04:07:36', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(146, '2016-11-13 04:07:36', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(147, '2016-11-13 04:07:38', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(148, '2016-11-13 04:07:38', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(149, '2016-11-13 04:07:38', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(150, '2016-11-13 04:07:38', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(151, '2016-11-13 04:07:40', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(152, '2016-11-13 04:07:40', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(153, '2016-11-13 04:07:40', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(154, '2016-11-13 04:07:40', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(155, '2016-11-13 04:08:07', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(156, '2016-11-13 04:08:07', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(157, '2016-11-13 04:08:07', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(158, '2016-11-13 04:08:07', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(159, '2016-11-13 04:08:09', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(160, '2016-11-13 04:08:09', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(161, '2016-11-13 04:08:09', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(162, '2016-11-13 04:08:09', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(163, '2016-11-13 04:08:11', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(164, '2016-11-13 04:08:11', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(165, '2016-11-13 04:08:11', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(166, '2016-11-13 04:08:11', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(167, '2016-11-13 04:08:13', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(168, '2016-11-13 04:08:13', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(169, '2016-11-13 04:08:13', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(170, '2016-11-13 04:08:13', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(171, '2016-11-13 04:08:31', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(172, '2016-11-13 04:08:31', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(173, '2016-11-13 04:08:31', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(174, '2016-11-13 04:08:31', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(175, '2016-11-13 04:08:33', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(176, '2016-11-13 04:08:33', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(177, '2016-11-13 04:08:33', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(178, '2016-11-13 04:08:34', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(179, '2016-11-13 04:09:56', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(180, '2016-11-13 04:09:56', 'Erro ao tentar importar componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(181, '2016-11-13 04:09:56', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(182, '2016-11-13 04:09:59', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(183, '2016-11-13 04:09:59', 'Erro ao tentar importar componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(184, '2016-11-13 04:09:59', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(185, '2016-11-13 04:10:02', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(186, '2016-11-13 04:10:02', 'Erro ao tentar importar componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(187, '2016-11-13 04:10:02', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(188, '2016-11-13 04:10:02', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(189, '2016-11-13 04:10:02', 'Erro ao tentar importar componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(190, '2016-11-13 04:10:02', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(191, '2016-11-13 04:10:04', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(192, '2016-11-13 04:10:04', 'Erro ao gerenciar threads, AgendamentoID: 6', 'BS'),
(193, '2016-11-13 04:10:04', 'Erro ao tentar importar componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(194, '2016-11-13 04:10:04', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(195, '2016-11-13 04:10:05', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(196, '2016-11-13 04:10:05', 'Erro ao tentar importar componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(197, '2016-11-13 04:10:05', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(198, '2016-11-13 04:10:07', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(199, '2016-11-13 04:10:07', 'Erro ao tentar importar componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(200, '2016-11-13 04:10:07', 'Erro ao tentar executar o componente. ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(201, '2016-11-13 04:10:07', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(202, '2016-11-13 04:10:07', 'Erro ao tentar importar componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(203, '2016-11-13 04:10:07', 'Erro ao tentar executar o componente. ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(204, '2016-11-13 04:13:40', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(205, '2016-11-13 04:13:40', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(206, '2016-11-13 04:15:10', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(207, '2016-11-13 04:15:10', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(208, '2016-11-13 04:15:56', 'Erro ao gerenciar threads, AgendamentoID: 6', 'BS'),
(209, '2016-11-13 04:17:51', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(210, '2016-11-13 04:17:51', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(211, '2016-11-13 04:27:05', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(212, '2016-11-13 04:27:06', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(213, '2016-11-13 04:27:09', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(214, '2016-11-13 04:27:09', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(215, '2016-11-13 04:27:53', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(216, '2016-11-13 04:27:55', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(217, '2016-11-13 04:27:57', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(218, '2016-11-13 04:27:58', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(219, '2016-11-13 04:28:01', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(220, '2016-11-13 04:28:01', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(221, '2016-11-13 04:30:53', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(222, '2016-11-13 04:30:53', 'Erro ao gerenciar threads, AgendamentoID: 6', 'BS'),
(223, '2016-11-13 04:30:55', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(224, '2016-11-13 04:30:55', 'Erro ao gerenciar threads, AgendamentoID: 6', 'BS'),
(225, '2016-11-13 04:30:57', 'Erro ao gerenciar threads, AgendamentoID: 6', 'BS'),
(226, '2016-11-13 04:30:57', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(227, '2016-11-13 04:31:17', 'Erro ao gerenciar threads, AgendamentoID: 6', 'BS'),
(228, '2016-11-13 04:31:17', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(229, '2016-11-13 04:31:20', 'Erro ao gerenciar threads, AgendamentoID: 6', 'BS'),
(230, '2016-11-13 04:31:20', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(231, '2016-11-13 04:31:47', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(232, '2016-11-13 04:32:33', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(233, '2016-11-13 04:32:35', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(234, '2016-11-13 04:32:37', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(235, '2016-11-13 04:32:38', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(236, '2016-11-13 04:33:18', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(237, '2016-11-13 04:33:18', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(238, '2016-11-13 04:33:21', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(239, '2016-11-13 04:33:21', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(240, '2016-11-13 04:33:24', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(241, '2016-11-13 04:33:24', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(242, '2016-11-13 04:33:27', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(243, '2016-11-13 04:33:27', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(244, '2016-11-13 04:33:30', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(245, '2016-11-13 04:33:30', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(246, '2016-11-13 04:36:25', 'Executando ComponenteID:12 ComponenteNome:ComponentePDB AgendamentoID: 6', 'BS'),
(247, '2016-11-13 04:36:25', 'Executando ComponenteID:13 ComponenteNome:PubMED AgendamentoID: 6', 'BS'),
(248, '2016-11-14 04:20:37', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(249, '2016-11-14 04:20:37', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(250, '2016-11-14 04:20:37', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(251, '2016-11-14 04:20:39', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(252, '2016-11-14 04:20:39', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(253, '2016-11-14 04:20:39', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(254, '2016-11-14 04:24:29', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(255, '2016-11-14 04:24:29', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(256, '2016-11-14 04:24:29', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(257, '2016-11-14 04:24:31', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(258, '2016-11-14 04:24:31', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(259, '2016-11-14 04:24:31', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(260, '2016-11-14 04:24:33', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(261, '2016-11-14 04:24:33', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(262, '2016-11-14 04:24:33', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(263, '2016-11-14 04:25:02', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(264, '2016-11-14 04:25:02', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(265, '2016-11-14 04:25:02', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(266, '2016-11-14 04:25:02', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(267, '2016-11-14 04:25:04', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(268, '2016-11-14 04:25:04', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(269, '2016-11-14 04:25:04', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(270, '2016-11-14 04:25:05', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(271, '2016-11-14 04:25:31', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(272, '2016-11-14 04:25:31', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(273, '2016-11-14 04:25:31', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(274, '2016-11-14 04:25:31', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(275, '2016-11-14 04:25:33', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(276, '2016-11-14 04:25:33', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(277, '2016-11-14 04:25:33', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(278, '2016-11-14 04:25:33', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(279, '2016-11-14 04:25:35', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(280, '2016-11-14 04:25:35', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(281, '2016-11-14 04:25:35', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(282, '2016-11-14 04:25:35', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(283, '2016-11-14 04:25:37', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(284, '2016-11-14 04:25:37', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(285, '2016-11-14 04:25:37', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(286, '2016-11-14 04:25:37', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(287, '2016-11-14 04:25:54', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(288, '2016-11-14 04:25:54', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(289, '2016-11-14 04:25:54', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(290, '2016-11-14 04:25:54', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(291, '2016-11-14 04:25:56', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(292, '2016-11-14 04:25:56', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(293, '2016-11-14 04:25:56', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(294, '2016-11-14 04:25:56', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(295, '2016-11-14 04:25:58', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(296, '2016-11-14 04:25:58', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(297, '2016-11-14 04:25:58', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(298, '2016-11-14 04:25:59', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(299, '2016-11-14 04:26:01', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(300, '2016-11-14 04:26:01', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(301, '2016-11-14 04:26:01', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(302, '2016-11-14 04:26:01', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(303, '2016-11-14 04:26:12', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(304, '2016-11-14 04:26:12', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(305, '2016-11-14 04:26:12', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(306, '2016-11-14 04:26:12', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(307, '2016-11-14 04:26:35', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(308, '2016-11-14 04:26:36', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(309, '2016-11-14 04:26:36', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(310, '2016-11-14 04:26:36', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(311, '2016-11-14 04:26:38', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(312, '2016-11-14 04:26:38', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(313, '2016-11-14 04:26:38', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(314, '2016-11-14 04:26:38', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(315, '2016-11-14 04:27:36', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(316, '2016-11-14 04:27:36', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(317, '2016-11-14 04:27:36', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(318, '2016-11-14 04:27:36', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(319, '2016-11-14 04:27:38', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(320, '2016-11-14 04:27:38', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(321, '2016-11-14 04:27:38', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(322, '2016-11-14 04:27:38', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(323, '2016-11-14 04:28:14', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(324, '2016-11-14 04:28:14', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(325, '2016-11-14 04:28:14', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(326, '2016-11-14 04:28:14', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(327, '2016-11-14 04:28:16', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(328, '2016-11-14 04:28:16', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(329, '2016-11-14 04:28:16', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(330, '2016-11-14 04:28:16', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(331, '2016-11-14 04:28:34', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(332, '2016-11-14 04:28:34', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(333, '2016-11-14 04:28:34', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(334, '2016-11-14 04:28:34', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(335, '2016-11-14 04:28:36', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(336, '2016-11-14 04:28:36', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(337, '2016-11-14 04:28:36', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(338, '2016-11-14 04:28:36', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(339, '2016-11-14 04:29:08', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(340, '2016-11-14 04:29:08', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(341, '2016-11-14 04:29:08', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(342, '2016-11-14 04:29:29', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(343, '2016-11-14 04:29:29', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(344, '2016-11-14 04:29:29', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(345, '2016-11-14 04:29:47', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(346, '2016-11-14 04:29:47', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(347, '2016-11-14 04:29:47', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(348, '2016-11-14 04:31:30', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(349, '2016-11-14 04:31:30', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(350, '2016-11-14 04:31:30', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(351, '2016-11-14 04:31:32', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(352, '2016-11-14 04:31:32', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(353, '2016-11-14 04:31:32', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(354, '2016-11-14 04:32:01', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(355, '2016-11-14 04:32:02', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(356, '2016-11-14 04:32:02', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(357, '2016-11-14 04:32:02', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(358, '2016-11-14 04:32:04', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(359, '2016-11-14 04:32:04', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(360, '2016-11-14 04:32:04', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(361, '2016-11-14 04:32:04', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(362, '2016-11-14 04:32:06', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(363, '2016-11-14 04:32:06', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(364, '2016-11-14 04:32:06', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(365, '2016-11-14 04:32:06', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(366, '2016-11-14 04:32:08', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(367, '2016-11-14 04:32:08', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(368, '2016-11-14 04:32:08', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(369, '2016-11-14 04:32:08', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(370, '2016-11-14 04:32:57', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(371, '2016-11-14 04:32:57', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(372, '2016-11-14 04:32:57', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(373, '2016-11-14 04:32:59', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(374, '2016-11-14 04:32:59', 'Erro ao gerenciar threads, AgendamentoID: 16', 'BS'),
(375, '2016-11-14 04:33:01', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(376, '2016-11-14 04:33:01', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(377, '2016-11-14 04:33:01', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(378, '2016-11-14 04:33:17', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(379, '2016-11-14 04:33:17', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(380, '2016-11-14 04:33:17', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(381, '2016-11-14 04:33:38', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(382, '2016-11-14 04:33:38', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(383, '2016-11-14 04:33:38', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(384, '2016-11-14 04:33:38', 'Erro ao tentar executar o componente. ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(385, '2016-11-14 04:35:56', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(386, '2016-11-14 04:35:56', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(387, '2016-11-14 04:35:56', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(388, '2016-11-14 04:36:02', 'Erro ao gerenciar threads, AgendamentoID: 16', 'BS'),
(389, '2016-11-14 04:39:28', 'Executando ComponenteID:37 ComponenteNome:PubMED AgendamentoID: 16', 'BS'),
(390, '2016-11-14 04:39:28', 'Erro ao tentar executar o componente. ComponenteID:37 ComponenteNome:PubMED AgendamentoID: 16', 'BS'),
(391, '2016-11-14 04:40:12', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(392, '2016-11-14 04:40:12', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(393, '2016-11-14 04:40:12', 'Executando ComponenteID:37 ComponenteNome:PubMED AgendamentoID: 16', 'BS'),
(394, '2016-11-14 04:40:12', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 16', 'BS'),
(395, '2016-11-14 04:44:31', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 17', 'BS'),
(396, '2016-11-14 04:44:31', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 17', 'BS'),
(397, '2016-11-14 04:44:31', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 17', 'BS'),
(398, '2016-11-14 04:44:31', 'Executando ComponenteID:37 ComponenteNome:PubMED AgendamentoID: 17', 'BS'),
(399, '2016-11-14 04:48:23', 'Executando ComponenteID:36 ComponenteNome:ComponentePDB AgendamentoID: 17', 'BS'),
(400, '2016-11-14 04:48:23', 'Executando ComponenteID:34 ComponenteNome:ComponentePDB AgendamentoID: 17', 'BS'),
(401, '2016-11-14 04:48:23', 'Executando ComponenteID:35 ComponenteNome:ComponentePDB AgendamentoID: 17', 'BS'),
(402, '2016-11-14 04:48:23', 'Executando ComponenteID:37 ComponenteNome:PubMED AgendamentoID: 17', 'BS'),
(403, '2016-11-15 04:40:24', 'Erro ao gerenciar threads, AgendamentoID: 18', 'BS'),
(404, '2016-11-15 04:42:25', 'Erro ao gerenciar threads, AgendamentoID: 18', 'BS'),
(405, '2016-11-15 04:43:08', 'Erro ao gerenciar threads, AgendamentoID: 18', 'BS'),
(406, '2016-11-15 04:44:13', 'Erro ao gerenciar threads, AgendamentoID: 18', 'BS'),
(407, '2016-11-15 04:44:56', 'Erro ao gerenciar threads, AgendamentoID: 18', 'BS'),
(408, '2016-11-15 04:48:19', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(409, '2016-11-15 05:02:03', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(410, '2016-11-15 05:02:03', 'Erro ao tentar executar o componente. ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(411, '2016-11-15 05:02:58', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(412, '2016-11-15 05:03:28', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(413, '2016-11-15 05:06:33', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(414, '2016-11-15 05:08:43', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(415, '2016-11-15 05:11:38', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(416, '2016-11-15 05:12:15', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(417, '2016-11-15 05:12:49', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(418, '2016-11-15 05:13:12', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(419, '2016-11-15 05:17:19', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(420, '2016-11-15 05:18:05', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(421, '2016-11-15 05:19:10', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(422, '2016-11-15 05:21:16', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(423, '2016-11-15 05:22:31', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(424, '2016-11-15 05:27:34', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(425, '2016-11-15 05:32:44', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(426, '2016-11-15 05:33:50', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(427, '2016-11-15 05:42:38', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(428, '2016-11-15 05:45:34', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(429, '2016-11-15 05:54:19', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(430, '2016-11-15 05:55:54', 'Executando ComponenteID:39 ComponenteNome:Graficos AgendamentoID: 18', 'BS'),
(431, '2016-11-15 06:06:57', 'Executando ComponenteID:46 ComponenteNome:Graficos AgendamentoID: 20', 'BS'),
(432, '2016-11-15 06:06:57', 'Executando ComponenteID:43 ComponenteNome:ComponentePDB AgendamentoID: 19', 'BS'),
(433, '2016-11-15 06:06:57', 'Executando ComponenteID:45 ComponenteNome:ComponentePDB AgendamentoID: 19', 'BS'),
(434, '2016-11-15 06:06:57', 'Executando ComponenteID:44 ComponenteNome:ComponentePDB AgendamentoID: 19', 'BS'),
(435, '2016-11-15 06:10:05', 'Executando ComponenteID:46 ComponenteNome:Graficos AgendamentoID: 21', 'BS');

-- --------------------------------------------------------

--
-- Table structure for table `pdb_Arginase`
--

CREATE TABLE IF NOT EXISTS `pdb_Arginase` (
`id` int(10) unsigned NOT NULL,
  `estrutura` varchar(32) NOT NULL,
  `descricao` varchar(512) CHARACTER SET utf16 DEFAULT NULL,
  `link` varchar(512) DEFAULT NULL,
  `doi` varchar(128) DEFAULT NULL,
  `classificacao` varchar(512) DEFAULT NULL,
  `depositado` date DEFAULT NULL,
  `lancado` date DEFAULT NULL,
  `organismo` varchar(512) DEFAULT NULL,
  `expressao` varchar(512) DEFAULT NULL,
  `metodo` varchar(255) DEFAULT NULL,
  `resolucao` varchar(16) DEFAULT NULL,
  `artigoorigem` text,
  `artigoautores` text CHARACTER SET utf16,
  `macromolecula` text,
  `ligantes` text,
  `tabelaligantes` text
) ENGINE=InnoDB AUTO_INCREMENT=65 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `pdb_Arginase`
--

INSERT INTO `pdb_Arginase` (`id`, `estrutura`, `descricao`, `link`, `doi`, `classificacao`, `depositado`, `lancado`, `organismo`, `expressao`, `metodo`, `resolucao`, `artigoorigem`, `artigoautores`, `macromolecula`, `ligantes`, `tabelaligantes`) VALUES
(1, '1D3V', 'CRYSTAL STRUCTURE OF THE BINUCLEAR MANGANESE METALLOENZYME ARGINASE COMPLEXED WITH 2(S)-AMINO-6-BORONOHEXANOIC ACID, AN L-ARGININE ANALOG', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1D3V', '10.2210/pdb1d3v/pdb', 'HYDROLASE', '1999-10-01', '1999-11-17', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.7', 'Arginase-boronic acid complex highlights a physiological role in erectile function.', 'Cox, J.D.#Kim, N.N.#Traish, A.M.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" PROTEIN (ARGINASE) ",\n                                               " A,B ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', '2(S)-AMINO-6-BORONOHEXANOIC ACID,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ABH ",\n                                                " 2(S)-AMINO-6-BORONOHEXANOIC ACID ",\n                                                " C6 H15 B N O5 -1 ",\n                                                " BLVGFZFOWWBCCZ-YFKPBYRVSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(2, '1HQ5', 'CRYSTAL STRUCTURE OF THE BINUCLEAR MANGANESE METALLOENZYME ARGINASE COMPLEXED WITH S-(2-BORONOETHYL)-L-CYSTEINE, AN L-ARGININE ANALOGUE', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1HQ5', '10.2210/pdb1hq5/pdb', 'HYDROLASE', '2000-12-14', '2001-04-04', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.3', 'Probing erectile function: S-(2-boronoethyl)-L-cysteine binds to arginase as a transition state analogue and enhances smooth muscle relaxation in human penile corpus cavernosum.', 'Kim, N.N.#Cox, J.D.#Baggio, R.F.#Emig, F.A.#Mistry, S.K.#Harper, S.L.#Speicher, D.W.#Morris Jr., S.M.#Ash, D.E.#Traish, A.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE 1 ",\n                                               " A,B ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION,S-2-(BORONOETHYL)-L-CYSTEINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" S2C ",\n                                                " S-2-(BORONOETHYL)-L-CYSTEINE ",\n                                                " C5 H13 B N O5 S -1 ",\n                                                " XLVRIIJULVQAMP-BYPYZUCNSA-N "]\n                ]}\n            '),
(3, '1HQF', 'CRYSTAL STRUCTURE OF THE BINUCLEAR MANGANESE METALLOENZYME ARGINASE COMPLEXED WITH N-HYDROXY-L-ARGININE', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1HQF', '10.2210/pdb1hqf/pdb', 'HYDROLASE', '2000-12-16', '2001-04-04', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.9', 'Mechanistic and metabolic inferences from the binding of substrate analogues and products to arginase.', 'Cox, J.D.#Cama, E.#Colleluori, D.M.#Pethe, S.#Boucher, J.L.#Mansuy, D.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE 1 ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'N-OMEGA-HYDROXY-L-ARGININE,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" HAR ",\n                                                " N-OMEGA-HYDROXY-L-ARGININE ",\n                                                " C6 H14 N4 O3 ",\n                                                " FQWRAVYMZULPNK-BYPYZUCNSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(4, '1HQG', 'CRYSTAL STRUCTURE OF THE H141C ARGINASE VARIANT COMPLEXED WITH PRODUCTS ORNITHINE AND UREA', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1HQG', '10.2210/pdb1hqg/pdb', 'HYDROLASE', '2000-12-16', '2001-04-04', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.0', 'Mechanistic and metabolic inferences from the binding of substrate analogues and products to arginase.', 'Cox, J.D.#Cama, E.#Colleluori, D.M.#Pethe, S.#Boucher, J.L.#Mansuy, D.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE 1 ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'S,S-(2-HYDROXYETHYL)THIOCYSTEINE,MANGANESE (II) ION,L-ORNITHINE,UREA', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" CME ",\n                                                " S,S-(2-HYDROXYETHYL)THIOCYSTEINE ",\n                                                " C5 H11 N O3 S2 ",\n                                                " YPUBRSXDQSFQBA-BYPYZUCNSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" ORN ",\n                                                " L-ORNITHINE ",\n                                                " C5 H12 N2 O2 ",\n                                                " AHLPHDHHMVZTML-BYPYZUCNSA-N "]\n                , [" URE ",\n                                                " UREA ",\n                                                " C H4 N2 O ",\n                                                " XSQUKJJJFZCRTK-UHFFFAOYSA-N "]\n                ]}\n            '),
(5, '1HQH', 'CRYSTAL STRUCTURE OF THE BINUCLEAR MANGANESE METALLOENZYME ARGINASE COMPLEXED WITH NOR-N-HYDROXY-L-ARGININE', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1HQH', '10.2210/pdb1hqh/pdb', 'HYDROLASE', '2000-12-16', '2001-04-04', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.8', 'Mechanistic and metabolic inferences from the binding of substrate analogues and products to arginase.', 'Cox, J.D.#Cama, E.#Colleluori, D.M.#Pethe, S.#Boucher, J.L.#Mansuy, D.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE 1 ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION,NOR-N-OMEGA-HYDROXY-L-ARGININE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" NNH ",\n                                                " NOR-N-OMEGA-HYDROXY-L-ARGININE ",\n                                                " C5 H12 N4 O3 ",\n                                                " KOBHCUDVWOTEKO-VKHMYHEASA-N "]\n                ]}\n            '),
(6, '1HQX', 'R308K ARGINASE VARIANT', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1HQX', '10.2210/pdb1hqx/pdb', 'HYDROLASE', '2000-12-20', '2001-06-06', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '3.0', 'Subunit-subunit interactions in trimeric arginase. Generation of active monomers by mutation of a single amino acid.', 'Lavulo, L.T.#Sossong Jr., T.M.#Brigham-Burke, M.R.#Doyle, M.L.#Cox, J.D.#Christianson, D.W.#Ash, D.E.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(7, '1P8M', 'Structural and Functional Importance of First-Shell Metal Ligands in the Binuclear Manganese Cluster of Arginase I.', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1P8M', '10.2210/pdb1p8m/pdb', 'HYDROLASE', '2003-05-07', '2003-06-17', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.84', 'Structural and functional importance of first-shell metal ligands in the binuclear   manganese cluster of arginase I', 'Cama, E.#Emig, F.A.#Ash, E-D.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(8, '1P8N', 'Structural and Functional Importance of First-Shell Metal Ligands in the Binuclear Manganese Cluster of Arginase I.', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1P8N', '10.2210/pdb1p8n/pdb', 'HYDROLASE', '2003-05-07', '2003-06-17', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.9', 'Structural and functional importance of first-shell metal ligands in the binuclear   manganese cluster of arginase I', 'Cama, E.#Emig, F.A.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(9, '1P8O', 'Structural and Functional Importance of First-Shell Metal Ligands in the Binuclear Manganese Cluster of Arginase I.', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1P8O', '10.2210/pdb1p8o/pdb', 'HYDROLASE', '2003-05-07', '2003-06-17', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.96', 'Structural and functional importance of first-shell metal ligands in the binuclear   manganese cluster of arginase I', 'Cama, E.#Emig, F.A.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(10, '1P8P', 'Structural and Functional Importance of First-Shell Metal Ligands in the Binuclear Manganese Cluster of Arginase I.', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1P8P', '10.2210/pdb1p8p/pdb', 'HYDROLASE', '2003-05-07', '2003-06-17', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.5', 'Structural and functional importance of first-shell metal ligands in the binuclear   manganese cluster of arginase I', 'Cama, E.#Emig, F.A.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(11, '1P8Q', 'Structural and Functional Importance of First-Shell Metal Ligands in the Binuclear Cluster of Arginase I.', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1P8Q', '10.2210/pdb1p8q/pdb', 'HYDROLASE', '2003-05-07', '2003-06-17', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.95', 'Structural and functional importance of first-shell metal ligands in the binuclear   manganese cluster of arginase I', 'Cama, E.#Emig, F.A.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(12, '1P8R', 'Structural and Functional Importance of First-Shell Metal Ligands in the Binuclear Manganese Cluster of Arginase I.', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1P8R', '10.2210/pdb1p8r/pdb', 'HYDROLASE', '2003-05-07', '2003-06-17', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.5', 'Structural and functional importance of first-shell metal ligands in the binuclear   manganese cluster of arginase I', 'Cama, E.#Emig, F.A.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B ",\n                                               " 308 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'CHLORIDE ION,MANGANESE (II) ION,S-2-(BORONOETHYL)-L-CYSTEINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" CL ",\n                                                " CHLORIDE ION ",\n                                                " CL -1 ",\n                                                " VEXZGXHMUGYJMC-UHFFFAOYSA-M "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" S2C ",\n                                                " S-2-(BORONOETHYL)-L-CYSTEINE ",\n                                                " C5 H13 B N O5 S -1 ",\n                                                " XLVRIIJULVQAMP-BYPYZUCNSA-N "]\n                ]}\n            '),
(13, '1P8S', 'Structural and Functional Importance of First-Shell Metal Ligands in the Binuclear Manganese Cluster of Arginase I.', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1P8S', '10.2210/pdb1p8s/pdb', 'HYDROLASE', '2003-05-07', '2003-06-17', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '3.2', 'Structural and functional importance of first-shell metal ligands in the binuclear   manganese cluster of arginase I', 'Cama, E.#Emig, F.A.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(14, '1R1O', 'Amino Acid Sulfonamides as Transition-State Analogue Inhibitors of Arginase', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1R1O', '10.2210/pdb1r1o/pdb', 'hydrolase', '2003-09-24', '2003-10-28', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.8', 'Design of Amino Acid Sulfonamides as Transition-State Analogue Inhibitors of Arginase', 'Cama, E.#Shin, H.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION,S-[2-(AMINOSULFONYL)ETHYL]-D-CYSTEINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" SDC ",\n                                                " S-[2-(AMINOSULFONYL)ETHYL]-D-CYSTEINE ",\n                                                " C5 H12 N2 O4 S2 ",\n                                                " ZRCBQBJBRHIRRG-BYPYZUCNSA-N "]\n                ]}\n            '),
(15, '1RLA', 'THREE-DIMENSIONAL STRUCTURE OF RAT LIVER ARGINASE, THE BINUCLEAR MANGANESE METALLOENZYME OF THE UREA CYCLE', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1RLA', '10.2210/pdb1rla/pdb', 'HYDROLASE', '1996-08-15', '1997-10-15', 'Rattus norvegicus', 'null', 'X-RAY DIFFRACTION', '2.1', 'Structure of a unique binuclear manganese cluster in arginase.', 'Kanyo, Z.F.#Scolnick, L.R.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(16, '1T4P', 'Arginase-dehydro-ABH complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1T4P', '10.2210/pdb1t4p/pdb', 'HYDROLASE', '2004-04-30', '2005-04-12', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.6', 'Inhibitor coordination interactions in the binuclear manganese cluster of arginase', 'Cama, E.#Pethe, S.#Boucher, J.-L.#Han, S.#Emig, F.A.#Ash, D.E.#Viola, R.E.#Mansuy, D.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', '[(1E,5S)-5-AMINO-5-CARBOXYPENT-1-ENYL](TRIHYDROXY)BORATE(1-),MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 2BH ",\n                                                " [(1E,5S)-5-AMINO-5-CARBOXYPENT-1-ENYL](TRIHYDROXY)BORATE(1-) ",\n                                                " C6 H13 B N O5 -1 ",\n                                                " CCCYGXMAMSUXAV-FYTLMZHYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(17, '1T4R', 'arginase-descarboxy-nor-NOHA complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1T4R', '10.2210/pdb1t4r/pdb', 'HYDROLASE', '2004-04-30', '2005-04-12', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.6', 'Inhibitor coordination interactions in the binuclear manganese cluster of arginase', 'Cama, E.#Pethe, S.#Boucher, J.-L.#Han, S.#Emig, F.A.#Ash, D.E.#Viola, R.E.#Mansuy, D.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', '3-{[(E)-AMINO(HYDROXYIMINO)METHYL]AMINO}PROPAN-1-AMINIUM,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" AHI ",\n                                                " 3-{[(E)-AMINO(HYDROXYIMINO)METHYL]AMINO}PROPAN-1-AMINIUM ",\n                                                " C4 H13 N4 O 1 ",\n                                                " HYBLXYGPQAIGPY-UHFFFAOYSA-O "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(18, '1T4S', 'arginase-L-valine complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1T4S', '10.2210/pdb1t4s/pdb', 'HYDROLASE', '2004-04-30', '2004-10-12', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.8', 'Inhibitor coordination interactions in the binuclear manganese cluster of arginase', 'Cama, E.#Pethe, S.#Boucher, J.-L.#Han, S.#Emig, F.A.#Ash, D.E.#Viola, R.E.#Mansuy, D.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION,VALINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" VAL ",\n                                                " VALINE ",\n                                                " C5 H11 N O2 ",\n                                                " KZSNJWFQEVHDMF-BYPYZUCNSA-N "]\n                ]}\n            '),
(19, '1T4T', 'arginase-dinor-NOHA complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1T4T', '10.2210/pdb1t4t/pdb', 'HYDROLASE', '2004-04-30', '2005-04-12', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.2', 'Inhibitor coordination interactions in the binuclear manganese cluster of arginase', 'Cama, E.#Pethe, S.#Boucher, J.-L.#Han, S.#Emig, F.A.#Ash, D.E.#Viola, R.E.#Mansuy, D.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', '3-{[(E)-AMINO(HYDROXYIMINO)METHYL]AMINO}ALANINE,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" DIR ",\n                                                " 3-{[(E)-AMINO(HYDROXYIMINO)METHYL]AMINO}ALANINE ",\n                                                " C4 H10 N4 O3 ",\n                                                " RPHCSGPGZUWMRV-REOHCLBHSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(20, '1T5F', 'arginase I-AOH complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1T5F', '10.2210/pdb1t5f/pdb', 'HYDROLASE', '2004-05-04', '2005-05-17', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.2', 'Design of amino acid aldehydes as transition-state analogue inhibitors of arginase', 'Shin, H.#Cama, E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', '(S)-2-AMINO-7,7-DIHYDROXYHEPTANOIC ACID,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" DHH ",\n                                                " (S)-2-AMINO-7,7-DIHYDROXYHEPTANOIC ACID ",\n                                                " C7 H15 N O4 ",\n                                                " KSOGSUKZGWOZJN-YFKPBYRVSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(21, '1T5G', 'Arginase-F2-L-Arginine complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1T5G', '10.2210/pdb1t5g/pdb', 'HYDROLASE', '2004-05-04', '2004-10-12', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.4', 'Inhibitor coordination interactions in the binuclear manganese cluster of arginase', 'Cama, E.#Pethe, S.#Boucher, J.-L.#Han, S.#Emig, F.A.#Ash, D.E.#Viola, R.E.#Mansuy, D.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'ARGININE,FLUORIDE ION,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ARG ",\n                                                " ARGININE ",\n                                                " C6 H15 N4 O2 1 ",\n                                                " ODKSFYDXXFIFQN-BYPYZUCNSA-O "]\n                , [" F ",\n                                                " FLUORIDE ION ",\n                                                " F -1 ",\n                                                " KRHYYFGTRYWZRS-UHFFFAOYSA-M "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(22, '1TA1', 'H141C mutant of rat liver arginase I', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1TA1', '10.2210/pdb1ta1/pdb', 'HYDROLASE', '2004-05-19', '2005-08-16', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.5', 'Probing the role of the hyper-reactive histidine residue of arginase.', 'Colleluori, D.M.#Reczkowski, R.S.#Emig, F.A.#Cama, E.#Cox, J.D.#Scolnick, L.R.#Compher, K.#Jude, K.#Han, S.#Viola, R.E.#Christianson, D.W.#Ash, D.E.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(23, '1TBH', 'H141D mutant of rat liver arginase I', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1TBH', '10.2210/pdb1tbh/pdb', 'HYDROLASE', '2004-05-20', '2005-08-16', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.7', 'Probing the role of the hyper-reactive histidine residue of arginase.', 'Colleluori, D.M.#Reczkowski, R.S.#Emig, F.A.#Cama, E.#Cox, J.D.#Scolnick, L.R.#Compher, K.#Jude, K.#Han, S.#Viola, R.E.#Christianson, D.W.#Ash, D.E.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(24, '1TBJ', 'H141A mutant of rat liver arginase I', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1TBJ', '10.2210/pdb1tbj/pdb', 'HYDROLASE', '2004-05-20', '2005-08-16', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.8', 'Probing the role of the hyper-reactive histidine residue of arginase.', 'Colleluori, D.M.#Reczkowski, R.S.#Emig, F.A.#Cama, E.#Cox, J.D.#Scolnick, L.R.#Compher, K.#Jude, K.#Han, S.#Viola, R.E.#Christianson, D.W.#Ash, D.E.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(25, '1TBL', 'H141N mutant of rat liver arginase I', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1TBL', '10.2210/pdb1tbl/pdb', 'HYDROLASE', '2004-05-20', '2005-08-16', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '3.1', 'Probing the role of the hyper-reactive histidine residue of arginase.', 'Colleluori, D.M.#Reczkowski, R.S.#Emig, F.A.#Cama, E.#Cox, J.D.#Scolnick, L.R.#Compher, K.#Jude, K.#Han, S.#Viola, R.E.#Christianson, D.W.#Ash, D.E.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(26, '1ZPE', 'Arginase I covalently modified with butylamine at Q19C', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1ZPE', '10.2210/pdb1zpe/pdb', 'HYDROLASE', '2005-05-16', '2005-12-06', 'Rattus norvegicus', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '1.7', 'Probing the role of the hyper-reactive histidine residue of arginase.', 'Colleluori, D.M.#Reczkowski, R.S.#Emig, F.A.#Cama, E.#Cox, J.D.#Scolnick, L.R.#Compher, K.#Jude, K.#Han, S.#Viola, R.E.#Christianson, D.W.#Ash, D.E.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', '3-[(4-AMINOBUTYL)SULFINYL]-2-IMINOPROPAN-1-OL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" BBC ",\n                                                " 3-[(4-AMINOBUTYL)SULFINYL]-2-IMINOPROPAN-1-OL ",\n                                                " C7 H16 N2 O2 S ",\n                                                " VUEASHHOUFPCLO-CRALRDPISA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(27, '1ZPG', 'Arginase I covalently modified with propylamine at Q19C', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1ZPG', '10.2210/pdb1zpg/pdb', 'HYDROLASE', '2005-05-16', '2005-12-06', 'Rattus norvegicus', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '1.9', 'Probing the role of the hyper-reactive histidine residue of arginase.', 'Colleluori, D.M.#Reczkowski, R.S.#Emig, F.A.#Cama, E.#Cox, J.D.#Scolnick, L.R.#Compher, K.#Jude, K.#Han, S.#Viola, R.E.#Christianson, D.W.#Ash, D.E.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase 1 ",\n                                               " A,B,C ",\n                                               " 314 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', '(2S)-2-AMINO-3-[(3-AMINOPROPYL)SULFANYL]PROPAN-1-OL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" BPE ",\n                                                " (2S)-2-AMINO-3-[(3-AMINOPROPYL)SULFANYL]PROPAN-1-OL ",\n                                                " C6 H16 N2 O S ",\n                                                " RYOVYWMBACBGOD-ZCFIWIBFSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(28, '2EF4', 'Crystal structure of the arginase from thermus thermophilus', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2EF4', '10.2210/pdb2ef4/pdb', 'HYDROLASE', '2007-02-20', '2007-04-14', 'Thermus thermophilus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.3', 'Crystal structure of the arginase from thermus thermophilus', 'Kumarevel, T.S.#Karthe, P.#Kuramitsu, S.#Yokoyama, S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 290 ",\n                                               " Thermus thermophilus ",\n                                               "Gene Name(s): TTHA1496 "]\n                ]}\n            ', '', NULL),
(29, '2EF5', 'Crystal structure of the arginase from thermus thermophilus', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2EF5', '10.2210/pdb2ef5/pdb', 'HYDROLASE', '2007-02-20', '2007-04-14', 'Thermus thermophilus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.0', 'Crystal structure of the arginase from thermus thermophilus', 'Kumarevel, T.S.#Karthe, P.#Kuramitsu, S.#Yokoyama, S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,D,E,F,G ",\n                                               " 290 ",\n                                               " Thermus thermophilus ",\n                                               "Gene Name(s): TTHA1496 "]\n                ]}\n            ', 'GUANIDINE,LYSINE,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GAI ",\n                                                " GUANIDINE ",\n                                                " C H5 N3 ",\n                                                " ZRALSGWEFCBTJO-UHFFFAOYSA-N "]\n                , [" LYS ",\n                                                " LYSINE ",\n                                                " C6 H15 N2 O2 1 ",\n                                                " KDXKERNSBIXSRK-YFKPBYRVSA-O "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(30, '2EIV', 'Crystal Structure of the arginase from Thermus thermophilus', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2EIV', '10.2210/pdb2eiv/pdb', 'HYDROLASE', '2007-03-13', '2007-09-18', 'Thermus thermophilus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.91', 'Crystal Structure of the arginase from Thermus thermophilus', 'Kumarevel, T.S.#Karthe, P.#Kuramitsu, S.#Yokoyama, S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,C,D,E,F,G,H,I,J,K,L,M ",\n                                               " 291 ",\n                                               " Thermus thermophilus ",\n                                               "Gene Name(s): TTHA1496 "]\n                ]}\n            ', '', NULL);
INSERT INTO `pdb_Arginase` (`id`, `estrutura`, `descricao`, `link`, `doi`, `classificacao`, `depositado`, `lancado`, `organismo`, `expressao`, `metodo`, `resolucao`, `artigoorigem`, `artigoautores`, `macromolecula`, `ligantes`, `tabelaligantes`) VALUES
(31, '2RLA', 'ALTERING THE BINUCLEAR MANGANESE CLUSTER OF ARGINASE DIMINISHES THERMOSTABILITY AND CATALYTIC FUNCTION', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2RLA', '10.2210/pdb2rla/pdb', 'HYDROLASE', '1997-05-07', '1998-05-13', 'Rattus norvegicus', 'null', 'X-RAY DIFFRACTION', '3.0', 'Altering the binuclear manganese cluster of arginase diminishes thermostability and catalytic function.', 'Scolnick, L.R.#Kanyo, Z.F.#Cavalli, R.C.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(32, '3E8Q', 'X-ray structure of rat arginase I-T135A: the unliganded complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3E8Q', '10.2210/pdb3e8q/pdb', 'HYDROLASE', '2008-08-20', '2008-12-02', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.9', 'Probing the specificity determinants of amino acid recognition by arginase.', 'Shishova, E.Y.#Di Costanzo, L.#Emig, F.A.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase-1 ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(33, '3E8Z', 'X-ray structure of rat arginase I-N130A mutant: the unliganded complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3E8Z', '10.2210/pdb3e8z/pdb', 'HYDROLASE', '2008-08-20', '2008-12-02', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.0', 'Probing the specificity determinants of amino acid recognition by arginase.', 'Shishova, E.Y.#Di Costanzo, L.#Emig, F.A.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase-1 ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(34, '3E9B', 'X-ray structure of rat arginase I-T135A mutant: the complex with BEC', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3E9B', '10.2210/pdb3e9b/pdb', 'HYDROLASE', '2008-08-21', '2008-12-02', 'Rattus norvegicus', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.15', 'Probing the specificity determinants of amino acid recognition by arginase.', 'Shishova, E.Y.#Di Costanzo, L.#Emig, F.A.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase-1 ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION,S-2-(BORONOETHYL)-L-CYSTEINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" S2C ",\n                                                " S-2-(BORONOETHYL)-L-CYSTEINE ",\n                                                " C5 H13 B N O5 S -1 ",\n                                                " XLVRIIJULVQAMP-BYPYZUCNSA-N "]\n                ]}\n            '),
(35, '3MMR', 'Structure of Plasmodium falciparum Arginase in complex with ABH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3MMR', '10.2210/pdb3mmr/pdb', 'HYDROLASE', '2010-04-20', '2010-06-23', 'Plasmodium falciparum', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.14', 'Crystal structure of arginase from Plasmodium falciparum and implications for L-arginine depletion in malarial infection .', 'Dowling, D.P.#Ilies, M.#Olszewski, K.L.#Portugal, S.#Mota, M.M.#Llinas, M.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 413 ",\n                                               " Plasmodium falciparum ",\n                                               "Gene Name(s): PFI0320w "]\n                ]}\n            ', '2(S)-AMINO-6-BORONOHEXANOIC ACID,BETA-MERCAPTOETHANOL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ABH ",\n                                                " 2(S)-AMINO-6-BORONOHEXANOIC ACID ",\n                                                " C6 H15 B N O5 -1 ",\n                                                " BLVGFZFOWWBCCZ-YFKPBYRVSA-N "]\n                , [" BME ",\n                                                " BETA-MERCAPTOETHANOL ",\n                                                " C2 H6 O S ",\n                                                " DGVVWUTYPXICAM-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(36, '3RLA', 'ALTERING THE BINUCLEAR MANGANESE CLUSTER OF ARGINASE DIMINISHES THERMOSTABILITY AND CATALYTIC FUNCTION', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3RLA', '10.2210/pdb3rla/pdb', 'HYDROLASE', '1997-05-07', '1998-05-13', 'Rattus norvegicus', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.54', 'Altering the binuclear manganese cluster of arginase diminishes thermostability and catalytic function.', 'Scolnick, L.R.#Kanyo, Z.F.#Cavalli, R.C.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(37, '3SL0', 'Crystal Structure of P. falciparum arginase complexed with 2-amino-6-borono-2-(difluoromethyl)hexanoic acid', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3SL0', '10.2210/pdb3sl0/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2011-06-23', '2011-07-20', 'Plasmodium falciparum', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.0', 'Binding of alpha,alpha-Disubstituted Amino Acids to Arginase Suggests New Avenues for Inhibitor Design', 'Ilies, M.#Di Costanzo, L.#Dowling, D.P.#Thorn, K.J.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 413 ",\n                                               " Plasmodium falciparum ",\n                                               "Gene Name(s): PFI0320w "]\n                ]}\n            ', '2-(DIFLUOROMETHYL)-6-(DIHYDROXYBORANYL)-L-NORLEUCINE,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" FB5 ",\n                                                " 2-(DIFLUOROMETHYL)-6-(DIHYDROXYBORANYL)-L-NORLEUCINE ",\n                                                " C7 H14 B F2 N O4 ",\n                                                " QIYCLWKKKPVZJC-SSDOTTSWSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(38, '3SL1', 'Crystal Structure of P. falciparum arginase complexed with 2-amino-6-borono-2-methylhexanoic acid', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3SL1', '10.2210/pdb3sl1/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2011-06-23', '2011-07-20', 'Plasmodium falciparum', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.9', 'Binding of alpha,alpha-Disubstituted Amino Acids to Arginase Suggests New Avenues for Inhibitor Design', 'Ilies, M.#Di Costanzo, L.#Dowling, D.P.#Thorn, K.J.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 413 ",\n                                               " Plasmodium falciparum ",\n                                               "Gene Name(s): PFI0320w "]\n                ]}\n            ', '6-(DIHYDROXYBORANYL)-2-METHYL-L-NORLEUCINE,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" FB6 ",\n                                                " 6-(DIHYDROXYBORANYL)-2-METHYL-L-NORLEUCINE ",\n                                                " C7 H16 B N O4 ",\n                                                " VJCNFQNUKLFMMR-ZETCQYMHSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(39, '4ITY', 'Crystal structure of Leishmania mexicana arginase', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4ITY', '10.2210/pdb4ity/pdb', 'HYDROLASE', '2013-01-19', '2013-01-30', 'Leishmania mexicana', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.8', 'Crystal structure of arginase from Leishmania mexicana and implications for the inhibition of polyamine biosynthesis in parasitic infections.', 'D''Antonio, E.L.#Ullman, B.#Roberts, S.C.#Dixit, U.G.#Wilson, M.E.#Hai, Y.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 330 ",\n                                               " Leishmania mexicana ",\n                                               "Gene Name(s): ARG "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(40, '4IU0', 'Crystal structure of Leishmania mexicana arginase in complex with inhibitor ABH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4IU0', '10.2210/pdb4iu0/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2013-01-19', '2013-01-30', 'Leishmania mexicana', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.77', 'Crystal structure of arginase from Leishmania mexicana and implications for the inhibition of polyamine biosynthesis in parasitic infections.', 'D''Antonio, E.L.#Ullman, B.#Roberts, S.C.#Dixit, U.G.#Wilson, M.E.#Hai, Y.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 330 ",\n                                               " Leishmania mexicana ",\n                                               "Gene Name(s): ARG "]\n                ]}\n            ', '2(S)-AMINO-6-BORONOHEXANOIC ACID,GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ABH ",\n                                                " 2(S)-AMINO-6-BORONOHEXANOIC ACID ",\n                                                " C6 H15 B N O5 -1 ",\n                                                " BLVGFZFOWWBCCZ-YFKPBYRVSA-N "]\n                , [" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(41, '4IU1', 'Crystal structure of Leishmania mexicana arginase in complex with inhibitor nor-NOHA', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4IU1', '10.2210/pdb4iu1/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2013-01-19', '2013-01-30', 'Leishmania mexicana', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.95', 'Crystal structure of arginase from Leishmania mexicana and implications for the inhibition of polyamine biosynthesis in parasitic infections.', 'D''Antonio, E.L.#Ullman, B.#Roberts, S.C.#Dixit, U.G.#Wilson, M.E.#Hai, Y.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 330 ",\n                                               " Leishmania mexicana ",\n                                               "Gene Name(s): ARG "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION,NOR-N-OMEGA-HYDROXY-L-ARGININE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" NNH ",\n                                                " NOR-N-OMEGA-HYDROXY-L-ARGININE ",\n                                                " C5 H12 N4 O3 ",\n                                                " KOBHCUDVWOTEKO-VKHMYHEASA-N "]\n                ]}\n            '),
(42, '4IU4', 'Crystal structure of Leishmania mexicana arginase in complex with inhibitor BEC', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4IU4', '10.2210/pdb4iu4/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2013-01-19', '2013-01-30', 'Leishmania mexicana', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.8', 'Crystal structure of arginase from Leishmania mexicana and implications for the inhibition of polyamine biosynthesis in parasitic infections.', 'D''Antonio, E.L.#Ullman, B.#Roberts, S.C.#Dixit, U.G.#Wilson, M.E.#Hai, Y.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 330 ",\n                                               " Leishmania mexicana ",\n                                               "Gene Name(s): ARG "]\n                ]}\n            ', 'BETA-MERCAPTOETHANOL,GLYCEROL,MANGANESE (II) ION,S-2-(BORONOETHYL)-L-CYSTEINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" BME ",\n                                                " BETA-MERCAPTOETHANOL ",\n                                                " C2 H6 O S ",\n                                                " DGVVWUTYPXICAM-UHFFFAOYSA-N "]\n                , [" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" S2C ",\n                                                " S-2-(BORONOETHYL)-L-CYSTEINE ",\n                                                " C5 H13 B N O5 S -1 ",\n                                                " XLVRIIJULVQAMP-BYPYZUCNSA-N "]\n                ]}\n            '),
(43, '4IU5', 'Crystal structure of Leishmania mexicana arginase in complex with catalytic product L-ornithine', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4IU5', '10.2210/pdb4iu5/pdb', 'HYDROLASE', '2013-01-19', '2013-01-30', 'Leishmania mexicana', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.95', 'Crystal structure of arginase from Leishmania mexicana and implications for the inhibition of polyamine biosynthesis in parasitic infections.', 'D''Antonio, E.L.#Ullman, B.#Roberts, S.C.#Dixit, U.G.#Wilson, M.E.#Hai, Y.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 330 ",\n                                               " Leishmania mexicana ",\n                                               "Gene Name(s): ARG "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION,L-ORNITHINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" ORN ",\n                                                " L-ORNITHINE ",\n                                                " C5 H12 N2 O2 ",\n                                                " AHLPHDHHMVZTML-BYPYZUCNSA-N "]\n                ]}\n            '),
(44, '4Q3P', 'Crystal structure of Schistosoma mansoni arginase', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q3P', '10.2210/pdb4q3p/pdb', 'HYDROLASE', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.5', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(45, '4Q3Q', 'Crystal structure of Schistosoma mansoni arginase in complex with inhibitor ABH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q3Q', '10.2210/pdb4q3q/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.0', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', '2(S)-AMINO-6-BORONOHEXANOIC ACID,GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ABH ",\n                                                " 2(S)-AMINO-6-BORONOHEXANOIC ACID ",\n                                                " C6 H15 B N O5 -1 ",\n                                                " BLVGFZFOWWBCCZ-YFKPBYRVSA-N "]\n                , [" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(46, '4Q3R', 'Crystal structure of Schistosoma mansoni arginase in complex with inhibitor ABHDP', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q3R', '10.2210/pdb4q3r/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.17', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'CHLORIDE ION,CESIUM ION,S-HYDROXYCYSTEINE,GLYCEROL,MANGANESE (II) ION,TRIETHYLENE GLYCOL,(R)-2-AMINO-6-BORONO-2-(1-(3,4-DICHLOROBENZYL)PIPERIDIN-4-YL)HEXANOIC ACID', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" CL ",\n                                                " CHLORIDE ION ",\n                                                " CL -1 ",\n                                                " VEXZGXHMUGYJMC-UHFFFAOYSA-M "]\n                , [" CS ",\n                                                " CESIUM ION ",\n                                                " CS 1 ",\n                                                " NCMHKCKGHRPLCM-UHFFFAOYSA-N "]\n                , [" CSO ",\n                                                " S-HYDROXYCYSTEINE ",\n                                                " C3 H7 N O3 S ",\n                                                " FXIRVRPOOYSARH-REOHCLBHSA-N "]\n                , [" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" PGE ",\n                                                " TRIETHYLENE GLYCOL ",\n                                                " C6 H14 O4 ",\n                                                " ZIBGPFATKBEMQZ-UHFFFAOYSA-N "]\n                , [" XA2 ",\n                                                " (R)-2-AMINO-6-BORONO-2-(1-(3,4-DICHLOROBENZYL)PIPERIDIN-4-YL)HEXANOIC ACID ",\n                                                " C18 H28 B CL2 N2 O5 -1 ",\n                                                " LYUGMNLZENMUTE-GOSISDBHSA-N "]\n                ]}\n            '),
(47, '4Q3R', 'Crystal structure of Schistosoma mansoni arginase in complex with inhibitor ABHDP', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q3R', '10.2210/pdb4q3r/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.17', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'CHLORIDE ION,CESIUM ION,S-HYDROXYCYSTEINE,GLYCEROL,MANGANESE (II) ION,TRIETHYLENE GLYCOL,(R)-2-AMINO-6-BORONO-2-(1-(3,4-DICHLOROBENZYL)PIPERIDIN-4-YL)HEXANOIC ACID', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" CL ",\n                                                " CHLORIDE ION ",\n                                                " CL -1 ",\n                                                " VEXZGXHMUGYJMC-UHFFFAOYSA-M "]\n                , [" CS ",\n                                                " CESIUM ION ",\n                                                " CS 1 ",\n                                                " NCMHKCKGHRPLCM-UHFFFAOYSA-N "]\n                , [" CSO ",\n                                                " S-HYDROXYCYSTEINE ",\n                                                " C3 H7 N O3 S ",\n                                                " FXIRVRPOOYSARH-REOHCLBHSA-N "]\n                , [" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" PGE ",\n                                                " TRIETHYLENE GLYCOL ",\n                                                " C6 H14 O4 ",\n                                                " ZIBGPFATKBEMQZ-UHFFFAOYSA-N "]\n                , [" XA2 ",\n                                                " (R)-2-AMINO-6-BORONO-2-(1-(3,4-DICHLOROBENZYL)PIPERIDIN-4-YL)HEXANOIC ACID ",\n                                                " C18 H28 B CL2 N2 O5 -1 ",\n                                                " LYUGMNLZENMUTE-GOSISDBHSA-N "]\n                ]}\n            '),
(48, '4Q3S', 'Crystal structure of Schistosoma mansoni arginase in complex with inhibitor ABHPE', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q3S', '10.2210/pdb4q3s/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.11', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION,[(5R)-5-AMINO-5-CARBOXY-7-(PIPERIDIN-1-YL)HEPTYL](TRIHYDROXY)BORATE(1-)', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" X7A ",\n                                                " [(5R)-5-AMINO-5-CARBOXY-7-(PIPERIDIN-1-YL)HEPTYL](TRIHYDROXY)BORATE(1-) ",\n                                                " C13 H28 B N2 O5 -1 ",\n                                                " SCRPJEVETAWLOZ-CYBMUJFWSA-N "]\n                ]}\n            '),
(49, '4Q3T', 'Crystal structure of Schistosoma mansoni arginase in complex with inhibitor NOHA', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q3T', '10.2210/pdb4q3t/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.14', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'GLYCEROL,N-OMEGA-HYDROXY-L-ARGININE,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" HAR ",\n                                                " N-OMEGA-HYDROXY-L-ARGININE ",\n                                                " C6 H14 N4 O3 ",\n                                                " FQWRAVYMZULPNK-BYPYZUCNSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(50, '4Q3U', 'Crystal structure of Schistosoma mansoni arginase in complex with inhibitor nor-NOHA', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q3U', '10.2210/pdb4q3u/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.5', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION,NOR-N-OMEGA-HYDROXY-L-ARGININE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" NNH ",\n                                                " NOR-N-OMEGA-HYDROXY-L-ARGININE ",\n                                                " C5 H12 N4 O3 ",\n                                                " KOBHCUDVWOTEKO-VKHMYHEASA-N "]\n                ]}\n            '),
(51, '4Q3V', 'Crystal structure of Schistosoma mansoni arginase in complex with inhibitor BEC', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q3V', '10.2210/pdb4q3v/pdb', 'HYDROLASE/HYDROLASE INHIBITOR', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.7', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION,S-2-(BORONOETHYL)-L-CYSTEINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" S2C ",\n                                                " S-2-(BORONOETHYL)-L-CYSTEINE ",\n                                                " C5 H13 B N O5 S -1 ",\n                                                " XLVRIIJULVQAMP-BYPYZUCNSA-N "]\n                ]}\n            '),
(52, '4Q40', 'Crystal structure of Schistosoma mansoni arginase in complex with L-valine', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q40', '10.2210/pdb4q40/pdb', 'HYDROLASE', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.83', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'GLYCEROL,IMIDAZOLE,MANGANESE (II) ION,VALINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" IMD ",\n                                                " IMIDAZOLE ",\n                                                " C3 H5 N2 1 ",\n                                                " RAXXELZNTBOGNW-UHFFFAOYSA-O "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" VAL ",\n                                                " VALINE ",\n                                                " C5 H11 N O2 ",\n                                                " KZSNJWFQEVHDMF-BYPYZUCNSA-N "]\n                ]}\n            '),
(53, '4Q41', 'Crystal structure of Schistosoma mansoni arginase in complex with L-lysine', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q41', '10.2210/pdb4q41/pdb', 'HYDROLASE', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.2', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'GLYCEROL,LYSINE,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" LYS ",\n                                                " LYSINE ",\n                                                " C6 H15 N2 O2 1 ",\n                                                " KDXKERNSBIXSRK-YFKPBYRVSA-O "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(54, '4Q42', 'Crystal structure of Schistosoma mansoni arginase in complex with L-ornithine', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4Q42', '10.2210/pdb4q42/pdb', 'HYDROLASE', '2014-04-12', '2014-07-16', 'Schistosoma mansoni', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.05', 'Crystal Structure of Schistosoma mansoni Arginase, a Potential Drug Target for the Treatment of Schistosomiasis.', 'Hai, Y.#Edwards, J.E.#Van Zandt, M.C.#Hoffmann, K.F.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D ",\n                                               " 385 ",\n                                               " Schistosoma mansoni ",\n                                               "Gene Name(s): Smp_059980 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION,L-ORNITHINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" ORN ",\n                                                " L-ORNITHINE ",\n                                                " C5 H12 N2 O2 ",\n                                                " AHLPHDHHMVZTML-BYPYZUCNSA-N "]\n                ]}\n            '),
(55, '4RHI', 'Crystal structure of SeMet-labeled wild-type T. brucei arginase-like protein in P321 space group', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4RHI', '10.2210/pdb4rhi/pdb', 'UNKNOWN FUNCTION', '2014-10-02', '2014-12-31', 'Trypanosoma brucei', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.55', 'Crystal Structure of an Arginase-like Protein from Trypanosoma brucei That Evolved without a Binuclear Manganese Cluster.', 'Hai, Y.#Kerkhoven, E.J.#Barrett, M.P.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B ",\n                                               " 351 ",\n                                               " Trypanosoma brucei ",\n                                               "Gene Name(s): Tb927.8.2020 "]\n                ]}\n            ', 'GLYCEROL,SELENOMETHIONINE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MSE ",\n                                                " SELENOMETHIONINE ",\n                                                " C5 H11 N O2 SE ",\n                                                " RJFAYQIBOAGBLC-BYPYZUCNSA-N "]\n                ]}\n            ');
INSERT INTO `pdb_Arginase` (`id`, `estrutura`, `descricao`, `link`, `doi`, `classificacao`, `depositado`, `lancado`, `organismo`, `expressao`, `metodo`, `resolucao`, `artigoorigem`, `artigoautores`, `macromolecula`, `ligantes`, `tabelaligantes`) VALUES
(56, '4RHJ', 'Crystal structure of wild-type T. brucei arginase-like protein in a reduced form', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4RHJ', '10.2210/pdb4rhj/pdb', 'UNKNOWN FUNCTION', '2014-10-02', '2014-12-31', 'Trypanosoma brucei', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.8', 'Crystal Structure of an Arginase-like Protein from Trypanosoma brucei That Evolved without a Binuclear Manganese Cluster.', 'Hai, Y.#Kerkhoven, E.J.#Barrett, M.P.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C,D,E,F ",\n                                               " 351 ",\n                                               " Trypanosoma brucei ",\n                                               "Gene Name(s): Tb927.8.2020 "]\n                ]}\n            ', '1,2-ETHANEDIOL', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" EDO ",\n                                                " 1,2-ETHANEDIOL ",\n                                                " C2 H6 O2 ",\n                                                " LYCAIKOWRPUZTN-UHFFFAOYSA-N "]\n                ]}\n            '),
(57, '4RHK', 'Crystal structure of T. brucei arginase-like protein in an oxidized form', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4RHK', '10.2210/pdb4rhk/pdb', 'Unknown function', '2014-10-02', '2014-12-31', 'Trypanosoma brucei', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.38', 'Crystal Structure of an Arginase-like Protein from Trypanosoma brucei That Evolved without a Binuclear Manganese Cluster.', 'Hai, Y.#Kerkhoven, E.J.#Barrett, M.P.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 351 ",\n                                               " Trypanosoma brucei ",\n                                               "Gene Name(s): Tb927.8.2020 "]\n                ]}\n            ', 'GLYCEROL', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                ]}\n            '),
(58, '4RHL', 'Crystal structure of T. brucei arginase-like protein triple mutant S149D/R151H/S226D bound with Mn2+', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4RHL', '10.2210/pdb4rhl/pdb', 'UNKNOWN FUNCTION', '2014-10-02', '2014-12-31', 'Trypanosoma brucei', 'Escherichia coli', 'X-RAY DIFFRACTION', '3.1', 'Crystal Structure of an Arginase-like Protein from Trypanosoma brucei That Evolved without a Binuclear Manganese Cluster.', 'Hai, Y.#Kerkhoven, E.J.#Barrett, M.P.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B ",\n                                               " 351 ",\n                                               " Trypanosoma brucei ",\n                                               "Gene Name(s): Tb927.8.2020 "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(59, '4RHM', 'Crystal structure of T. brucei arginase-like protein quadruple mutant S149D/R151H/S153D/S226D', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4RHM', '10.2210/pdb4rhm/pdb', 'UNKNOWN FUNCTION', '2014-10-02', '2014-12-31', 'Trypanosoma brucei', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.95', 'Crystal Structure of an Arginase-like Protein from Trypanosoma brucei That Evolved without a Binuclear Manganese Cluster.', 'Hai, Y.#Kerkhoven, E.J.#Barrett, M.P.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C ",\n                                               " 351 ",\n                                               " Trypanosoma brucei ",\n                                               "Gene Name(s): Tb927.8.2020 "]\n                ]}\n            ', '1,2-ETHANEDIOL,GLYCEROL,MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" EDO ",\n                                                " 1,2-ETHANEDIOL ",\n                                                " C2 H6 O2 ",\n                                                " LYCAIKOWRPUZTN-UHFFFAOYSA-N "]\n                , [" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(60, '4RHQ', 'Crystal structure of T. brucei arginase-like protein double mutant S149D/S153D', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4RHQ', '10.2210/pdb4rhq/pdb', 'Unknown function', '2014-10-02', '2014-12-31', 'Trypanosoma brucei', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.18', 'Crystal Structure of an Arginase-like Protein from Trypanosoma brucei That Evolved without a Binuclear Manganese Cluster.', 'Hai, Y.#Kerkhoven, E.J.#Barrett, M.P.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A,B,C ",\n                                               " 351 ",\n                                               " Trypanosoma brucei ",\n                                               "Gene Name(s): Tb927.8.2020 "]\n                ]}\n            ', '1,2-ETHANEDIOL,GLYCEROL', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" EDO ",\n                                                " 1,2-ETHANEDIOL ",\n                                                " C2 H6 O2 ",\n                                                " LYCAIKOWRPUZTN-UHFFFAOYSA-N "]\n                , [" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                ]}\n            '),
(61, '4RLA', 'ALTERING THE BINUCLEAR MANGANESE CLUSTER OF ARGINASE DIMINISHES THERMOSTABILITY AND CATALYTIC FUNCTION', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4RLA', '10.2210/pdb4rla/pdb', 'HYDROLASE', '1997-05-07', '1998-05-13', 'Rattus norvegicus', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.94', 'Altering the binuclear manganese cluster of arginase diminishes thermostability and catalytic function.', 'Scolnick, L.R.#Kanyo, Z.F.#Cavalli, R.C.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            '),
(62, '5HJ9', 'Crystal structure of Leishmania mexicana arginase in complex with inhibitor ABHPE', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=5HJ9', '10.2210/pdb5hj9/pdb', 'HYDROLASE', '2016-01-13', '2016-04-13', 'Leishmania mexicana', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.28', 'Crystal structures of Leishmania mexicana arginase complexed with alpha,alpha-disubstituted boronic amino-acid inhibitors', 'Hai, Y.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 330 ",\n                                               " Leishmania mexicana ",\n                                               "Gene Name(s): ARG "]\n                ]}\n            ', '1,2-ETHANEDIOL,GLYCEROL,MANGANESE (II) ION,TRIETHYLENE GLYCOL,SULFATE ION,[(5R)-5-amino-5-carboxy-7-(piperidin-1-yl)heptyl](trihydroxy)borate(1-)', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" EDO ",\n                                                " 1,2-ETHANEDIOL ",\n                                                " C2 H6 O2 ",\n                                                " LYCAIKOWRPUZTN-UHFFFAOYSA-N "]\n                , [" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " Mn 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" PGE ",\n                                                " TRIETHYLENE GLYCOL ",\n                                                " C6 H14 O4 ",\n                                                " ZIBGPFATKBEMQZ-UHFFFAOYSA-N "]\n                , [" SO4 ",\n                                                " SULFATE ION ",\n                                                " O4 S -2 ",\n                                                " QAOWNCQODCNURD-UHFFFAOYSA-L "]\n                , [" X7A ",\n                                                " [(5R)-5-amino-5-carboxy-7-(piperidin-1-yl)heptyl](trihydroxy)borate(1-) ",\n                                                " C13 H28 B N2 O5 -1 ",\n                                                " SCRPJEVETAWLOZ-CYBMUJFWSA-N "]\n                ]}\n            '),
(63, '5HJA', 'Crystal structure of Leishmania mexicana arginase in complex with inhibitor ABHDP', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=5HJA', '10.2210/pdb5hja/pdb', 'HYDROLASE', '2016-01-13', '2016-04-13', 'Leishmania mexicana', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.65', 'Crystal structures of Leishmania mexicana arginase complexed with alpha,alpha-disubstituted boronic amino-acid inhibitors', 'Hai, Y.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Arginase ",\n                                               " A ",\n                                               " 330 ",\n                                               " Leishmania mexicana ",\n                                               "Gene Name(s): ARG "]\n                ]}\n            ', 'GLYCEROL,MANGANESE (II) ION,(R)-2-amino-6-borono-2-(1-(3,4-dichlorobenzyl)piperidin-4-yl)hexanoic acid', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GOL ",\n                                                " GLYCEROL ",\n                                                " C3 H8 O3 ",\n                                                " PEDCQBHIVMGVHV-UHFFFAOYSA-N "]\n                , [" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " Mn 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                , [" XA2 ",\n                                                " (R)-2-amino-6-borono-2-(1-(3,4-dichlorobenzyl)piperidin-4-yl)hexanoic acid ",\n                                                " C18 H28 B Cl2 N2 O5 -1 ",\n                                                " LYUGMNLZENMUTE-GOSISDBHSA-N "]\n                ]}\n            '),
(64, '5RLA', 'ALTERING THE BINUCLEAR MANGANESE CLUSTER OF ARGINASE DIMINISHES THERMOSTABILITY AND CATALYTIC FUNCTION', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=5RLA', '10.2210/pdb5rla/pdb', 'HYDROLASE', '1997-05-07', '1998-05-13', 'Rattus norvegicus', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.74', 'Altering the binuclear manganese cluster of arginase diminishes thermostability and catalytic function.', 'Scolnick, L.R.#Kanyo, Z.F.#Cavalli, R.C.#Ash, D.E.#Christianson, D.W.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ARGINASE ",\n                                               " A,B,C ",\n                                               " 323 ",\n                                               " Rattus norvegicus ",\n                                               "Gene Name(s): Arg1 "]\n                ]}\n            ', 'MANGANESE (II) ION', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MN ",\n                                                " MANGANESE (II) ION ",\n                                                " MN 2 ",\n                                                " WAEMQWOKJMHJLA-UHFFFAOYSA-N "]\n                ]}\n            ');

-- --------------------------------------------------------

--
-- Table structure for table `pdb_Hemoglobin`
--

CREATE TABLE IF NOT EXISTS `pdb_Hemoglobin` (
`id` int(10) unsigned NOT NULL,
  `estrutura` varchar(32) NOT NULL,
  `descricao` varchar(512) CHARACTER SET utf16 DEFAULT NULL,
  `link` varchar(512) DEFAULT NULL,
  `doi` varchar(128) DEFAULT NULL,
  `classificacao` varchar(512) DEFAULT NULL,
  `depositado` date DEFAULT NULL,
  `lancado` date DEFAULT NULL,
  `organismo` varchar(512) DEFAULT NULL,
  `expressao` varchar(512) DEFAULT NULL,
  `metodo` varchar(255) DEFAULT NULL,
  `resolucao` varchar(16) DEFAULT NULL,
  `artigoorigem` text,
  `artigoautores` text CHARACTER SET utf16,
  `macromolecula` text,
  `ligantes` text,
  `tabelaligantes` text
) ENGINE=InnoDB DEFAULT CHARSET=latin1;

-- --------------------------------------------------------

--
-- Table structure for table `pdb_InhA`
--

CREATE TABLE IF NOT EXISTS `pdb_InhA` (
`id` int(10) unsigned NOT NULL,
  `estrutura` varchar(32) NOT NULL,
  `descricao` varchar(512) CHARACTER SET utf16 DEFAULT NULL,
  `link` varchar(512) DEFAULT NULL,
  `doi` varchar(128) DEFAULT NULL,
  `classificacao` varchar(512) DEFAULT NULL,
  `depositado` date DEFAULT NULL,
  `lancado` date DEFAULT NULL,
  `organismo` varchar(512) DEFAULT NULL,
  `expressao` varchar(512) DEFAULT NULL,
  `metodo` varchar(255) DEFAULT NULL,
  `resolucao` varchar(16) DEFAULT NULL,
  `artigoorigem` text,
  `artigoautores` text CHARACTER SET utf16,
  `macromolecula` text,
  `ligantes` text,
  `tabelaligantes` text
) ENGINE=InnoDB AUTO_INCREMENT=71 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `pdb_InhA`
--

INSERT INTO `pdb_InhA` (`id`, `estrutura`, `descricao`, `link`, `doi`, `classificacao`, `depositado`, `lancado`, `organismo`, `expressao`, `metodo`, `resolucao`, `artigoorigem`, `artigoautores`, `macromolecula`, `ligantes`, `tabelaligantes`) VALUES
(1, '1BVR', 'M.TB. ENOYL-ACP REDUCTASE (INHA) IN COMPLEX WITH NAD+ AND C16-FATTY-ACYL-SUBSTRATE', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1BVR', '10.2210/pdb1bvr/pdb', 'OXIDOREDUCTASE', '1998-09-17', '1999-09-17', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.8', 'Crystal structure of the Mycobacterium tuberculosis enoyl-ACP reductase, InhA, in complex with NAD+ and a C16 fatty acyl substrate.', 'Rozwarski, D.A.#Vilcheze, C.#Sugantino, M.#Bittman, R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" PROTEIN (ENOYL-ACYL CARRIER PROTEIN (ACP) REDUCTASE) ",\n                                               " A,B,C,D,E,F ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'NICOTINAMIDE-ADENINE-DINUCLEOTIDE,TRANS-2-HEXADECENOYL-(N-ACETYL-CYSTEAMINE)-THIOESTER', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" THT ",\n                                                " TRANS-2-HEXADECENOYL-(N-ACETYL-CYSTEAMINE)-THIOESTER ",\n                                                " C20 H39 N O2 S ",\n                                                " GDVZALUOXPTSHD-UHFFFAOYSA-N "]\n                ]}\n            '),
(2, '1ENY', 'CRYSTAL STRUCTURE AND FUNCTION OF THE ISONIAZID TARGET OF MYCOBACTERIUM TUBERCULOSIS', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1ENY', '10.2210/pdb1eny/pdb', 'OXIDOREDUCTASE', '1995-01-27', '1996-01-29', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.2', 'Crystal structure and function of the isoniazid target of Mycobacterium tuberculosis.', 'Dessen, A.#Quemard, A.#Blanchard, J.S.#Jacobs Jr., W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-ACYL CARRIER PROTEIN (ACP) REDUCTASE ",\n                                               " A ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(3, '1ENZ', 'CRYSTAL STRUCTURE AND FUNCTION OF THE ISONIAZID TARGET OF MYCOBACTERIUM TUBERCULOSIS', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1ENZ', '10.2210/pdb1enz/pdb', 'OXIDOREDUCTASE', '1995-01-27', '1996-01-29', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.7', 'Crystal structure and function of the isoniazid target of Mycobacterium tuberculosis.', 'Dessen, A.#Quemard, A.#Blanchard, J.S.#Jacobs Jr., W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-ACYL CARRIER PROTEIN (ACP) REDUCTASE ",\n                                               " A ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(4, '1P44', 'Targeting tuberculosis and malaria through inhibition of enoyl reductase: compound activity and structural data', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1P44', '10.2210/pdb1p44/pdb', 'OXIDOREDUCTASE', '2003-04-21', '2003-09-16', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.7', 'Targeting tuberculosis and malaria through inhibition of Enoyl reductase: compound activity and structural data.', 'Kuo, M.R.#Morbidoni, H.R.#Alland, D.#Sneddon, S.F.#Gourlie, B.B.#Staveski, M.M.#Leonard, M.#Gregory, J.S.#Janjigian, A.D.#Yee, C.#Musser, J.M.#Kreiswirth, B.#Iwamoto, H.#Perozzo, R.#Jacobs, W.R.#Sacchettini, J.C.#Fidock, D.A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D,E,F ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '5-{[4-(9H-FLUOREN-9-YL)PIPERAZIN-1-YL]CARBONYL}-1H-INDOLE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" GEQ ",\n                                                " 5-{[4-(9H-FLUOREN-9-YL)PIPERAZIN-1-YL]CARBONYL}-1H-INDOLE ",\n                                                " C26 H23 N3 O ",\n                                                " YYMZSGIXLQPFAC-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(5, '1P45', 'Targeting tuberculosis and malaria through inhibition of enoyl reductase: compound activity and structural data', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1P45', '10.2210/pdb1p45/pdb', 'OXIDOREDUCTASE', '2003-04-21', '2003-09-16', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.6', 'Targeting tuberculosis and malaria through inhibition of Enoyl reductase: compound activity and structural data.', 'Kuo, M.R.#Morbidoni, H.R.#Alland, D.#Sneddon, S.F.#Gourlie, B.B.#Staveski, M.M.#Leonard, M.#Gregory, J.S.#Janjigian, A.D.#Yee, C.#Musser, J.M.#Kreiswirth, B.#Iwamoto, H.#Perozzo, R.#Jacobs, W.R.#Sacchettini, J.C.#Fidock, D.A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'NICOTINAMIDE-ADENINE-DINUCLEOTIDE,TRICLOSAN', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" TCL ",\n                                                " TRICLOSAN ",\n                                                " C12 H7 CL3 O2 ",\n                                                " XEFQLINVKFYRCS-UHFFFAOYSA-N "]\n                ]}\n            '),
(6, '1ZID', 'LONG FATTY ACID CHAIN ENOYL-ACP REDUCTASE (INHA) IN COMPLEX WITH AN ISONICOTINIC-ACYL-NADH INHIBITOR', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=1ZID', '10.2210/pdb1zid/pdb', 'OXIDOREDUCTASE', '1997-03-25', '1998-03-25', 'Mycobacterium tuberculosis', 'Escherichia coli BL21', 'X-RAY DIFFRACTION', '2.7', 'Modification of the NADH of the isoniazid target (InhA) from Mycobacterium tuberculosis.', 'Rozwarski, D.A.#Grant, G.A.#Barton, D.H.#Jacobs Jr., W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE ",\n                                               " A ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ZID ",\n                                                " ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE ",\n                                                " C27 H30 N8 O15 P2 ",\n                                                " SURAWYIAXPVHGO-XDBKRARRSA-N "]\n                ]}\n            '),
(7, '2AQ8', 'Crystal structure of wild-type of Enoyl-ACP(CoA) reductase from Mycobacterium tuberculosis in complex with NADH.', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2AQ8', '10.2210/pdb2aq8/pdb', 'OXIDOREDUCTASE', '2005-08-17', '2006-05-23', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '1.92', 'Crystallographic and Pre-steady-state Kinetics Studies on Binding of NADH to Wild-type and Isoniazid-resistant Enoyl-ACP(CoA) Reductase Enzymes from Mycobacterium tuberculosis.', 'Oliveira, J.S.#Pereira, J.H.#Canduri, F.#Rodrigues, N.C.#de Souza, O.N.#de Azevedo Jr., W.F.#Basso, L.A.#Santos, D.S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-Acyl-carrier-protein reductase ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'LYSINE,1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" LYS ",\n                                                " LYSINE ",\n                                                " C6 H15 N2 O2 1 ",\n                                                " KDXKERNSBIXSRK-YFKPBYRVSA-O "]\n                , [" NAI ",\n                                                " 1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE ",\n                                                " C21 H29 N7 O14 P2 ",\n                                                " BOPGDPNILDQYTO-NNYOXOHSSA-N "]\n                ]}\n            '),
(8, '2AQH', 'Crystal structure of Isoniazid-resistant I21V Enoyl-ACP(CoA) reductase mutant enzyme from Mycobacterium tuberculosis in complex with NADH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2AQH', '10.2210/pdb2aqh/pdb', 'Oxidoreductase', '2005-08-18', '2006-05-23', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.01', 'Crystallographic and Pre-steady-state Kinetics Studies on Binding of NADH to Wild-type and Isoniazid-resistant Enoyl-ACP(CoA) Reductase Enzymes from Mycobacterium tuberculosis.', 'Oliveira, J.S.#Pereira, J.H.#Canduri, F.#Rodrigues, N.C.#de Souza, O.N.#de Azevedo Jr., W.F.#Basso, L.A.#Santos, D.S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAI ",\n                                                " 1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE ",\n                                                " C21 H29 N7 O14 P2 ",\n                                                " BOPGDPNILDQYTO-NNYOXOHSSA-N "]\n                ]}\n            '),
(9, '2AQI', 'Crystal structure of Isoniazid-resistant I47T Enoyl-ACP(CoA) reductase mutant enzyme from Mycobacterium tuberculosis in complex with NADH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2AQI', '10.2210/pdb2aqi/pdb', 'Oxidoreductase', '2005-08-18', '2006-05-23', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.2', 'Crystallographic and Pre-steady-state Kinetics Studies on Binding of NADH to Wild-type and Isoniazid-resistant Enoyl-ACP(CoA) Reductase Enzymes from Mycobacterium tuberculosis.', 'Oliveira, J.S.#Pereira, J.H.#Canduri, F.#Rodrigues, N.C.#de Souza, O.N.#de Azevedo Jr., W.F.#Basso, L.A.#Santos, D.S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAI ",\n                                                " 1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE ",\n                                                " C21 H29 N7 O14 P2 ",\n                                                " BOPGDPNILDQYTO-NNYOXOHSSA-N "]\n                ]}\n            '),
(10, '2AQK', 'Crystal structure of Isoniazid-resistant S94A Enoyl-ACP(CoA) reductase mutant enzyme from Mycobacterium tuberculosis in complex with NADH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2AQK', '10.2210/pdb2aqk/pdb', 'Oxidoreductase', '2005-08-18', '2006-05-23', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.3', 'Crystallographic and Pre-steady-state Kinetics Studies on Binding of NADH to Wild-type and Isoniazid-resistant Enoyl-ACP(CoA) Reductase Enzymes from Mycobacterium tuberculosis.', 'Oliveira, J.S.#Pereira, J.H.#Canduri, F.#Rodrigues, N.C.#de Souza, O.N.#de Azevedo Jr., W.F.#Basso, L.A.#Santos, D.S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAI ",\n                                                " 1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE ",\n                                                " C21 H29 N7 O14 P2 ",\n                                                " BOPGDPNILDQYTO-NNYOXOHSSA-N "]\n                ]}\n            '),
(11, '2B35', 'Crystal structure of Mycobacterium tuberculosis enoyl reductase (InhA) inhibited by triclosan', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2B35', '10.2210/pdb2b35/pdb', 'OXIDOREDUCTASE', '2005-09-19', '2006-03-07', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.3', 'High Affinity InhA Inhibitors with Activity against Drug-Resistant Strains  of Mycobacterium tuberculosis', 'Sullivan, T.J.#Truglio, J.J.#Boyne, M.E.#Novichenok, P.#Zhang, X.#Stratton, C.#Li, H.J.#Kaur, T.#Amin, A.#Johnson, F.#Slayden, R.A.#Kisker, C.#Tonge, P.J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D,E,F ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'NICOTINAMIDE-ADENINE-DINUCLEOTIDE,TRICLOSAN', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" TCL ",\n                                                " TRICLOSAN ",\n                                                " C12 H7 CL3 O2 ",\n                                                " XEFQLINVKFYRCS-UHFFFAOYSA-N "]\n                ]}\n            '),
(12, '2B36', 'Crystal structure of Mycobacterium tuberculosis enoyl reductase (InhA) inhibited by 5-pentyl-2-phenoxyphenol', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2B36', '10.2210/pdb2b36/pdb', 'OXIDOREDUCTASE', '2005-09-19', '2006-03-07', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.8', 'High Affinity InhA Inhibitors with Activity against Drug-Resistant Strains  of Mycobacterium tuberculosis', 'Sullivan, T.J.#Truglio, J.J.#Boyne, M.E.#Novichenok, P.#Zhang, X.#Stratton, C.#Li, H.J.#Kaur, T.#Amin, A.#Johnson, F.#Slayden, R.A.#Kisker, C.#Tonge, P.J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D,E,F ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '5-PENTYL-2-PHENOXYPHENOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 5PP ",\n                                                " 5-PENTYL-2-PHENOXYPHENOL ",\n                                                " C17 H20 O2 ",\n                                                " OJLYTHOKCYLPMA-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(13, '2B37', 'Crystal structure of Mycobacterium tuberculosis enoyl reductase (InhA) inhibited by 5-octyl-2-phenoxyphenol', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2B37', '10.2210/pdb2b37/pdb', 'OXIDOREDUCTASE', '2005-09-19', '2006-03-07', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.6', 'High Affinity InhA Inhibitors with Activity against Drug-Resistant Strains  of Mycobacterium tuberculosis', 'Sullivan, T.J.#Truglio, J.J.#Boyne, M.E.#Novichenok, P.#Zhang, X.#Stratton, C.#Li, H.J.#Kaur, T.#Amin, A.#Johnson, F.#Slayden, R.A.#Kisker, C.#Tonge, P.J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D,E,F ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '5-OCTYL-2-PHENOXYPHENOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 8PS ",\n                                                " 5-OCTYL-2-PHENOXYPHENOL ",\n                                                " C20 H26 O2 ",\n                                                " JOWYBLIPWAMIHM-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(14, '2H9I', 'Mycobacterium tuberculosis InhA bound with ETH-NAD adduct', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2H9I', '10.2210/pdb2h9i/pdb', 'OXIDOREDUCTASE', '2006-06-09', '2007-01-30', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.2', 'Mechanism of thioamide drug action against tuberculosis and leprosy.', 'Wang, F.#Langley, R.#Gulten, G.#Dover, L.G.#Besra, G.S.#Jacobs, W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '{(2R,3S,4R,5R)-5-[(4S)-3-(AMINOCARBONYL)-4-(2-ETHYLISONICOTINOYL)PYRIDIN-1(4H)-YL]-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL}METHYL [(2R,3S,4R,5R)-5-(6-AMINO-9H-PURIN-9-YL)-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL]METHYL DIHYDROGEN DIPHOSPHATE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" EAD ",\n                                                " {(2R,3S,4R,5R)-5-[(4S)-3-(AMINOCARBONYL)-4-(2-ETHYLISONICOTINOYL)PYRIDIN-1(4H)-YL]-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL}METHYL [(2R,3S,4R,5R)-5-(6-AMINO-9H-PURIN-9-YL)-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL]METHYL DIHYDROGEN DIPHOSPHATE ",\n                                                " C29 H36 N8 O15 P2 ",\n                                                " KVEIKWLOSUPDGF-OYBINNFYSA-N "]\n                ]}\n            '),
(15, '2IDZ', 'Crystal structure of wild type Enoyl-ACP(CoA) reductase from Mycobacterium tuberculosis in complex with NADH-INH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2IDZ', '10.2210/pdb2idz/pdb', 'OXIDOREDUCTASE', '2006-09-15', '2007-07-24', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.0', 'Crystallographic studies on the binding of isonicotinyl-NAD adduct to wild-type and isoniazid resistant 2-trans-enoyl-ACP (CoA) reductase from Mycobacterium tuberculosis.', 'Dias, M.V.#Vasconcelos, I.B.#Prado, A.M.#Fadel, V.#Basso, L.A.#de Azevedo, W.F.#Santos, D.S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ZID ",\n                                                " ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE ",\n                                                " C27 H30 N8 O15 P2 ",\n                                                " SURAWYIAXPVHGO-XDBKRARRSA-N "]\n                ]}\n            '),
(16, '2IE0', 'Crystal Structure of Isoniazid-resistant I21V Enoyl-ACP(COA) Reductase Mutant Enzyme From MYCOBACTERIUM TUBERCULOSIS in Complex with NADH-INH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2IE0', '10.2210/pdb2ie0/pdb', 'OXIDOREDUCTASE', '2006-09-15', '2007-07-24', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.2', 'Crystallographic studies on the binding of isonicotinyl-NAD adduct to wild-type and isoniazid resistant 2-trans-enoyl-ACP (CoA) reductase from Mycobacterium tuberculosis.', 'Dias, M.V.#Vasconcelos, I.B.#Prado, A.M.#Fadel, V.#Basso, L.A.#de Azevedo, W.F.#Santos, D.S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ZID ",\n                                                " ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE ",\n                                                " C27 H30 N8 O15 P2 ",\n                                                " SURAWYIAXPVHGO-XDBKRARRSA-N "]\n                ]}\n            '),
(17, '2IEB', 'Crystal Structure of Isoniazid-resistant S94A ENOYL-ACP(COA) Reductase Mutant Enzyme from MYCOBACTERIUM TUBERCULOSIS in Complex with NADH-INH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2IEB', '10.2210/pdb2ieb/pdb', 'OXIDOREDUCTASE', '2006-09-18', '2007-07-24', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.2', 'Crystallographic studies on the binding of isonicotinyl-NAD adduct to wild-type and isoniazid resistant 2-trans-enoyl-ACP (CoA) reductase from Mycobacterium tuberculosis.', 'Dias, M.V.#Vasconcelos, I.B.#Prado, A.M.#Fadel, V.#Basso, L.A.#de Azevedo, W.F.#Santos, D.S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ZID ",\n                                                " ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE ",\n                                                " C27 H30 N8 O15 P2 ",\n                                                " SURAWYIAXPVHGO-XDBKRARRSA-N "]\n                ]}\n            '),
(18, '2IED', 'CRYSTAL STRUCTURE of ISONIAZID-RESISTANT S94A ENOYL-ACP(COA) REDUCTASE MUTANT ENZYME FROM MYCOBACTERIUM TUBERCULOSIS UNCOMPLEXED', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2IED', '10.2210/pdb2ied/pdb', 'OXIDOREDUCTASE', '2006-09-18', '2007-07-24', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.14', 'Crystallographic studies on the binding of isonicotinyl-NAD adduct to wild-type and isoniazid resistant 2-trans-enoyl-ACP (CoA) reductase from Mycobacterium tuberculosis.', 'Dias, M.V.#Vasconcelos, I.B.#Prado, A.M.#Fadel, V.#Basso, L.A.#de Azevedo, W.F.#Santos, D.S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '', NULL),
(19, '2NSD', 'Enoyl acyl carrier protein reductase InhA in complex with N-(4-methylbenzoyl)-4-benzylpiperidine', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2NSD', '10.2210/pdb2nsd/pdb', 'OXIDOREDUCTASE', '2006-11-03', '2007-09-18', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.9', 'Inhibition of the Mycobacterium tuberculosis enoyl acyl carrier protein reductase InhA by arylamides.', 'He, X.#Alian, A.#Ortiz de Montellano, P.R.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase ",\n                                               " A,B ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'N-(4-METHYLBENZOYL)-4-BENZYLPIPERIDINE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 4PI ",\n                                                " N-(4-METHYLBENZOYL)-4-BENZYLPIPERIDINE ",\n                                                " C20 H23 N O ",\n                                                " DVOLWKZEIDCCES-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(20, '2NTJ', 'Mycobacterium tuberculosis InhA bound with PTH-NAD adduct', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2NTJ', '10.2210/pdb2ntj/pdb', 'OXIDOREDUCTASE', '2006-11-07', '2007-01-30', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.5', 'Mechanism of thioamide drug action against tuberculosis and leprosy.', 'Wang, F.#Langley, R.#Gulten, G.#Dover, L.G.#Besra, G.S.#Jacobs, W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH ",\n                                               " A,B ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '{(2R,3S,4R,5R)-5-[(4S)-3-(AMINOCARBONYL)-4-(2-PROPYLISONICOTINOYL)PYRIDIN-1(4H)-YL]-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL}METHYL [(2R,3S,4R,5R)-5-(6-AMINO-9H-PURIN-9-YL)-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL]METHYL DIHYDROGEN DIPHOSPHATE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" P1H ",\n                                                " {(2R,3S,4R,5R)-5-[(4S)-3-(AMINOCARBONYL)-4-(2-PROPYLISONICOTINOYL)PYRIDIN-1(4H)-YL]-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL}METHYL [(2R,3S,4R,5R)-5-(6-AMINO-9H-PURIN-9-YL)-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL]METHYL DIHYDROGEN DIPHOSPHATE ",\n                                                " C30 H38 N8 O15 P2 ",\n                                                " INZZWAGIWIYXKO-NNTKUHMYSA-N "]\n                ]}\n            '),
(21, '2NTV', 'Mycobacterium leprae InhA bound with PTH-NAD adduct', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2NTV', '10.2210/pdb2ntv/pdb', 'OXIDOREDUCTASE', '2006-11-08', '2007-01-30', 'Mycobacterium leprae', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.1', 'Mechanism of thioamide drug action against tuberculosis and leprosy.', 'Wang, F.#Langley, R.#Gulten, G.#Dover, L.G.#Besra, G.S.#Jacobs, W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[ACP] reductase ",\n                                               " A,B ",\n                                               " 268 ",\n                                               " Mycobacterium leprae ",\n                                               "Gene Name(s): inhA "]\n                ]}\n            ', '{(2R,3S,4R,5R)-5-[(4S)-3-(AMINOCARBONYL)-4-(2-PROPYLISONICOTINOYL)PYRIDIN-1(4H)-YL]-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL}METHYL [(2R,3S,4R,5R)-5-(6-AMINO-9H-PURIN-9-YL)-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL]METHYL DIHYDROGEN DIPHOSPHATE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" P1H ",\n                                                " {(2R,3S,4R,5R)-5-[(4S)-3-(AMINOCARBONYL)-4-(2-PROPYLISONICOTINOYL)PYRIDIN-1(4H)-YL]-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL}METHYL [(2R,3S,4R,5R)-5-(6-AMINO-9H-PURIN-9-YL)-3,4-DIHYDROXYTETRAHYDROFURAN-2-YL]METHYL DIHYDROGEN DIPHOSPHATE ",\n                                                " C30 H38 N8 O15 P2 ",\n                                                " INZZWAGIWIYXKO-NNTKUHMYSA-N "]\n                ]}\n            '),
(22, '2NV6', 'Mycobacterium tuberculosis InhA (S94A) bound with INH-NAD adduct', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2NV6', '10.2210/pdb2nv6/pdb', 'OXIDOREDUCTASE', '2006-11-10', '2006-11-21', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '1.9', 'Transfer of a point mutation in Mycobacterium tuberculosis inhA resolves the target of isoniazid.', 'Vilcheze, C.#Wang, F.#Arai, M.#Hazbon, M.H.#Colangeli, R.#Kremer, L.#Weisbrod, T.R.#Alland, D.#Sacchettini, J.C.#Jacobs Jr., W.R.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH ",\n                                               " A ",\n                                               " 268 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ZID ",\n                                                " ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE ",\n                                                " C27 H30 N8 O15 P2 ",\n                                                " SURAWYIAXPVHGO-XDBKRARRSA-N "]\n                ]}\n            '),
(23, '2PR2', 'Structure of Mycobacterium tuberculosis enoyl-ACP reductase with bound INH-NADP.', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2PR2', '10.2210/pdb2pr2/pdb', 'OXIDOREDUCTASE', '2007-05-03', '2008-01-15', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.5', 'New insight into the mechanism of action of and resistance to isoniazid: interaction of Mycobacterium tuberculosis enoyl-ACP reductase with INH-NADP', 'Argyrou, A.#Vetting, M.W.#Blanchard, J.S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" enoyl-ACP Reductase ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '(4S)-ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" DG1 ",\n                                                " (4S)-ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE ",\n                                                " C27 H33 N8 O18 P3 ",\n                                                " FPZCWRGJBUDVSB-BGARNJTLSA-N "]\n                ]}\n            '),
(24, '2X22', 'CRYSTAL STRUCTURE OF M. TUBERCULOSIS INHA INHIBITED BY PT70', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2X22', '10.2210/pdb2x22/pdb', 'OXIDOREDUCTASE', '2010-01-10', '2010-03-02', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '2.1', 'A Slow, Tight Binding Inhibitor of Inha, the Enoyl-Acyl Carrier Protein Reductase from Mycobacterium Tuberculosis.', 'Luckner, S.R.#Liu, N.#Am Ende, C.W.#Tonge, P.J.#Kisker, C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'DIMETHYL SULFOXIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE,5-HEXYL-2-(2-METHYLPHENOXY)PHENOL', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" DMS ",\n                                                " DIMETHYL SULFOXIDE ",\n                                                " C2 H6 O S ",\n                                                " IAZDPXIOMUYVGZ-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" TCU ",\n                                                " 5-HEXYL-2-(2-METHYLPHENOXY)PHENOL ",\n                                                " C19 H24 O2 ",\n                                                " KSWHNTPMGZKIOB-UHFFFAOYSA-N "]\n                ]}\n            '),
(25, '2X23', 'CRYSTAL STRUCTURE OF M. TUBERCULOSIS INHA INHIBITED BY PT70', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=2X23', '10.2210/pdb2x23/pdb', 'OXIDOREDUCTASE', '2010-01-10', '2010-03-02', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '1.81', 'A Slow, Tight Binding Inhibitor of Inha, the Enoyl-Acyl Carrier Protein Reductase from Mycobacterium Tuberculosis.', 'Luckner, S.R.#Liu, N.#Am Ende, C.W.#Tonge, P.J.#Kisker, C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B,E,G ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'DIMETHYL SULFOXIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE,5-HEXYL-2-(2-METHYLPHENOXY)PHENOL', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" DMS ",\n                                                " DIMETHYL SULFOXIDE ",\n                                                " C2 H6 O S ",\n                                                " IAZDPXIOMUYVGZ-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" TCU ",\n                                                " 5-HEXYL-2-(2-METHYLPHENOXY)PHENOL ",\n                                                " C19 H24 O2 ",\n                                                " KSWHNTPMGZKIOB-UHFFFAOYSA-N "]\n                ]}\n            '),
(26, '3FNE', 'Crystal structure of InhA bound to triclosan derivative 17', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3FNE', '10.2210/pdb3fne/pdb', 'OXIDOREDUCTASE', '2008-12-24', '2009-01-20', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.98', 'Triclosan Derivatives: Towards Potent Inhibitors of Drug-Sensitive and Drug-Resistant Mycobacterium tuberculosis.', 'Freundlich, J.S.#Wang, F.#Vilcheze, C.#Gulten, G.#Langley, R.#Schiehser, G.A.#Jacobus, D.P.#Jacobs, W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '2-(2,4-DICHLOROPHENOXY)-5-(PYRIDIN-2-YLMETHYL)PHENOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 8PC ",\n                                                " 2-(2,4-DICHLOROPHENOXY)-5-(PYRIDIN-2-YLMETHYL)PHENOL ",\n                                                " C18 H13 CL2 N O2 ",\n                                                " AOVDSWPGWPRTSR-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(27, '3FNF', 'Crystal structure of InhA bound to triclosan derivative', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3FNF', '10.2210/pdb3fnf/pdb', 'OXIDOREDUCTASE', '2008-12-24', '2009-01-20', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.3', 'Triclosan Derivatives: Towards Potent Inhibitors of Drug-Sensitive and Drug-Resistant Mycobacterium tuberculosis.', 'Freundlich, J.S.#Wang, F.#Vilcheze, C.#Gulten, G.#Langley, R.#Schiehser, G.A.#Jacobus, D.P.#Jacobs, W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '5-BENZYL-2-(2,4-DICHLOROPHENOXY)PHENOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" JPM ",\n                                                " 5-BENZYL-2-(2,4-DICHLOROPHENOXY)PHENOL ",\n                                                " C19 H14 CL2 O2 ",\n                                                " LUMKUDPYQGEIPU-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            ');
INSERT INTO `pdb_InhA` (`id`, `estrutura`, `descricao`, `link`, `doi`, `classificacao`, `depositado`, `lancado`, `organismo`, `expressao`, `metodo`, `resolucao`, `artigoorigem`, `artigoautores`, `macromolecula`, `ligantes`, `tabelaligantes`) VALUES
(28, '3FNG', 'Crystal structure of InhA bound to triclosan derivative', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3FNG', '10.2210/pdb3fng/pdb', 'OXIDOREDUCTASE', '2008-12-24', '2009-01-20', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.97', 'Triclosan Derivatives: Towards Potent Inhibitors of Drug-Sensitive and Drug-Resistant Mycobacterium tuberculosis.', 'Freundlich, J.S.#Wang, F.#Vilcheze, C.#Gulten, G.#Langley, R.#Schiehser, G.A.#Jacobus, D.P.#Jacobs, W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '5-(CYCLOHEXYLMETHYL)-2-(2,4-DICHLOROPHENOXY)PHENOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" JPL ",\n                                                " 5-(CYCLOHEXYLMETHYL)-2-(2,4-DICHLOROPHENOXY)PHENOL ",\n                                                " C19 H20 CL2 O2 ",\n                                                " AUJNRGORQMIJCP-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(29, '3FNH', 'Crystal structure of InhA bound to triclosan derivative', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3FNH', '10.2210/pdb3fnh/pdb', 'OXIDOREDUCTASE', '2008-12-24', '2009-01-20', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.8', 'Triclosan Derivatives: Towards Potent Inhibitors of Drug-Sensitive and Drug-Resistant Mycobacterium tuberculosis.', 'Freundlich, J.S.#Wang, F.#Vilcheze, C.#Gulten, G.#Langley, R.#Schiehser, G.A.#Jacobus, D.P.#Jacobs, W.R.#Sacchettini, J.C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '2-(2,4-DICHLOROPHENOXY)-5-(2-PHENYLETHYL)PHENOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" JPJ ",\n                                                " 2-(2,4-DICHLOROPHENOXY)-5-(2-PHENYLETHYL)PHENOL ",\n                                                " C20 H16 CL2 O2 ",\n                                                " SOSAEWQXZFTNAQ-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(30, '3OEW', 'Crystal structure of wild-type InhA:NADH complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3OEW', '10.2210/pdb3oew/pdb', 'OXIDOREDUCTASE', '2010-08-13', '2010-12-01', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.2', 'Phosphorylation of InhA inhibits mycolic acid biosynthesis and growth of Mycobacterium tuberculosis.', 'Molle, V.#Gulten, G.#Vilcheze, C.#Veyron-Churlet, R.#Zanella-Cleon, I.#Sacchettini, J.C.#Jacobs Jr, W.R.#Kremer, L.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '(4S)-2-METHYL-2,4-PENTANEDIOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MPD ",\n                                                " (4S)-2-METHYL-2,4-PENTANEDIOL ",\n                                                " C6 H14 O2 ",\n                                                " SVTBMSDMJJWYQN-YFKPBYRVSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(31, '3OEY', 'Crystal structure of InhA_T266E:NADH complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3OEY', '10.2210/pdb3oey/pdb', 'OXIDOREDUCTASE', '2010-08-13', '2010-12-01', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.0', 'Phosphorylation of InhA inhibits mycolic acid biosynthesis and growth of Mycobacterium tuberculosis.', 'Molle, V.#Gulten, G.#Vilcheze, C.#Veyron-Churlet, R.#Zanella-Cleon, I.#Sacchettini, J.C.#Jacobs Jr, W.R.#Kremer, L.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '(4S)-2-METHYL-2,4-PENTANEDIOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MPD ",\n                                                " (4S)-2-METHYL-2,4-PENTANEDIOL ",\n                                                " C6 H14 O2 ",\n                                                " SVTBMSDMJJWYQN-YFKPBYRVSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(32, '3OF2', 'Crystal structure of InhA_T266D:NADH complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=3OF2', '10.2210/pdb3of2/pdb', 'OXIDOREDUCTASE', '2010-08-13', '2010-12-01', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.0', 'Phosphorylation of InhA inhibits mycolic acid biosynthesis and growth of Mycobacterium tuberculosis.', 'Molle, V.#Gulten, G.#Vilcheze, C.#Veyron-Churlet, R.#Zanella-Cleon, I.#Sacchettini, J.C.#Jacobs Jr, W.R.#Kremer, L.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '(4S)-2-METHYL-2,4-PENTANEDIOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MPD ",\n                                                " (4S)-2-METHYL-2,4-PENTANEDIOL ",\n                                                " C6 H14 O2 ",\n                                                " SVTBMSDMJJWYQN-YFKPBYRVSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(33, '4BGE', 'Crystal structure of InhA(S94A) mutant in complex with pyridomycin', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4BGE', '10.2210/pdb4bge/pdb', 'OXIDOREDUCTASE', '2013-03-26', '2013-12-04', 'Mycobacterium tuberculosis', 'null', 'X-RAY DIFFRACTION', '2.25', 'Pyridomycin Bridges the Nadh and Substrate Binding Pockets of the Enoyl Reductase Inha', 'Hartkoorn, R.#Pojer, F.#Read, J.A.#Gingell, H.#Neres, J.#Horlacher, O.#Altmann, K.H.#Cole, S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B,C,D,E,F ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'PYRIDOMYCIN', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" PYW ",\n                                                " PYRIDOMYCIN ",\n                                                " C27 H32 N4 O8 ",\n                                                " WHIKSLGSXKIHCA-IGCCMALHSA-N "]\n                ]}\n            '),
(34, '4BGI', 'Crystal structure of InhA(S94A) mutant in complex with OH-141', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4BGI', '10.2210/pdb4bgi/pdb', 'OXIDOREDUCTASE', '2013-03-27', '2013-12-04', 'Mycobacterium tuberculosis', 'null', 'X-RAY DIFFRACTION', '2.09', 'Pyridomycin Bridges the Nadh and Substrate Binding Pockets of the Enoyl Reductase Inha', 'Hartkoorn, R.#Pojer, F.#Read, J.A.#Gingell, H.#Neres, J.#Horlacher, O.#Altmann, K.H.#Cole, S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B,C,D,E,F ",\n                                               " 281 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '3-HYDROXY-N-[(2R,5R,6S,9S,10S,11R)-10-HYDROXY-5,11-DIMETHYL-3,7,12-TRIOXO-2-(PROPAN-2-YL)-9-(PYRIDIN-3-YLMETHYL)-1,4-DIOXA-8-AZACYCLODODECAN-6-YL]PYRIDINE-2-CARBOXAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" I4I ",\n                                                " 3-HYDROXY-N-[(2R,5R,6S,9S,10S,11R)-10-HYDROXY-5,11-DIMETHYL-3,7,12-TRIOXO-2-(PROPAN-2-YL)-9-(PYRIDIN-3-YLMETHYL)-1,4-DIOXA-8-AZACYCLODODECAN-6-YL]PYRIDINE-2-CARBOXAMIDE ",\n                                                " C26 H32 N4 O8 ",\n                                                " ZCNHOXIPSWXQHI-PMWBZGJESA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(35, '4BII', 'How nature bridges the gap: Crystallographic elucidation of pyridomycin binding to InhA', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4BII', '10.2210/pdb4bii/pdb', 'OXIDOREDUCTASE', '2013-04-10', '2013-12-04', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '1.95', 'Pyridomycin Bridges the Nadh and Substrate Binding Pockets of the Enoyl Reductase Inha', 'Hartkoorn, R.#Pojer, F.#Read, J.A.#Gingell, H.#Neres, J.#Horlacher, O.#Altmann, K.H.#Cole, S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B,C,D ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'NICOTINAMIDE-ADENINE-DINUCLEOTIDE,PYRIDOMYCIN', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" PYW ",\n                                                " PYRIDOMYCIN ",\n                                                " C27 H32 N4 O8 ",\n                                                " WHIKSLGSXKIHCA-IGCCMALHSA-N "]\n                ]}\n            '),
(36, '4BQP', 'Mtb InhA complex with Methyl-thiazole compound 7', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4BQP', '10.2210/pdb4bqp/pdb', 'OXIDOREDUCTASE', '2013-05-31', '2013-12-11', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '1.89', 'Methyl-Thiazoles: A Novel Mode of Inhibition with the Potential to Develop Novel Inhibitors Targeting Inha in Mycobacterium Tuberculosis.', 'Shirude, P.S.#Madhavapeddi, P.#Naik, M.#Murugan, K.#Shinde, V.#Nandishaiah, R.#Bhat, J.#Kumar, A.#Hameed, S.#Holdgate, G.#Davies, G.#Mcmiken, H.#Hegde, N.#Ambady, A.#Venkatraman, J.#Panda, M.#Bandodkar, B.#Sambandamurthy, V.K.#Read, J.A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B,C,D,E,F ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'SODIUM ION,NICOTINAMIDE-ADENINE-DINUCLEOTIDE,(1S)-1-(5-{[1-(2,6-DIFLUOROBENZYL)-1H-PYRAZOL-3-YL]AMINO}-1,3,4-THIADIAZOL-2-YL)-1-(4-METHYL-1,3-THIAZOL-2-YL)ETHANOL', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NA ",\n                                                " SODIUM ION ",\n                                                " NA 1 ",\n                                                " FKNQFGJONOIPTF-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" VMY ",\n                                                " (1S)-1-(5-{[1-(2,6-DIFLUOROBENZYL)-1H-PYRAZOL-3-YL]AMINO}-1,3,4-THIADIAZOL-2-YL)-1-(4-METHYL-1,3-THIAZOL-2-YL)ETHANOL ",\n                                                " C18 H16 F2 N6 O S2 ",\n                                                " UXRSVTXAGGLLHU-SFHVURJKSA-N "]\n                ]}\n            '),
(37, '4BQR', 'Mtb InhA complex with Methyl-thiazole compound 11', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4BQR', '10.2210/pdb4bqr/pdb', 'OXIDOREDUCTASE', '2013-05-31', '2013-12-11', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '2.05', 'Methyl-Thiazoles: A Novel Mode of Inhibition with the Potential to Develop Novel Inhibitors Targeting Inha in Mycobacterium Tuberculosis.', 'Shirude, P.S.#Madhavapeddi, P.#Naik, M.#Murugan, K.#Shinde, V.#Nandishaiah, R.#Bhat, J.#Kumar, A.#Hameed, S.#Holdgate, G.#Davies, G.#Mcmiken, H.#Hegde, N.#Ambady, A.#Venkatraman, J.#Panda, M.#Bandodkar, B.#Sambandamurthy, V.K.#Read, J.A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B,C,D ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '(NZ)-2-[2,6-BIS(FLUORANYL)PHENYL]-N-[5-[(1S)-1-(4-METHYL-1,3-THIAZOL-2-YL)-1-OXIDANYL-ETHYL]-3H-1,3,4-THIADIAZOL-2-YLIDENE]ETHANAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" IBH ",\n                                                " (NZ)-2-[2,6-BIS(FLUORANYL)PHENYL]-N-[5-[(1S)-1-(4-METHYL-1,3-THIAZOL-2-YL)-1-OXIDANYL-ETHYL]-3H-1,3,4-THIADIAZOL-2-YLIDENE]ETHANAMIDE ",\n                                                " C16 H14 F2 N4 O2 S2 ",\n                                                " LBTQISHPRWJOLN-INIZCTEOSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(38, '4COD', 'Encoded library technology as a source of hits for the discovery and lead optimization of a potent and selective class of bactericidal direct inhibitors of Mycobacterium tuberculosis InhA', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4COD', '10.2210/pdb4cod/pdb', 'TRANSFERASE', '2014-01-28', '2014-02-12', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '2.4', 'Encoded Library Technology as a Source of Hits for the Discovery and Lead Optimization of a Potent and Selective Class of Bactericidal Direct Inhibitors of Mycobacterium Tuberculosis Inha.', 'Encinas, L.#O''Keefe, H.#Neu, M.#Remuinan, M.J.#Patel, A.M.#Guardia, A.#Davie, C.P.#Perez-Macias, N.#Yang, H.#Convery, M.A.#Messer, J.A.#Perez-Herran, E.#Centrella, P.A.#Alvarez-Gomez, D.#Clark, M.A.#Huss, S.#O''Donovan, G.K.#Ortega-Muro, F.#Mcdowell, W.#Castaneda, P.#Arico-Muendel, C.C.#Pajk, S.#Rullas, J.#Angulo-Barturen, I.#Alvarez-Ruiz, E.#Mendoza-Losana, A.#Pages, L.B.#Castro-Pichel, J.#Evindar, G.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " B,D,F,H ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#MT1531 "]\n                ]}\n            ', 'N-((3R,5S)-1-(BENZOFURAN-3-CARBONYL)-5-(ETHYLCARBAMOYL)PYRROLIDIN-3-YL)-3-ETHYL-1-METHYL-1H-PYRAZOLE-5-CARBOXAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" KV1 ",\n                                                " N-((3R,5S)-1-(BENZOFURAN-3-CARBONYL)-5-(ETHYLCARBAMOYL)PYRROLIDIN-3-YL)-3-ETHYL-1-METHYL-1H-PYRAZOLE-5-CARBOXAMIDE ",\n                                                " C23 H27 N5 O4 ",\n                                                " NCHSTBMDNNMMEJ-BEFAXECRSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(39, '4D0R', 'Mtb InhA complex with Pyradizinone compound 1', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4D0R', '10.2210/pdb4d0r/pdb', 'OXIDOREDUCTASE', '2014-04-29', '2015-05-20', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '2.75', 'Pyridazinones: A Novel Scaffold with Excellent Physicochemical Properties and Safety Profile for a Clinically Validated Target of Mycobacterium Tuberculosis', 'Lange, S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '4-(4-CHLOROPHENOXY)-6-OXO-1-PHENYL-1,6-DIHYDROPYRIDAZINE-3-CARBOXAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 41I ",\n                                                " 4-(4-CHLOROPHENOXY)-6-OXO-1-PHENYL-1,6-DIHYDROPYRIDAZINE-3-CARBOXAMIDE ",\n                                                " C17 H12 CL N3 O3 ",\n                                                " JNJFDMSSDRBEMQ-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(40, '4D0S', 'Mtb InhA complex with Pyradizinone compound 14', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4D0S', '10.2210/pdb4d0s/pdb', 'OXIDOREDUCTASE', '2014-04-29', '2015-05-20', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '1.64', 'Pyridazinones: A Novel Scaffold with Excellent Physicochemical Properties and Safety Profile for a Clinically Validated Target of Mycobacterium Tuberculosis', 'Lange, S.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B,C,D ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '1-{4-[(ACETYLAMINO)METHYL]PHENYL}-4-(4-CHLOROPHENOXY)-6-OXO-1,6-DIHYDROPYRIDAZINE-3-CARBOXAMIDE,MAGNESIUM ION,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 9G4 ",\n                                                " 1-{4-[(ACETYLAMINO)METHYL]PHENYL}-4-(4-CHLOROPHENOXY)-6-OXO-1,6-DIHYDROPYRIDAZINE-3-CARBOXAMIDE ",\n                                                " C20 H17 CL N4 O4 ",\n                                                " SYAJIRKWQQECEF-UHFFFAOYSA-N "]\n                , [" MG ",\n                                                " MAGNESIUM ION ",\n                                                " MG 2 ",\n                                                " JLVVSXFLKOJNIY-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(41, '4DQU', 'Mycobacterium tuberculosis InhA-D148G mutant in complex with NADH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4DQU', '10.2210/pdb4dqu/pdb', 'OXIDOREDUCTASE', '2012-02-16', '2012-10-03', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.45', 'Towards a new tuberculosis drug: pyridomycin - nature''s isoniazid.', 'Hartkoorn, R.C.#Sala, C.#Neres, J.#Pojer, F.#Magnet, S.#Mukherjee, R.#Uplekar, S.#Boy-Rottger, S.#Altmann, K.H.#Cole, S.T.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAI ",\n                                                " 1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE ",\n                                                " C21 H29 N7 O14 P2 ",\n                                                " BOPGDPNILDQYTO-NNYOXOHSSA-N "]\n                ]}\n            '),
(42, '4DRE', 'Mycobacterium tuberculosis InhA in complex with NADH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4DRE', '10.2210/pdb4dre/pdb', 'OXIDOREDUCTASE', '2012-02-17', '2012-10-03', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.4', 'Towards a new tuberculosis drug: pyridomycin - nature''s isoniazid.', 'Hartkoorn, R.C.#Sala, C.#Neres, J.#Pojer, F.#Magnet, S.#Mukherjee, R.#Uplekar, S.#Boy-Rottger, S.#Altmann, K.H.#Cole, S.T.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAI ",\n                                                " 1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE ",\n                                                " C21 H29 N7 O14 P2 ",\n                                                " BOPGDPNILDQYTO-NNYOXOHSSA-N "]\n                ]}\n            '),
(43, '4DTI', 'Mycobacterium tuberculosis InhA-S94A mutant in complex with NADH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4DTI', '10.2210/pdb4dti/pdb', 'OXIDOREDUCTASE', '2012-02-21', '2012-10-03', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.9', 'Towards a new tuberculosis drug: pyridomycin - nature''s isoniazid.', 'Hartkoorn, R.C.#Sala, C.#Neres, J.#Pojer, F.#Magnet, S.#Mukherjee, R.#Uplekar, S.#Boy-Rottger, S.#Altmann, K.H.#Cole, S.T.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAI ",\n                                                " 1,4-DIHYDRONICOTINAMIDE ADENINE DINUCLEOTIDE ",\n                                                " C21 H29 N7 O14 P2 ",\n                                                " BOPGDPNILDQYTO-NNYOXOHSSA-N "]\n                ]}\n            '),
(44, '4OHU', 'Crystal structure of Mycobacterium tuberculosis InhA in complex with inhibitor PT92', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4OHU', '10.2210/pdb4ohu/pdb', 'OXIDOREDUCTASE/OXIDOREDUCTASE INHIBITOR', '2014-01-18', '2014-04-30', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.6', 'A Structural and Energetic Model for the Slow-Onset Inhibition of the Mycobacterium tuberculosis Enoyl-ACP Reductase InhA.', 'Li, H.J.#Lai, C.T.#Pan, P.#Yu, W.#Liu, N.#Bommineni, G.R.#Garcia-Diaz, M.#Simmerling, C.#Tonge, P.J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '2-(2-BROMOPHENOXY)-5-HEXYLPHENOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 2TK ",\n                                                " 2-(2-BROMOPHENOXY)-5-HEXYLPHENOL ",\n                                                " C18 H21 BR O2 ",\n                                                " XMFJZZCTDKMLKQ-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(45, '4OIM', 'Crystal structure of Mycobacterium tuberculosis InhA in complex with inhibitor PT119 in 2.4 M acetate', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4OIM', '10.2210/pdb4oim/pdb', 'OXIDOREDUCTASE/OXIDOREDUCTASE INHIBITOR', '2014-01-20', '2014-04-23', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.85', 'Time-Dependent Diaryl Ether Inhibitors of InhA: Structure-Activity Relationship Studies of Enzyme Inhibition, Antibacterial Activity, and in vivo Efficacy.', 'Pan, P.#Knudson, S.E.#Bommineni, G.R.#Li, H.J.#Lai, C.T.#Liu, N.#Garcia-Diaz, M.#Simmerling, C.#Patil, S.S.#Slayden, R.A.#Tonge, P.J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'ACETATE ION,2-(2-CYANOPHENOXY)-5-HEXYLPHENOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ACT ",\n                                                " ACETATE ION ",\n                                                " C2 H3 O2 -1 ",\n                                                " QTBSBXVTEAMEQO-UHFFFAOYSA-M "]\n                , [" JUS ",\n                                                " 2-(2-CYANOPHENOXY)-5-HEXYLPHENOL ",\n                                                " C19 H21 N O2 ",\n                                                " RPZKERMNVCALKE-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(46, '4OXK', 'Multiple binding modes of inhibitor PT155 to the Mycobacterium tuberculosis enoyl-ACP reductase InhA within a tetramer', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4OXK', '10.2210/pdb4oxk/pdb', 'oxidoreductase/oxidoreductase inhibitor', '2014-02-05', '2014-04-30', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.84', 'A Structural and Energetic Model for the Slow-Onset Inhibition of the Mycobacterium tuberculosis Enoyl-ACP Reductase InhA.', 'Li, H.J.#Lai, C.T.#Pan, P.#Yu, W.#Liu, N.#Bommineni, G.R.#Garcia-Diaz, M.#Simmerling, C.#Tonge, P.J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '5-(4-amino-2-methylphenoxy)-2-hexyl-4-hydroxy-1-methylpyridinium,3,6,9,12,15-pentaoxaoctadecan-17-amine,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 1S5 ",\n                                                " 5-(4-amino-2-methylphenoxy)-2-hexyl-4-hydroxy-1-methylpyridinium ",\n                                                " C19 H26 N2 O2 ",\n                                                " YALGUPBMXZEXMF-UHFFFAOYSA-O "]\n                , [" 2NV ",\n                                                " 3,6,9,12,15-pentaoxaoctadecan-17-amine ",\n                                                " C13 H29 N O5 ",\n                                                " LPPZTQZWMYYVLK-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(47, '4OXN', 'Substrate-like binding mode of inhibitor PT155 to the Mycobacterium tuberculosis enoyl-ACP reductase InhA', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4OXN', '10.2210/pdb4oxn/pdb', 'OXIDOREDUCTASE/OXIDOREDUCTASE INHIBITOR', '2014-02-05', '2014-04-30', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.29', 'A Structural and Energetic Model for the Slow-Onset Inhibition of the Mycobacterium tuberculosis Enoyl-ACP Reductase InhA.', 'Li, H.J.#Lai, C.T.#Pan, P.#Yu, W.#Liu, N.#Bommineni, G.R.#Garcia-Diaz, M.#Simmerling, C.#Tonge, P.J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '5-(4-amino-2-methylphenoxy)-2-hexyl-4-hydroxy-1-methylpyridinium,3,6,9,12,15-pentaoxaoctadecan-17-amine,CHLORIDE ION,4-(2-HYDROXYETHYL)-1-PIPERAZINE ETHANESULFONIC ACID,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 1S5 ",\n                                                " 5-(4-amino-2-methylphenoxy)-2-hexyl-4-hydroxy-1-methylpyridinium ",\n                                                " C19 H26 N2 O2 ",\n                                                " YALGUPBMXZEXMF-UHFFFAOYSA-O "]\n                , [" 2NV ",\n                                                " 3,6,9,12,15-pentaoxaoctadecan-17-amine ",\n                                                " C13 H29 N O5 ",\n                                                " LPPZTQZWMYYVLK-UHFFFAOYSA-N "]\n                , [" CL ",\n                                                " CHLORIDE ION ",\n                                                " Cl -1 ",\n                                                " VEXZGXHMUGYJMC-UHFFFAOYSA-M "]\n                , [" EPE ",\n                                                " 4-(2-HYDROXYETHYL)-1-PIPERAZINE ETHANESULFONIC ACID ",\n                                                " C8 H18 N2 O4 S ",\n                                                " JKMHFZQWWAIEOD-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(48, '4OXY', 'Substrate-binding loop movement with inhibitor PT10 in the tetrameric Mycobacterium tuberculosis enoyl-ACP reductase InhA', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4OXY', '10.2210/pdb4oxy/pdb', 'OXIDOREDUCTASE/OXIDOREDUCTASE INHIBITOR', '2014-02-10', '2014-04-30', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.35', 'A Structural and Energetic Model for the Slow-Onset Inhibition of the Mycobacterium tuberculosis Enoyl-ACP Reductase InhA.', 'Li, H.J.#Lai, C.T.#Pan, P.#Yu, W.#Liu, N.#Bommineni, G.R.#Garcia-Diaz, M.#Simmerling, C.#Tonge, P.J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '5-hexyl-2-(2-nitrophenoxy)phenol,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 1TN ",\n                                                " 5-hexyl-2-(2-nitrophenoxy)phenol ",\n                                                " C18 H21 N O4 ",\n                                                " ASCHWJRXPZBPHV-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(49, '4OYR', 'Competition of the small inhibitor PT91 with large fatty acyl substrate of the Mycobacterium tuberculosis enoyl-ACP reductase InhA by induced substrate-binding loop refolding', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4OYR', '10.2210/pdb4oyr/pdb', 'OXIDOREDUCTASE/OXIDOREDUCTASE INHIBITOR', '2014-02-13', '2014-04-30', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.3', 'A Structural and Energetic Model for the Slow-Onset Inhibition of the Mycobacterium tuberculosis Enoyl-ACP Reductase InhA.', 'Li, H.J.#Lai, C.T.#Pan, P.#Yu, W.#Liu, N.#Bommineni, G.R.#Garcia-Diaz, M.#Simmerling, C.#Tonge, P.J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '2-(2-chloranylphenoxy)-5-hexyl-phenol,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 1US ",\n                                                " 2-(2-chloranylphenoxy)-5-hexyl-phenol ",\n                                                " C18 H21 Cl O2 ",\n                                                " QXCNLBOWURGQPC-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(50, '4QXM', 'Crystal structure of the InhA:GSK_SB713 complex', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4QXM', '10.2210/pdb4qxm/pdb', 'OXIDOREDUCTASE/OXIDOREDUCTASE INHIBITOR', '2014-07-21', '2015-07-22', 'Mycobacterium tuberculosis', 'Escherichia coli BL21(DE3)', 'X-RAY DIFFRACTION', '2.2', 'N-Benzyl-4-((heteroaryl)methyl)benzamides: A New Class of Direct NADH-Dependent 2-trans Enoyl-Acyl Carrier Protein Reductase (InhA) Inhibitors with Antitubercular Activity.', 'Guardia, A.#Gulten, G.#Fernandez, R.#Gomez, J.#Wang, F.#Convery, M.#Blanco, D.#Martinez, M.#Perez-Herran, E.#Alonso, M.#Ortega, F.#Rullas, J.#Calvo, D.#Mata, L.#Young, R.#Sacchettini, J.C.#Mendoza-Losana, A.#Remuinan, M.#Ballell Pages, L.#Castro-Pichel, J.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,C,E,G ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'N-(2-CHLORO-4-FLUOROBENZYL)-4-[(3,5-DIMETHYL-1H-PYRAZOL-1-YL)METHYL]BENZAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 713 ",\n                                                " N-(2-CHLORO-4-FLUOROBENZYL)-4-[(3,5-DIMETHYL-1H-PYRAZOL-1-YL)METHYL]BENZAMIDE ",\n                                                " C20 H19 CL F N3 O ",\n                                                " PUPDWYSTSLHVKJ-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(51, '4R9R', 'Mycobacterium tuberculosis InhA bound to NITD-564', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4R9R', '10.2210/pdb4r9r/pdb', 'OXIDOREDUCTASE', '2014-09-07', '2015-01-21', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.9', 'Direct inhibitors of InhA are active against Mycobacterium tuberculosis', 'Manjunatha, U.H.#Rao, S.P.S.#Kondreddi, R.R.#Noble, C.G.#Camacho, L.R.#Tan, B.H.#Ng, S.H.#Ng, P.S.#Ma, N.L.#Lakshminarayana, S.B.#Herve, M.#Barnes, S.W.#Yu, W.#Kuhen, K.#Blasco, F.#Beer, D.#Walker, J.R.#Tonge, P.J.#Glynne, R.#Smith, P.W.#Diagana, T.T.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,C,E,G ",\n                                               " 272 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '6-(CYCLOHEXYLMETHYL)-4-HYDROXY-3-PHENYLPYRIDIN-2(1H)-ONE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 3KX ",\n                                                " 6-(CYCLOHEXYLMETHYL)-4-HYDROXY-3-PHENYLPYRIDIN-2(1H)-ONE ",\n                                                " C18 H21 N O2 ",\n                                                " UVVRYOIOMHFQGU-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            ');
INSERT INTO `pdb_InhA` (`id`, `estrutura`, `descricao`, `link`, `doi`, `classificacao`, `depositado`, `lancado`, `organismo`, `expressao`, `metodo`, `resolucao`, `artigoorigem`, `artigoautores`, `macromolecula`, `ligantes`, `tabelaligantes`) VALUES
(52, '4R9S', 'Mycobacterium tuberculosis InhA bound to NITD-916', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4R9S', '10.2210/pdb4r9s/pdb', 'OXIDOREDUCTASE', '2014-09-07', '2015-01-21', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '3.2', 'Direct inhibitors of InhA are active against Mycobacterium tuberculosis', 'Manjunatha, U.H.#Rao, S.P.S.#Kondreddi, R.R.#Noble, C.G.#Camacho, L.R.#Tan, B.H.#Ng, S.H.#Ng, P.S.#Ma, N.L.#Lakshminarayana, S.B.#Herve, M.#Barnes, S.W.#Yu, W.#Kuhen, K.#Blasco, F.#Beer, D.#Walker, J.R.#Tonge, P.J.#Glynne, R.#Smith, P.W.#Diagana, T.T.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,C,E,G ",\n                                               " 272 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '6-[(4,4-DIMETHYLCYCLOHEXYL)METHYL]-4-HYDROXY-3-PHENYLPYRIDIN-2(1H)-ONE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 3KY ",\n                                                " 6-[(4,4-DIMETHYLCYCLOHEXYL)METHYL]-4-HYDROXY-3-PHENYLPYRIDIN-2(1H)-ONE ",\n                                                " C20 H25 N O2 ",\n                                                " WKDRRKVUJLNFSR-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(53, '4TRJ', 'Crystal structure of Mycobacterium tuberculosis enoyl reductase (INHA) complexed with N-(3-bromophenyl)-1-cyclohexyl-5-oxopyrrolidine-3-carboxamide, refined with new ligand restraints', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4TRJ', '10.2210/pdb4trj/pdb', 'OXIDOREDUCTASE', '2014-06-17', '2014-08-13', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.73', 'Pyrrolidine carboxamides as a novel class of inhibitors of enoyl acyl carrier protein reductase from Mycobacterium tuberculosis', 'He, X.#Alian, A.#Stroud, R.M.#Ortiz de Montellano, P.R.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#MT1531 "]\n                ]}\n            ', '(3S)-N-(3-BROMOPHENYL)-1-CYCLOHEXYL-5-OXOPYRROLIDINE-3-CARBOXAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 665 ",\n                                                " (3S)-N-(3-BROMOPHENYL)-1-CYCLOHEXYL-5-OXOPYRROLIDINE-3-CARBOXAMIDE ",\n                                                " C17 H21 Br N2 O2 ",\n                                                " MVPIURCUINFSAB-LBPRGKRZSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(54, '4TRM', 'Structure of the apo form of InhA from Mycobacterium tuberculosis', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4TRM', '10.2210/pdb4trm/pdb', 'OXIDOREDUCTASE', '2014-06-17', '2015-04-29', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.8', 'Crystal structure of the enoyl-ACP reductase of Mycobacterium tuberculosis (InhA) in the apo-form and in complex with the active metabolite of isoniazid pre-formed by a biomimetic approach.', 'Chollet, A.#Mourey, L.#Lherbet, C.#Delbot, A.#Julien, S.#Baltas, M.#Bernadou, J.#Pratviel, G.#Maveyraud, L.#Bernardes-Genisson, V.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D,E,F ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '2-(N-MORPHOLINO)-ETHANESULFONIC ACID', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" MES ",\n                                                " 2-(N-MORPHOLINO)-ETHANESULFONIC ACID ",\n                                                " C6 H13 N O4 S ",\n                                                " SXGZJKUKBWWHRA-UHFFFAOYSA-N "]\n                ]}\n            '),
(55, '4TRN', 'STRUCTURE OF INHA FROM MYCOBACTERIUM TUBERCULOSIS COMPLEXED TO NADH', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4TRN', '10.2210/pdb4trn/pdb', 'OXIDOREDUCTASE', '2014-06-17', '2015-04-29', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.95', 'Crystal structure of the enoyl-ACP reductase of Mycobacterium tuberculosis (InhA) in the apo-form and in complex with the active metabolite of isoniazid pre-formed by a biomimetic approach.', 'Chollet, A.#Mourey, L.#Lherbet, C.#Delbot, A.#Julien, S.#Baltas, M.#Bernadou, J.#Pratviel, G.#Maveyraud, L.#Bernardes-Genisson, V.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" INHA ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'DIMETHYL SULFOXIDE,(4S)-2-METHYL-2,4-PENTANEDIOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" DMS ",\n                                                " DIMETHYL SULFOXIDE ",\n                                                " C2 H6 O S ",\n                                                " IAZDPXIOMUYVGZ-UHFFFAOYSA-N "]\n                , [" MPD ",\n                                                " (4S)-2-METHYL-2,4-PENTANEDIOL ",\n                                                " C6 H14 O2 ",\n                                                " SVTBMSDMJJWYQN-YFKPBYRVSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(56, '4TRO', 'Structure of the enoyl-ACP reductase of Mycobacterium tuberculosis InhA, inhibited with the active metabolite of isoniazid', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4TRO', '10.2210/pdb4tro/pdb', 'OXIDOREDUCTASE', '2014-06-17', '2015-04-29', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.4', 'Crystal structure of the enoyl-ACP reductase of Mycobacterium tuberculosis (InhA) in the apo-form and in complex with the active metabolite of isoniazid pre-formed by a biomimetic approach.', 'Chollet, A.#Mourey, L.#Lherbet, C.#Delbot, A.#Julien, S.#Baltas, M.#Bernadou, J.#Pratviel, G.#Maveyraud, L.#Bernardes-Genisson, V.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'DIMETHYL SULFOXIDE,4-(2-HYDROXYETHYL)-1-PIPERAZINE ETHANESULFONIC ACID,SODIUM ION,NICOTINAMIDE-ADENINE-DINUCLEOTIDE,ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" DMS ",\n                                                " DIMETHYL SULFOXIDE ",\n                                                " C2 H6 O S ",\n                                                " IAZDPXIOMUYVGZ-UHFFFAOYSA-N "]\n                , [" EPE ",\n                                                " 4-(2-HYDROXYETHYL)-1-PIPERAZINE ETHANESULFONIC ACID ",\n                                                " C8 H18 N2 O4 S ",\n                                                " JKMHFZQWWAIEOD-UHFFFAOYSA-N "]\n                , [" NA ",\n                                                " SODIUM ION ",\n                                                " Na 1 ",\n                                                " FKNQFGJONOIPTF-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" ZID ",\n                                                " ISONICOTINIC-ACETYL-NICOTINAMIDE-ADENINE DINUCLEOTIDE ",\n                                                " C27 H30 N8 O15 P2 ",\n                                                " SURAWYIAXPVHGO-XDBKRARRSA-N "]\n                ]}\n            '),
(57, '4TZK', 'Crystal structure of Mycobacterium tuberculosis enoyl reductase (INHA) complexed WITH 1-CYCLOHEXYL-N-(3,5-DICHLOROPHENYL)-5-OXOPYRROLIDINE-3-CARBOXAMIDE', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4TZK', '10.2210/pdb4tzk/pdb', 'OXIDOREDUCTASE', '2014-07-10', '2014-08-20', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.62', 'Pyrrolidine carboxamides as a novel class of inhibitors of enoyl acyl carrier protein reductase from Mycobacterium tuberculosis', 'He, X.#Alian, A.#Stroud, R.M.#Ortiz de Montellano, P.R.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '(3S)-1-CYCLOHEXYL-N-(3,5-DICHLOROPHENYL)-5-OXOPYRROLIDINE-3-CARBOXAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 641 ",\n                                                " (3S)-1-CYCLOHEXYL-N-(3,5-DICHLOROPHENYL)-5-OXOPYRROLIDINE-3-CARBOXAMIDE ",\n                                                " C17 H20 Cl2 N2 O2 ",\n                                                " YUFADRZDHJKVOT-NSHDSACASA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(58, '4TZT', 'CRYSTAL STRUCTURE OF MYCOBACTERIUM TUBERCULOSIS ENOYL REDUCTASE (INHA) COMPLEXED WITH N-(3-CHLORO-2-METHYLPHENYL)-1-CYCLOHEXYL- 5-OXOPYRROLIDINE-3-CARBOXAMIDE', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4TZT', '10.2210/pdb4tzt/pdb', 'OXIDOREDUCTASE', '2014-07-10', '2014-08-20', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.86', 'Pyrrolidine carboxamides as a novel class of inhibitors of enoyl acyl carrier protein reductase from Mycobacterium tuberculosis', 'He, X.#Alian, A.#Stroud, R.M.#Ortiz de Montellano, P.R.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '(3S)-N-(3-CHLORO-2-METHYLPHENYL)-1-CYCLOHEXYL-5-OXOPYRROLIDINE-3-CARBOXAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 468 ",\n                                                " (3S)-N-(3-CHLORO-2-METHYLPHENYL)-1-CYCLOHEXYL-5-OXOPYRROLIDINE-3-CARBOXAMIDE ",\n                                                " C18 H23 Cl N2 O2 ",\n                                                " NJNMAZNXKKBTPS-ZDUSSCGKSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(59, '4U0J', 'Crystal structure of Mycobacterium tuberculosis enoyl reductase (INHA) complexed with 1-CYCLOHEXYL-5-OXO-N-PHENYLPYRROLIDINE-3-CARBOXAMIDE, refined with new ligand restraints', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4U0J', '10.2210/pdb4u0j/pdb', 'OXIDOREDUCTASE', '2014-07-11', '2014-07-30', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.62', 'Pyrrolidine carboxamides as a novel class of inhibitors of enoyl acyl carrier protein reductase from Mycobacterium tuberculosis', 'He, X.#Alian, A.#Stroud, R.M.#Ortiz de Montellano, P.R.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '(3S)-1-CYCLOHEXYL-5-OXO-N-PHENYLPYRROLIDINE-3-CARBOXAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 566 ",\n                                                " (3S)-1-CYCLOHEXYL-5-OXO-N-PHENYLPYRROLIDINE-3-CARBOXAMIDE ",\n                                                " C17 H22 N2 O2 ",\n                                                " BVUSHGJZBZMDML-ZDUSSCGKSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(60, '4U0K', 'Crystal structure of Mycobacterium tuberculosis enoyl reductase complexed with N-(5-chloro-2-methylphenyl)-1-cyclohexyl-5-oxopyrrolidine-3-carboxamide', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4U0K', '10.2210/pdb4u0k/pdb', 'OXIDOREDUCTASE', '2014-07-11', '2014-07-30', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '1.9', 'Pyrrolidine carboxamides as a novel class of inhibitors of enoyl acyl carrier protein reductase from Mycobacterium tuberculosis', 'He, X.#Alian, A.#Stroud, R.M.#Ortiz de Montellano, P.R.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '(3S)-N-(5-CHLORO-2-METHYLPHENYL)-1-CYCLOHEXYL-5-OXOPYRROLIDINE-3-CARBOXAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 744 ",\n                                                " (3S)-N-(5-CHLORO-2-METHYLPHENYL)-1-CYCLOHEXYL-5-OXOPYRROLIDINE-3-CARBOXAMIDE ",\n                                                " C18 H23 Cl N2 O2 ",\n                                                " RJWMDETWDDESBP-ZDUSSCGKSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(61, '4UVD', 'Discovery of pyrimidine isoxazoles InhA in complex with compound 6', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4UVD', '10.2210/pdb4uvd/pdb', 'OXIDOREDUCTASE', '2014-08-05', '2015-09-30', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '1.82', 'Hitting the Target in More Than One Way: Novel, Direct Inhibitors of Mycobacterium Tuberculosis Enoyl Acp Reductase', 'Madhavapeddi, P.#Kale, R.R.#Cowen, S.D.#Ghorpade, S.R.#Davies, G.#Bellale, E.V.#Kale, M.G.#Srivastava, A.#Spadola, L.#Kawatkar, A.#Raichurkar, A.V.#Tonge, M.#Nandishaiah, R.#Guptha, S.#Narayan, A.#Gingell, H.#Plant, D.#Landge, S.#Menasinakai, S.#Prabhakar, K.R.#Achar, V.#Ambady, A.#Sambandamurthy, V.K.#Ramachandran, V.#Panduga, V.#Reddy, J.#Kumar, C.N.N.#Kaur, P.#Shandil, R.#Iyer, P.S.#Narayanan, S.#Read, J.A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '2-[(4,6-DIMETHYLPYRIMIDIN-2-YL)SULFANYL]-N-[(2Z)-5-[3-(TRIFLUOROMETHYL)BENZYL]-1,3-THIAZOL-2(3H)-YLIDENE]ACETAMIDE,MAGNESIUM ION,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" HRW ",\n                                                " 2-[(4,6-DIMETHYLPYRIMIDIN-2-YL)SULFANYL]-N-[(2Z)-5-[3-(TRIFLUOROMETHYL)BENZYL]-1,3-THIAZOL-2(3H)-YLIDENE]ACETAMIDE ",\n                                                " C19 H17 F3 N4 O S2 ",\n                                                " WWJGDLQISVVNRI-UHFFFAOYSA-N "]\n                , [" MG ",\n                                                " MAGNESIUM ION ",\n                                                " MG 2 ",\n                                                " JLVVSXFLKOJNIY-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(62, '4UVE', 'Discovery of pyrimidine isoxazoles InhA in complex with compound 9', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4UVE', '10.2210/pdb4uve/pdb', 'OXIDOREDUCTASE', '2014-08-05', '2015-09-30', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '1.99', 'Hitting the Target in More Than One Way: Novel, Direct Inhibitors of Mycobacterium Tuberculosis Enoyl Acp Reductase', 'Madhavapeddi, P.#Kale, R.R.#Cowen, S.D.#Ghorpade, S.R.#Davies, G.#Bellale, E.V.#Kale, M.G.#Srivastava, A.#Spadola, L.#Kawatkar, A.#Raichurkar, A.V.#Tonge, M.#Nandishaiah, R.#Guptha, S.#Narayan, A.#Gingell, H.#Plant, D.#Landge, S.#Menasinakai, S.#Prabhakar, K.R.#Achar, V.#Ambady, A.#Sambandamurthy, V.K.#Ramachandran, V.#Panduga, V.#Reddy, J.#Kumar, C.N.N.#Kaur, P.#Shandil, R.#Iyer, P.S.#Narayanan, S.#Read, J.A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '2-(4,6-DIMETHYLPYRIMIDIN-2-YL)SULFANYLETHANOL,MAGNESIUM ION,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" KI5 ",\n                                                " 2-(4,6-DIMETHYLPYRIMIDIN-2-YL)SULFANYLETHANOL ",\n                                                " C8 H12 N2 O S ",\n                                                " DSRGZRHOYJEVPF-UHFFFAOYSA-N "]\n                , [" MG ",\n                                                " MAGNESIUM ION ",\n                                                " MG 2 ",\n                                                " JLVVSXFLKOJNIY-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(63, '4UVG', 'Discovery of pyrimidine isoxazoles InhA in complex with compound 15', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4UVG', '10.2210/pdb4uvg/pdb', 'OXIDOREDUCTASE', '2014-08-05', '2015-09-30', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '1.92', 'Hitting the Target in More Than One Way: Novel, Direct Inhibitors of Mycobacterium Tuberculosis Enoyl Acp Reductase', 'Madhavapeddi, P.#Kale, R.R.#Cowen, S.D.#Ghorpade, S.R.#Davies, G.#Bellale, E.V.#Kale, M.G.#Srivastava, A.#Spadola, L.#Kawatkar, A.#Raichurkar, A.V.#Tonge, M.#Nandishaiah, R.#Guptha, S.#Narayan, A.#Gingell, H.#Plant, D.#Landge, S.#Menasinakai, S.#Prabhakar, K.R.#Achar, V.#Ambady, A.#Sambandamurthy, V.K.#Ramachandran, V.#Panduga, V.#Reddy, J.#Kumar, C.N.N.#Kaur, P.#Shandil, R.#Iyer, P.S.#Narayanan, S.#Read, J.A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'PENTAETHYLENE GLYCOL,5-[(4,6-DIMETHYLPYRIMIDIN-2-YL)SULFANYLMETHYL]ISOXAZOLE-3-CARBOXAMIDE,MAGNESIUM ION,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 1PE ",\n                                                " PENTAETHYLENE GLYCOL ",\n                                                " C10 H22 O6 ",\n                                                " JLFNLZLINWHATN-UHFFFAOYSA-N "]\n                , [" A0C ",\n                                                " 5-[(4,6-DIMETHYLPYRIMIDIN-2-YL)SULFANYLMETHYL]ISOXAZOLE-3-CARBOXAMIDE ",\n                                                " C11 H12 N4 O2 S ",\n                                                " LDIXARLLVUNHKY-UHFFFAOYSA-N "]\n                , [" MG ",\n                                                " MAGNESIUM ION ",\n                                                " MG 2 ",\n                                                " JLVVSXFLKOJNIY-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(64, '4UVH', 'Discovery of pyrimidine isoxazoles InhA in complex with compound 10', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4UVH', '10.2210/pdb4uvh/pdb', 'OXIDOREDUCTASE', '2014-08-05', '2015-09-30', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '1.89', 'Hitting the Target in More Than One Way: Novel, Direct Inhibitors of Mycobacterium Tuberculosis Enoyl Acp Reductase', 'Madhavapeddi, P.#Kale, R.R.#Cowen, S.D.#Ghorpade, S.R.#Davies, G.#Bellale, E.V.#Kale, M.G.#Srivastava, A.#Spadola, L.#Kawatkar, A.#Raichurkar, A.V.#Tonge, M.#Nandishaiah, R.#Guptha, S.#Narayan, A.#Gingell, H.#Plant, D.#Landge, S.#Menasinakai, S.#Prabhakar, K.R.#Achar, V.#Ambady, A.#Sambandamurthy, V.K.#Ramachandran, V.#Panduga, V.#Reddy, J.#Kumar, C.N.N.#Kaur, P.#Shandil, R.#Iyer, P.S.#Narayanan, S.#Read, J.A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B,C,D ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'ACETATE ION,NICOTINAMIDE-ADENINE-DINUCLEOTIDE,N-(1,3-BENZOTHIAZOL-2-YL)ACETAMIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" ACT ",\n                                                " ACETATE ION ",\n                                                " C2 H3 O2 -1 ",\n                                                " QTBSBXVTEAMEQO-UHFFFAOYSA-M "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" UUD ",\n                                                " N-(1,3-BENZOTHIAZOL-2-YL)ACETAMIDE ",\n                                                " C9 H8 N2 O S ",\n                                                " RIYZWBJLPINWTC-UHFFFAOYSA-N "]\n                ]}\n            '),
(65, '4UVI', 'Discovery of pyrimidine isoxazoles InhA in complex with compound 23', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=4UVI', '10.2210/pdb4uvi/pdb', 'OXIDOREDUCTASE', '2014-08-05', '2015-09-30', 'Mycobacterium tuberculosis', 'ESCHERICHIA COLI', 'X-RAY DIFFRACTION', '1.73', 'Hitting the Target in More Than One Way: Novel, Direct Inhibitors of Mycobacterium Tuberculosis Enoyl Acp Reductase', 'Madhavapeddi, P.#Kale, R.R.#Cowen, S.D.#Ghorpade, S.R.#Davies, G.#Bellale, E.V.#Kale, M.G.#Srivastava, A.#Spadola, L.#Kawatkar, A.#Raichurkar, A.V.#Tonge, M.#Nandishaiah, R.#Guptha, S.#Narayan, A.#Gingell, H.#Plant, D.#Landge, S.#Menasinakai, S.#Prabhakar, K.R.#Achar, V.#Ambady, A.#Sambandamurthy, V.K.#Ramachandran, V.#Panduga, V.#Reddy, J.#Kumar, C.N.N.#Kaur, P.#Shandil, R.#Iyer, P.S.#Narayanan, S.#Read, J.A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" ENOYL-[ACYL-CARRIER-PROTEIN] REDUCTASE [NADH] ",\n                                               " A,B,C,D ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '5-{[(4,6-DIMETHYLPYRIMIDIN-2-YL)SULFANYL]METHYL}-N-[(2-METHYLPYRIDIN-4-YL)METHYL]-1,2-OXAZOLE-3-CARBOXAMIDE,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" KXU ",\n                                                " 5-{[(4,6-DIMETHYLPYRIMIDIN-2-YL)SULFANYL]METHYL}-N-[(2-METHYLPYRIDIN-4-YL)METHYL]-1,2-OXAZOLE-3-CARBOXAMIDE ",\n                                                " C18 H19 N5 O2 S ",\n                                                " FPLTYHWZNYWMJS-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(66, '5COQ', 'The effect of valine to alanine mutation on InhA enzyme crystallization pattern and substrate binding loop conformation and flexibility', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=5COQ', '10.2210/pdb5coq/pdb', 'OXIDOREDUCTASE', '2015-07-20', '2015-08-05', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.3', 'Rational Modulation of the Induced-Fit Conformational Change for Slow-Onset Inhibition in Mycobacterium tuberculosis InhA.', 'Lai, C.T.#Li, H.J.#Yu, W.#Shah, S.#Bommineni, G.R.#Perrone, V.#Garcia-Diaz, M.#Tonge, P.J.#Simmerling, C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'SODIUM ION,NICOTINAMIDE-ADENINE-DINUCLEOTIDE,5-HEXYL-2-(2-METHYLPHENOXY)PHENOL', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NA ",\n                                                " SODIUM ION ",\n                                                " Na 1 ",\n                                                " FKNQFGJONOIPTF-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" TCU ",\n                                                " 5-HEXYL-2-(2-METHYLPHENOXY)PHENOL ",\n                                                " C19 H24 O2 ",\n                                                " KSWHNTPMGZKIOB-UHFFFAOYSA-N "]\n                ]}\n            '),
(67, '5CP8', 'The effect of isoleucine to alanine mutation on InhA enzyme crystallization pattern and substrate binding loop conformation and flexibility', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=5CP8', '10.2210/pdb5cp8/pdb', 'OXIDOREDUCTASE', '2015-07-21', '2015-08-05', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.4', 'Rational Modulation of the Induced-Fit Conformational Change for Slow-Onset Inhibition in Mycobacterium tuberculosis InhA.', 'Lai, C.T.#Li, H.J.#Yu, W.#Shah, S.#Bommineni, G.R.#Perrone, V.#Garcia-Diaz, M.#Tonge, P.J.#Simmerling, C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', '4-(2-HYDROXYETHYL)-1-PIPERAZINE ETHANESULFONIC ACID,2-{2-[2-2-(METHOXY-ETHOXY)-ETHOXY]-ETHOXY}-ETHANOL,NICOTINAMIDE-ADENINE-DINUCLEOTIDE,5-HEXYL-2-(2-METHYLPHENOXY)PHENOL', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" EPE ",\n                                                " 4-(2-HYDROXYETHYL)-1-PIPERAZINE ETHANESULFONIC ACID ",\n                                                " C8 H18 N2 O4 S ",\n                                                " JKMHFZQWWAIEOD-UHFFFAOYSA-N "]\n                , [" ETE ",\n                                                " 2-{2-[2-2-(METHOXY-ETHOXY)-ETHOXY]-ETHOXY}-ETHANOL ",\n                                                " C9 H20 O5 ",\n                                                " ZNYRFEPBTVGZDN-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                , [" TCU ",\n                                                " 5-HEXYL-2-(2-METHYLPHENOXY)PHENOL ",\n                                                " C19 H24 O2 ",\n                                                " KSWHNTPMGZKIOB-UHFFFAOYSA-N "]\n                ]}\n            '),
(68, '5CPB', 'The effect of isoleucine to alanine mutation on InhA enzyme crystallization pattern and inhibition by ligand PT70 (TCU)', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=5CPB', '10.2210/pdb5cpb/pdb', 'OXIDOREDUCTASE', '2015-07-21', '2015-08-19', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.0', 'Rational Modulation of the Induced-Fit Conformational Change for Slow-Onset Inhibition in Mycobacterium tuberculosis InhA.', 'Lai, C.T.#Li, H.J.#Yu, W.#Shah, S.#Bommineni, G.R.#Perrone, V.#Garcia-Diaz, M.#Tonge, P.J.#Simmerling, C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D,E,F ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(69, '5CPF', 'Compensation of the effect of isoleucine to alanine mutation by designed inhibition in the InhA enzyme', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=5CPF', '10.2210/pdb5cpf/pdb', 'OXIDOREDUCTASE', '2015-07-21', '2015-08-12', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '3.41', 'Rational Modulation of the Induced-Fit Conformational Change for Slow-Onset Inhibition in Mycobacterium tuberculosis InhA.', 'Lai, C.T.#Li, H.J.#Yu, W.#Shah, S.#Bommineni, G.R.#Perrone, V.#Garcia-Diaz, M.#Tonge, P.J.#Simmerling, C.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 289 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#MT1531 "]\n                ]}\n            ', '2-(2-methylphenoxy)-5-[(4-phenyl-1H-1,2,3-triazol-1-yl)methyl]phenol,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 53K ",\n                                                " 2-(2-methylphenoxy)-5-[(4-phenyl-1H-1,2,3-triazol-1-yl)methyl]phenol ",\n                                                " C22 H19 N3 O2 ",\n                                                " RXXUNJCEEGMOQJ-UHFFFAOYSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            '),
(70, '5JFO', 'Structure of the M.tuberculosis enoyl-reductase InhA in complex with GSK625', 'http://www.rcsb.org/pdb/explore/explore.do?structureId=5JFO', '10.2210/pdb5jfo/pdb', 'OXIDOREDUCTASE/OXIDOREDUCTASE INHIBITOR', '2016-04-19', '2016-08-10', 'Mycobacterium tuberculosis', 'Escherichia coli', 'X-RAY DIFFRACTION', '2.91', 'Antitubercular drugs for an old target: GSK693 as a promising InhA direct inhibitor.', 'Martinez-Hoyos, M.#Perez-Herran, E.#Gulten, G.#Encinas, L.#Alvarez-Gomez, D.#Alvarez, E.#Ferrer-Bazaga, S.#Garcia-Perez, A.#Ortega, F.#Angulo-Barturen, I.#Rullas-Trincado, J.#Blanco Ruano, D.#Torres, P.#Castaneda, P.#Huss, S.#Fernandez Menendez, R.#Gonzalez Del Valle, S.#Ballell, L.#Barros, D.#Modha, S.#Dhar, N.#Signorino-Gelo, F.#McKinney, J.D.#Garcia-Bustos, J.F.#Lavandera, J.L.#Sacchettini, J.C.#Jimenez, M.S.#Martin-Casabona, N.#Castro-Pichel, J.#Mendoza-Losana, A.', '\n                {\n                    "nome":"Macromoléculas",\n                    "colunas":["Molécula","Cadeias","Tamanho","Organismo","Detalhes"],\n                    "valores":[[" Enoyl-[acyl-carrier-protein] reductase [NADH] ",\n                                               " A,B,C,D ",\n                                               " 269 ",\n                                               " Mycobacterium tuberculosis ",\n                                               "Gene Name(s): inhA#Rv1484#MTCY277.05 "]\n                ]}\n            ', 'N-{1-[(2-chloro-6-fluorophenyl)methyl]-1H-pyrazol-3-yl}-5-[(1S)-1-(3-methyl-1H-pyrazol-1-yl)ethyl]-1,3,4-thiadiazol-2-amine,NICOTINAMIDE-ADENINE-DINUCLEOTIDE', '\n                {\n                    "nome":"Ligantes",\n                    "colunas":["ID","Nome","Fórmula","InChIKey"],\n                    "valores":[[" 6KA ",\n                                                " N-{1-[(2-chloro-6-fluorophenyl)methyl]-1H-pyrazol-3-yl}-5-[(1S)-1-(3-methyl-1H-pyrazol-1-yl)ethyl]-1,3,4-thiadiazol-2-amine ",\n                                                " C18 H17 Cl F N7 S ",\n                                                " NXJALABMYVGINA-LBPRGKRZSA-N "]\n                , [" NAD ",\n                                                " NICOTINAMIDE-ADENINE-DINUCLEOTIDE ",\n                                                " C21 H27 N7 O14 P2 ",\n                                                " BAWFJGJZGIEFAR-NNYOXOHSSA-N "]\n                ]}\n            ');

-- --------------------------------------------------------

--
-- Table structure for table `proteinas`
--

CREATE TABLE IF NOT EXISTS `proteinas` (
`id` int(10) unsigned NOT NULL,
  `nome` varchar(511) NOT NULL,
  `dados` varchar(1023) DEFAULT NULL
) ENGINE=MyISAM AUTO_INCREMENT=153 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `proteinas`
--

INSERT INTO `proteinas` (`id`, `nome`, `dados`) VALUES
(1, 'InhA', NULL),
(151, 'Arginase', ''),
(152, 'Hemoglobin', NULL);

-- --------------------------------------------------------

--
-- Table structure for table `usuarios`
--

CREATE TABLE IF NOT EXISTS `usuarios` (
`id` int(10) unsigned NOT NULL,
  `username` varchar(63) NOT NULL,
  `password` char(32) NOT NULL,
  `authKey` varchar(256) DEFAULT NULL
) ENGINE=MyISAM AUTO_INCREMENT=2 DEFAULT CHARSET=latin1;

--
-- Dumping data for table `usuarios`
--

INSERT INTO `usuarios` (`id`, `username`, `password`, `authKey`) VALUES
(1, 'gustavo', '66c4e9d38f9c4256bf98d83e969a9563', NULL);

--
-- Indexes for dumped tables
--

--
-- Indexes for table `agendamentos`
--
ALTER TABLE `agendamentos`
 ADD PRIMARY KEY (`id`), ADD KEY `agendamento_ativo` (`ativo`);

--
-- Indexes for table `agendamentos_componentes`
--
ALTER TABLE `agendamentos_componentes`
 ADD PRIMARY KEY (`idAgendamento`,`idComponente`,`ordem`);

--
-- Indexes for table `artigos`
--
ALTER TABLE `artigos`
 ADD PRIMARY KEY (`id`), ADD KEY `proteina` (`proteina`) USING BTREE;

--
-- Indexes for table `autores`
--
ALTER TABLE `autores`
 ADD PRIMARY KEY (`id`);

--
-- Indexes for table `autores_artigos`
--
ALTER TABLE `autores_artigos`
 ADD PRIMARY KEY (`idArtigo`,`idAutor`);

--
-- Indexes for table `componentescoletarefinamento`
--
ALTER TABLE `componentescoletarefinamento`
 ADD PRIMARY KEY (`ID`);

--
-- Indexes for table `componentesvisuais`
--
ALTER TABLE `componentesvisuais`
 ADD PRIMARY KEY (`ID`), ADD UNIQUE KEY `nome` (`nome`);

--
-- Indexes for table `log`
--
ALTER TABLE `log`
 ADD PRIMARY KEY (`id`);

--
-- Indexes for table `pdb_Arginase`
--
ALTER TABLE `pdb_Arginase`
 ADD PRIMARY KEY (`id`);

--
-- Indexes for table `pdb_Hemoglobin`
--
ALTER TABLE `pdb_Hemoglobin`
 ADD PRIMARY KEY (`id`);

--
-- Indexes for table `pdb_InhA`
--
ALTER TABLE `pdb_InhA`
 ADD PRIMARY KEY (`id`);

--
-- Indexes for table `proteinas`
--
ALTER TABLE `proteinas`
 ADD PRIMARY KEY (`id`);

--
-- Indexes for table `usuarios`
--
ALTER TABLE `usuarios`
 ADD PRIMARY KEY (`id`) USING BTREE, ADD UNIQUE KEY `usuarios_nome` (`username`), ADD UNIQUE KEY `usuarios_authkey` (`authKey`);

--
-- AUTO_INCREMENT for dumped tables
--

--
-- AUTO_INCREMENT for table `agendamentos`
--
ALTER TABLE `agendamentos`
MODIFY `id` int(11) unsigned NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=22;
--
-- AUTO_INCREMENT for table `artigos`
--
ALTER TABLE `artigos`
MODIFY `id` int(11) unsigned NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=159;
--
-- AUTO_INCREMENT for table `autores`
--
ALTER TABLE `autores`
MODIFY `id` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=858;
--
-- AUTO_INCREMENT for table `componentescoletarefinamento`
--
ALTER TABLE `componentescoletarefinamento`
MODIFY `ID` int(10) unsigned NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=47;
--
-- AUTO_INCREMENT for table `componentesvisuais`
--
ALTER TABLE `componentesvisuais`
MODIFY `ID` int(11) NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=2;
--
-- AUTO_INCREMENT for table `log`
--
ALTER TABLE `log`
MODIFY `id` int(10) unsigned NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=436;
--
-- AUTO_INCREMENT for table `pdb_Arginase`
--
ALTER TABLE `pdb_Arginase`
MODIFY `id` int(10) unsigned NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=65;
--
-- AUTO_INCREMENT for table `pdb_Hemoglobin`
--
ALTER TABLE `pdb_Hemoglobin`
MODIFY `id` int(10) unsigned NOT NULL AUTO_INCREMENT;
--
-- AUTO_INCREMENT for table `pdb_InhA`
--
ALTER TABLE `pdb_InhA`
MODIFY `id` int(10) unsigned NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=71;
--
-- AUTO_INCREMENT for table `proteinas`
--
ALTER TABLE `proteinas`
MODIFY `id` int(10) unsigned NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=153;
--
-- AUTO_INCREMENT for table `usuarios`
--
ALTER TABLE `usuarios`
MODIFY `id` int(10) unsigned NOT NULL AUTO_INCREMENT,AUTO_INCREMENT=2;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
